# Identify genes with lineage-specific expression
cut -f1-9 /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/RNASeq/Berthelot2018Data/expressionSummary_averageTPMnormMedian.txt | grep -v NA > /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/RNASeq/Berthelot2018Data/expressionSummary_averageTPMnormMedian_modified.txt
cut -f1-14 /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/RNASeq/Berthelot2018Data/expressionSummary_averageTPMnormMedian.txt | grep -v NA > /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/RNASeq/Berthelot2018Data/expressionSummary_averageTPMnormMedian_modifiedPlus.txt
cut -f1-16 /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/RNASeq/Berthelot2018Data/expressionSummary_averageTPMnormMedian.txt | grep -v NA > /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/RNASeq/Berthelot2018Data/expressionSummary_averageTPMnormMedian_modifiedPlusPlus.txt
RScript /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/limmaCladeSpecificLiverExpr.r

# Identify OCRs with rodent-specific enhancer activity
awk 'BEGIN{OFS="\t"} $17 <= 1000000' /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/LiverPfenning_ATAC_out/atac/68eb27d1-23c7-4d42-80f9-b3dd46f5e18c/call-reproducibility_idr/execution/idr.optimal_peak.narrowPeak_inLiuAll_nonCDS_enhancerShort_closestTSS.bed > /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/LiverPfenning_ATAC_out/atac/68eb27d1-23c7-4d42-80f9-b3dd46f5e18c/call-reproducibility_idr/execution/idr.optimal_peak.narrowPeak_inLiuAll_nonCDS_enhancerShort_closestTSS_noMoreThan1Mb.bed
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/filterPeakName.py --unfilteredPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/LiverPfenning_ATAC_out/atac/68eb27d1-23c7-4d42-80f9-b3dd46f5e18c/call-reproducibility_idr/execution/idr.optimal_peak.narrowPeak_inLiuAll_nonCDS_enhancerShort_peakPredictions.txt --unfilteredPeakNameCol 0 --peakListFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/LiverPfenning_ATAC_out/atac/68eb27d1-23c7-4d42-80f9-b3dd46f5e18c/call-reproducibility_idr/execution/idr.optimal_peak.narrowPeak_inLiuAll_nonCDS_enhancerShort_closestTSS_noMoreThan1Mb.bed --peakNameCol 3 --outputFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/LiverPfenning_ATAC_out/atac/68eb27d1-23c7-4d42-80f9-b3dd46f5e18c/call-reproducibility_idr/execution/idr.optimal_peak.narrowPeak_inLiuAll_nonCDS_enhancerShort_peakPredictions_noMoreThan1MbFromTSS.txt

# Consider all OCRs within 1Mb of genes with rodent-specific expression
slopBed -i /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/RNASeq/Berthelot2018Data/RodentDiffGenesTSS.bed -g /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseGenome/mm10.chrom.sizes -b 1000000 > /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/RNASeq/Berthelot2018Data/RodentDiffGenesTSS_plusMinus1Mb.bed
sort -u -k1,1 -k2,2n -k3,3n /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/RNASeq/Berthelot2018Data/RodentDiffGenesTSS_plusMinus1Mb.bed | mergeBed -i stdin > /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/RNASeq/Berthelot2018Data/RodentDiffGenesTSS_plusMinus1Mb_merged.bed
intersectBed -wa -u -a /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/LiverPfenning_ATAC_out/atac/68eb27d1-23c7-4d42-80f9-b3dd46f5e18c/call-reproducibility_idr/execution/idr.optimal_peak.narrowPeak_inLiuAll_nonCDS_enhancerShort.narrowPeak.gz -b /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/RNASeq/Berthelot2018Data/RodentDiffGenesTSS_plusMinus1Mb_merged.bed > /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/LiverPfenning_ATAC_out/atac/68eb27d1-23c7-4d42-80f9-b3dd46f5e18c/call-reproducibility_idr/execution/idr.optimal_peak.narrowPeak_inLiuAll_nonCDS_enhancerShort_1MbFromRodentDiffExprGene.bed
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/filterPeakName.py --unfilteredPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/LiverPfenning_ATAC_out/atac/68eb27d1-23c7-4d42-80f9-b3dd46f5e18c/call-reproducibility_idr/execution/idr.optimal_peak.narrowPeak_inLiuAll_nonCDS_enhancerShort_peakPredictions.txt --unfilteredPeakNameCol 0 --peakListFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/LiverPfenning_ATAC_out/atac/68eb27d1-23c7-4d42-80f9-b3dd46f5e18c/call-reproducibility_idr/execution/idr.optimal_peak.narrowPeak_inLiuAll_nonCDS_enhancerShort_1MbFromRodentDiffExprGene.bed --peakNameCol 3 --outputFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/LiverPfenning_ATAC_out/atac/68eb27d1-23c7-4d42-80f9-b3dd46f5e18c/call-reproducibility_idr/execution/idr.optimal_peak.narrowPeak_inLiuAll_nonCDS_enhancerShort_peakPredictions_1MbFromRodentDiffExprGene.txt

# Consider all OCRs within 1Mb of genes with consistent expression that were highlighted in the RNA-seq manuscript
slopBed -i /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/RNASeq/Berthelot2018Data/ConsistentGenesTSS.bed -g /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseGenome/mm10.chrom.sizes -b 1000000 > /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/RNASeq/Berthelot2018Data/ConsistentGenesTSS_plusMinus1Mb.bed
sort -u -k1,1 -k2,2n -k3,3n /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/RNASeq/Berthelot2018Data/ConsistentGenesTSS_plusMinus1Mb.bed | mergeBed -i stdin > /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/RNASeq/Berthelot2018Data/ConsistentGenesTSS_plusMinus1Mb_merged.bed
intersectBed -wa -u -a /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/LiverPfenning_ATAC_out/atac/68eb27d1-23c7-4d42-80f9-b3dd46f5e18c/call-reproducibility_idr/execution/idr.optimal_peak.narrowPeak_inLiuAll_nonCDS_enhancerShort.narrowPeak.gz -b /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/RNASeq/Berthelot2018Data/ConsistentGenesTSS_plusMinus1Mb_merged.bed > /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/LiverPfenning_ATAC_out/atac/68eb27d1-23c7-4d42-80f9-b3dd46f5e18c/call-reproducibility_idr/execution/idr.optimal_peak.narrowPeak_inLiuAll_nonCDS_enhancerShort_1MbFromConsistentExprGene.bed
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/filterPeakName.py --unfilteredPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/LiverPfenning_ATAC_out/atac/68eb27d1-23c7-4d42-80f9-b3dd46f5e18c/call-reproducibility_idr/execution/idr.optimal_peak.narrowPeak_inLiuAll_nonCDS_enhancerShort_peakPredictions.txt --unfilteredPeakNameCol 0 --peakListFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/LiverPfenning_ATAC_out/atac/68eb27d1-23c7-4d42-80f9-b3dd46f5e18c/call-reproducibility_idr/execution/idr.optimal_peak.narrowPeak_inLiuAll_nonCDS_enhancerShort_1MbFromConsistentExprGene.bed --peakNameCol 3 --outputFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/LiverPfenning_ATAC_out/atac/68eb27d1-23c7-4d42-80f9-b3dd46f5e18c/call-reproducibility_idr/execution/idr.optimal_peak.narrowPeak_inLiuAll_nonCDS_enhancerShort_peakPredictions_1MbFromConsistentExprGene.txt

# Itersect OCRs near genes with consistent expression with H3K27ac peaks from Villar et al.
cat /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/VillarLiver/Mmus-H3K27Ac_replicated-peaks_macs /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/VillarLiver/Mmus-H3K27Ac_replicated-peaks_macs.1 /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/VillarLiver/Mmus-H3K27Ac_replicated-peaks_macs.2 /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/VillarLiver/Mmus-H3K27Ac_replicated-peaks_macs.3 | sort -u -k1,1 -k2,2n -k3,3n | grep -v Chrom | sed 's/^/chr/' | intersectBed -wa -wb -a /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/LiverPfenning_ATAC_out/atac/68eb27d1-23c7-4d42-80f9-b3dd46f5e18c/call-reproducibility_idr/execution/idr.optimal_peak.narrowPeak_inLiuAll_nonCDS_enhancerShort_1MbFromConsistentExprGene.bed -b stdin > /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/LiverPfenning_ATAC_out/atac/68eb27d1-23c7-4d42-80f9-b3dd46f5e18c/call-reproducibility_idr/execution/idr.optimal_peak.narrowPeak_inLiuAll_nonCDS_enhancerShort_1MbFromConsistentExprGene_inVillarPeak.bed

# Compare predictions to mouse H3K27ac
sed '1d' /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/VillarLiver/Mmus_Enhancers_conservationInOthers | sed 's/^/chr/' | sed 's/\-/0/g' | sed 's/NA/\-1/g' | intersectBed -wa -wb -a /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/LiverPfenning_ATAC_out/atac/68eb27d1-23c7-4d42-80f9-b3dd46f5e18c/call-reproducibility_idr/execution/idr.optimal_peak.narrowPeak_inLiuAll_nonCDS_enhancerShort.narrowPeak.gz -b stdin | cut -f4,14-26 | sort -k1,1 > /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/VillarLiver/Mmus_Enhancers_conservationInOthers_inNonCDSEnhancerShort.txt
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/convertH3K27acMatToBinaryMat.py --H3K27acRegionFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/VillarLiver/Mmus_Enhancers_conservationInOthers_inNonCDSEnhancerShort.txt --outputFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/VillarLiver/Mmus_Enhancers_conservationInOthers_inNonCDSEnhancerShort_condensed.txt
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/filterPeakName.py --unfilteredPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/LiverPfenning_ATAC_out/atac/68eb27d1-23c7-4d42-80f9-b3dd46f5e18c/call-reproducibility_idr/execution/idr.optimal_peak.narrowPeak_inLiuAll_nonCDS_enhancerShort_peakPredictions.txt --unfilteredPeakNameCol 0 --peakListFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/VillarLiver/Mmus_Enhancers_conservationInOthers_inNonCDSEnhancerShort_condensed.txt --outputFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/LiverPfenning_ATAC_out/atac/68eb27d1-23c7-4d42-80f9-b3dd46f5e18c/call-reproducibility_idr/execution/idr.optimal_peak.narrowPeak_inLiuAll_nonCDS_enhancerShort_peakPredictions_inH3K27ac.txt
sort -k1,1 /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/LiverPfenning_ATAC_out/atac/68eb27d1-23c7-4d42-80f9-b3dd46f5e18c/call-reproducibility_idr/execution/idr.optimal_peak.narrowPeak_inLiuAll_nonCDS_enhancerShort_peakPredictions_inH3K27ac.txt > /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/LiverPfenning_ATAC_out/atac/68eb27d1-23c7-4d42-80f9-b3dd46f5e18c/call-reproducibility_idr/execution/idr.optimal_peak.narrowPeak_inLiuAll_nonCDS_enhancerShort_peakPredictions_inH3K27ac_sorted.txt

# Compare predictions to rat H3K27ac
sed '1d' /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/VillarLiver/Rnor_Enhancers_conservationInOthers | sed 's/^/chr/' | sed 's/\-/0/g' | sed 's/NA/\-1/g' > /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/VillarLiver/Rnor_Enhancers_conservationInOthers.bed
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/convertH3K27acMatToBinaryMat.py --H3K27acRegionFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/VillarLiver/Rnor_Enhancers_conservationInOthers.bed --headerCol 4 --outputFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/VillarLiver/Rnor_Enhancers_conservationInOthers_binaryMat.bed
awk 'BEGIN{OFS="\t"} {print $1, $2, $3, $4, int($5), int($6), int($7), int($8), int($9), int($10), int($11), int($12), int($13), int($14)}' /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/VillarLiver/Rnor_Enhancers_conservationInOthers_binaryMat.bed | /home/ikaplow/RegulatoryElementEvolutionProject/src/liftOver -bedPlus=4 stdin /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/LiftoverChains/rn5ToRn6.over.chain.gz /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/VillarLiver/Rnor_Enhancers_conservationInOthers_rn6.bed /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/VillarLiver/Rnor_Enhancers_conservationInOthers_unmapped.bed
intersectBed -wa -wb -a /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/RatAtac/Liver_GoodReps/peak/idr_reproducibility/idr.optimal_peak_nonCDS_enhancerShort.bed.gz -b /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/VillarLiver/Rnor_Enhancers_conservationInOthers_rn6.bed | cut -f4,14-24 | sort -k1,1 > /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/VillarLiver/Rnor_Enhancers_conservationInOthers_inNonCDSEnhancerShort.txt
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/convertH3K27acMatToBinaryMat.py --H3K27acRegionFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/VillarLiver/Rnor_Enhancers_conservationInOthers_inNonCDSEnhancerShort.txt --outputFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/VillarLiver/Rnor_Enhancers_conservationInOthers_inNonCDSEnhancerShort_condensed.txt
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/filterPeakName.py --unfilteredPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/RatAtac/Liver_GoodReps/peak/idr_reproducibility/LiftedPeaks/idr.optimal_peak_nonCDS_enhancerShort_peakPredictions.txt --unfilteredPeakNameCol 0 --peakListFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/VillarLiver/Rnor_Enhancers_conservationInOthers_inNonCDSEnhancerShort_condensed.txt --outputFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/RatAtac/Liver_GoodReps/peak/idr_reproducibility/LiftedPeaks/idr.optimal_peak_nonCDS_enhancerShort_peakPredictions_inH3K27ac.txt
sort -k1,1 /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/RatAtac/Liver_GoodReps/peak/idr_reproducibility/LiftedPeaks/idr.optimal_peak_nonCDS_enhancerShort_peakPredictions_inH3K27ac.txt > /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/RatAtac/Liver_GoodReps/peak/idr_reproducibility/LiftedPeaks/idr.optimal_peak_nonCDS_enhancerShort_peakPredictions_inH3K27ac_sorted.txt

# Compare predictions to macaque H3K27ac
sed '1d' /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/VillarLiver/Mmul_Enhancers_conservationInOthers | sed 's/^/chr/' | sed 's/\-/0/g' | sed 's/NA/\-1/g' > /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/VillarLiver/Mmul_Enhancers_conservationInOthers.bed
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/convertH3K27acMatToBinaryMat.py --H3K27acRegionFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/VillarLiver/Mmul_Enhancers_conservationInOthers.bed --headerCol 4 --outputFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/VillarLiver/Mmul_Enhancers_conservationInOthers_binaryMat.bed
awk 'BEGIN{OFS="\t"} {print $1, $2, $3, $4, int($5), int($6), int($7), int($8), int($9), int($10), int($11), int($12), int($13), int($14)}' /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/VillarLiver/Mmul_Enhancers_conservationInOthers_binaryMat.bed | /home/ikaplow/RegulatoryElementEvolutionProject/src/liftOver -bedPlus=4 stdin /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/LiftoverChains/rheMac2ToRheMac8.over.chain.gz /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/VillarLiver/Mmul_Enhancers_conservationInOthers_rheMac8.bed /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/VillarLiver/Mmul_Enhancers_conservationInOthers_unmapped.bed
intersectBed -wa -wb -a /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MacaqueAtac/Liver/cromwell-executions/atac/1f5fd9f0-d389-4184-99cb-c1125fd7f064/call-reproducibility_idr/execution/optimal_peak_nonCDS_enhancerShort.bed -b /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/VillarLiver/Mmul_Enhancers_conservationInOthers_rheMac8.bed | cut -f4,14-24 | sort -k1,1 > /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/VillarLiver/Mmul_Enhancers_conservationInOthers_inNonCDSEnhancerShort.txt
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/convertH3K27acMatToBinaryMat.py --H3K27acRegionFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/VillarLiver/Mmul_Enhancers_conservationInOthers_inNonCDSEnhancerShort.txt --outputFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/VillarLiver/Mmul_Enhancers_conservationInOthers_inNonCDSEnhancerShort_condensed.txt
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/filterPeakName.py --unfilteredPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MacaqueAtac/Liver/cromwell-executions/atac/1f5fd9f0-d389-4184-99cb-c1125fd7f064/call-reproducibility_idr/execution/LiftedPeaks/optimal_peak_nonCDS_enhancerShort_GenBankNames_peakPredictions.txt --unfilteredPeakNameCol 0 --peakListFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/VillarLiver/Mmul_Enhancers_conservationInOthers_inNonCDSEnhancerShort_condensed.txt --outputFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MacaqueAtac/Liver/cromwell-executions/atac/1f5fd9f0-d389-4184-99cb-c1125fd7f064/call-reproducibility_idr/execution/LiftedPeaks/optimal_peak_nonCDS_enhancerShort_GenBankNames_peakPredictions_inH3K27ac.txt
sort -k1,1 /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MacaqueAtac/Liver/cromwell-executions/atac/1f5fd9f0-d389-4184-99cb-c1125fd7f064/call-reproducibility_idr/execution/LiftedPeaks/optimal_peak_nonCDS_enhancerShort_GenBankNames_peakPredictions_inH3K27ac.txt > /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MacaqueAtac/Liver/cromwell-executions/atac/1f5fd9f0-d389-4184-99cb-c1125fd7f064/call-reproducibility_idr/execution/LiftedPeaks/optimal_peak_nonCDS_enhancerShort_GenBankNames_peakPredictions_inH3K27ac_sorted.txt

# Compare human orthologs to genes with constitutive expression
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/filterPeakName.py --unfilteredPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/LiverPfenning_ATAC_out/atac/68eb27d1-23c7-4d42-80f9-b3dd46f5e18c/call-reproducibility_idr/execution/idr.optimal_peak.narrowPeak_inLiuAll_nonCDS_enhancerShort_hg38_summitExtendedMin50Max2XProtect5.bed --peakListFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/LiverPfenning_ATAC_out/atac/68eb27d1-23c7-4d42-80f9-b3dd46f5e18c/call-reproducibility_idr/execution/idr.optimal_peak.narrowPeak_inLiuAll_nonCDS_enhancerShort_constitutivePeakNames.txt --outputFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/LiverPfenning_ATAC_out/atac/68eb27d1-23c7-4d42-80f9-b3dd46f5e18c/call-reproducibility_idr/execution/idr.optimal_peak.narrowPeak_inLiuAll_nonCDS_enhancerShort_hg38_summitExtendedMin50Max2XProtect5_constitutivePeaks.bed
sort -u -k1,1 -k2,2n -k3,3n -k10,10n /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/LiverPfenning_ATAC_out/atac/68eb27d1-23c7-4d42-80f9-b3dd46f5e18c/call-reproducibility_idr/execution/idr.optimal_peak.narrowPeak_inLiuAll_nonCDS_enhancerShort_hg38_summitExtendedMin50Max2XProtect5_constitutivePeaks.bed | grep chr | closestBed -g /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/HumanGenome/hg38.chrom.sizes.sorted -t all -d -a stdin -b /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/HumanGenome/gencode.v27.annotation.protTranscript.TSSsWithStrand_sorted.bed | awk 'BEGIN{OFS="\t"} $17 < 1000000' | cut -f14 > /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/LiverPfenning_ATAC_out/atac/68eb27d1-23c7-4d42-80f9-b3dd46f5e18c/call-reproducibility_idr/execution/idr.optimal_peak.narrowPeak_inLiuAll_nonCDS_enhancerShort_hg38_summitExtendedMin50Max2XProtect5_constitutivePeaks_closestGenes.txt
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/filterPeakName.py --unfilteredPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/RNASeq/Berthelot2018Data/expressionSummary_averageTPMnormMedian.txt --unfilteredPeakNameCol 0 --peakListFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/LiverPfenning_ATAC_out/atac/68eb27d1-23c7-4d42-80f9-b3dd46f5e18c/call-reproducibility_idr/execution/bioDBnet_db2db_210312032017_1413230699.txt --peakNameCol 1 --outputFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/RNASeq/Berthelot2018Data/expressionSummary_averageTPMnormMedian_nearConstitutiveEnhancer.txt
cut -f20 /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/RNASeq/Berthelot2018Data/expressionSummary_averageTPMnormMedian_nearConstitutiveEnhancer.txt | sed 's/NA/NaN/g' > /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/RNASeq/Berthelot2018Data/expressionSummary_averageTPMnormMedian_nearConstitutiveEnhancer_cvexp.txt

# Compare differential expression for genes near rodent-specific enhancers and rodent-specific non-enhancers
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/filterPeakName.py --unfilteredPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/RNASeq/Berthelot2018Data/RodentDiffGenesAllResults.txt --unfilteredPeakNameCol 0 --peakListFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/LiverPfenning_ATAC_out/atac/68eb27d1-23c7-4d42-80f9-b3dd46f5e18c/call-reproducibility_idr/execution/mart_export_genesNearRodentSpecificEnh.txt --peakNameCol 1 --outputFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/RNASeq/Berthelot2018Data/RodentDiffGenesAllResults_nearRodentSpecificEnh.txt
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/filterPeakName.py --unfilteredPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/RNASeq/Berthelot2018Data/RodentDiffGenesAllResults.txt --unfilteredPeakNameCol 0 --peakListFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MacaqueAtac/Liver/cromwell-executions/atac/1f5fd9f0-d389-4184-99cb-c1125fd7f064/call-reproducibility_idr/execution/mart_export_genesNearRodentSpecificNonEnh.txt --peakNameCol 1 --outputFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/RNASeq/Berthelot2018Data/RodentDiffGenesAllResults_nearRodentSpecificNonEnh.txt
