# Get flanking regions
awk 'BEGIN{OFS="\t"} {print $1, $2-500, $3+500, $4, 0, ".", -1, -1, -1, 250}' /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_withPeakNames.bed | flankBed -i stdin -g /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseGenome/mm10.chrom.sizes -b 500 | subtractBed -A -a stdin -b /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/pooled_rep/Cortex_ATAC_AgeB_Rep1_techRepsPooled.nodup.tn5_pooled.pf.pval0.01.300K.filt_orAgeB_orStriatum.narrowPeak.gz | gzip > /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_withPeakNames_flank500bpSep500bp.bed.gz

# Get fasta file of test set
grep -P 'chr1\t|chr2\t' /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_summitPlusMinus250bp.bed | fastaFromBed -fi /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseGenome/mm10.fa -bed stdin -fo /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_summitPlusMinus250bp_test.fa

# Shuffle fasta file of test set
fasta-shuffle-letters -kmer 2 -copies 10 -dna /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_summitPlusMinus250bp_test.fa /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_summitPlusMinus250bp_test_shufDiNuc10x.fa

# Get human brain enhancer orthologs that do not overlap mouse enhancers and that are in the mouse test set
grep -P 'chr1\t|chr2\t' /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/HumanDNase/PMC/cromwell-executions/atac/45c4bd8c-c055-4cea-80c9-5ec0eeaf8024/call-reproducibility_idr/execution/optimal_peak.inStriatum_nonCDS_enhancerShort_mm10_summitExtendedMin50Max2XProtect5.bed > /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/HumanDNase/PMC/cromwell-executions/atac/45c4bd8c-c055-4cea-80c9-5ec0eeaf8024/call-reproducibility_idr/execution/optimal_peak.inStriatum_nonCDS_enhancerShort_mm10_summitExtendedMin50Max2XProtect5_mouseTest.bed
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/filterPeakName.py --unfilteredPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/HumanDNase/PMC/cromwell-executions/atac/45c4bd8c-c055-4cea-80c9-5ec0eeaf8024/call-reproducibility_idr/execution/optimal_peak.inStriatum_nonCDS_enhancerShort.bed --peakListFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/HumanDNase/PMC/cromwell-executions/atac/45c4bd8c-c055-4cea-80c9-5ec0eeaf8024/call-reproducibility_idr/execution/optimal_peak.inStriatum_nonCDS_enhancerShort_mm10_summitExtendedMin50Max2XProtect5_mouseTest.bed --peakNameCol 3 --outputFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/HumanDNase/PMC/cromwell-executions/atac/45c4bd8c-c055-4cea-80c9-5ec0eeaf8024/call-reproducibility_idr/execution/optimal_peak.inStriatum_nonCDS_enhancerShort_mouseTest.bed
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/filterPeakName.py --unfilteredPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/HumanDNase/PMC/cromwell-executions/atac/45c4bd8c-c055-4cea-80c9-5ec0eeaf8024/call-reproducibility_idr/execution/optimal_peak.inStriatum_nonCDS_enhancerShort_mouseTest.bed --peakListFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/HumanDNase/PMC/cromwell-executions/atac/45c4bd8c-c055-4cea-80c9-5ec0eeaf8024/call-reproducibility_idr/execution/optimal_peak.inStriatum_nonCDS_enhancerShort_mm10_summitExtendedMin50Max2XProtect5_nonMouseBrain.bed --peakNameCol 3 --outputFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/HumanDNase/PMC/cromwell-executions/atac/45c4bd8c-c055-4cea-80c9-5ec0eeaf8024/call-reproducibility_idr/execution/optimal_peak.inStriatum_nonCDS_enhancerShort_nonMouseBrain_mouseTest.bed
gzip /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/HumanDNase/PMC/cromwell-executions/atac/45c4bd8c-c055-4cea-80c9-5ec0eeaf8024/call-reproducibility_idr/execution/optimal_peak.inStriatum_nonCDS_enhancerShort_nonMouseBrain_mouseTest.bed

# Get human non-enhancers whose mouse orthologs are brain enhancers and are in the mouse test set
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/filterPeakName.py --unfilteredPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_withPeakNames_hg38Huge_summitExtendedMin50Max2Protect5.bed --unfilteredPeakNameCol 3 --peakListFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_withPeakNames_test.bed --peakNameCol 3 --outputFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_withPeakNames_hg38Huge_summitExtendedMin50Max2Protect5_mouseTest.bed
subtractBed -A -a /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_withPeakNames_hg38Huge_summitExtendedMin50Max2Protect5_mouseTest.bed -b /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/HumanDNase/PMC/cromwell-executions/atac/45c4bd8c-c055-4cea-80c9-5ec0eeaf8024/call-macs2_pooled/execution/SRR5367725_1.merged.nodup.tn5.pooled.pval0.01.300K.bfilt.narrowPeak.gz | subtractBed -A -a stdin -b /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/HumanDNase/CaudatePutamen_DNase_out/out/peak/macs2/pooled_rep/Caudate_DNase.nodup_Putamen_DNase.nodup.pf.pval0.01.300K.filt.narrowPeak.gz | subtractBed -A -a stdin -b /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/HumanDNase/PUT/cromwell-executions/atac/cee5204e-3e9a-4437-868e-9fc8f05312af/call-macs2_pooled/execution/SRR5367726_1.merged.nodup.tn5.pooled.pval0.01.300K.bfilt.narrowPeak.gz | subtractBed -A -a stdin -b /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/HumanDNase/NAC/cromwell-executions/atac/f47a76c2-4f55-458f-a1ac-08713d0f0608/call-macs2/shard-0/execution/SRR5367749_1.merged.nodup.tn5.pval0.01.300K.bfilt.narrowPeak.gz | gzip > /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_withPeakNames_hg38Huge_summitExtendedMin50Max2Protect5_nonHumanBrain_mouseTest.narrowPeak.gz

# Get macaque brain enhancer orthologs that do not overlap mouse enhancers and that are in the mouse test set
grep -P 'chr1\t|chr2\t' /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MacaqueAtac/OfM/peak/idr_reproducibility/idr.optimal_peak.inM1AndStriatum_nonCDS_enhancerShort_mm10_summitExtendedMin50Max2XProtect5.bed > /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MacaqueAtac/OfM/peak/idr_reproducibility/idr.optimal_peak.inM1AndStriatum_nonCDS_enhancerShort_mm10_summitExtendedMin50Max2XProtect5_mouseTest.bed
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/filterPeakName.py --unfilteredPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MacaqueAtac/OfM/peak/idr_reproducibility/idr.optimal_peak.inM1AndStriatum_nonCDS_enhancerShort.bed --peakListFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MacaqueAtac/OfM/peak/idr_reproducibility/idr.optimal_peak.inM1AndStriatum_nonCDS_enhancerShort_mm10_summitExtendedMin50Max2XProtect5_mouseTest.bed --peakNameCol 3 --outputFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MacaqueAtac/OfM/peak/idr_reproducibility/idr.optimal_peak.inM1AndStriatum_nonCDS_enhancerShort_mouseTest.bed
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/filterPeakName.py --unfilteredPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MacaqueAtac/OfM/peak/idr_reproducibility/idr.optimal_peak.inM1AndStriatum_nonCDS_enhancerShort_mouseTest.bed --peakListFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MacaqueAtac/OfM/peak/idr_reproducibility/idr.optimal_peak.inM1AndStriatum_nonCDS_enhancerShort_mm10_summitExtendedMin50Max2XProtect5_nonMouseBrain.bed.gz --peakNameCol 3 --outputFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MacaqueAtac/OfM/peak/idr_reproducibility/idr.optimal_peak.inM1AndStriatum_nonCDS_enhancerShort_nonMouseBrain_mouseTest.bed
gzip /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MacaqueAtac/OfM/peak/idr_reproducibility/idr.optimal_peak.inM1AndStriatum_nonCDS_enhancerShort_nonMouseBrain_mouseTest.bed

# Get macaque non-enhancers whose mouse orthologs are brain enhancers and are in the mouse test set
grep -P 'chr1\t|chr2\t' /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_withPeakNames.bed > /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_withPeakNames_test.bed
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/filterPeakName.py --unfilteredPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_withPeakNames_rheMac8Huge_summitExtendedMin50Max2Protect5_nonMacaqueBrain.bed --peakListFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_withPeakNames_test.bed --peakNameCol 3 --outputFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_withPeakNames_rheMac8Huge_summitExtendedMin50Max2Protect5_nonMacaqueBrain_mouseTest.bed
gzip /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_withPeakNames_rheMac8Huge_summitExtendedMin50Max2Protect5_nonMacaqueBrain_mouseTest.bed

# Get rat brain enhancer orthologs that do not overlap mouse enhancers and that are in the mouse test set
grep -P 'chr1\t|chr2\t' /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/RatAtac/M1_AllReps/peak/idr_reproducibility/idr.optimal_peak.inStriatum_nonCDS_enhancerShort_mm10_summitExtendedMin50Max2XProtect5.bed > /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/RatAtac/M1_AllReps/peak/idr_reproducibility/idr.optimal_peak.inStriatum_nonCDS_enhancerShort_mm10_summitExtendedMin50Max2XProtect5_test.bed
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/filterPeakName.py --unfilteredPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/RatAtac/M1_AllReps/peak/idr_reproducibility/idr.optimal_peak.inStriatum_nonCDS_enhancerShort.bed --peakListFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/RatAtac/M1_AllReps/peak/idr_reproducibility/idr.optimal_peak.inStriatum_nonCDS_enhancerShort_mm10_summitExtendedMin50Max2XProtect5_test.bed --peakNameCol 3 --outputFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/RatAtac/M1_AllReps/peak/idr_reproducibility/idr.optimal_peak.inStriatum_nonCDS_enhancerShort_mouseTest.bed
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/filterPeakName.py --unfilteredPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/RatAtac/M1_AllReps/peak/idr_reproducibility/idr.optimal_peak.inStriatum_nonCDS_enhancerShort_mouseTest.bed --peakListFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/RatAtac/M1_AllReps/peak/idr_reproducibility/idr.optimal_peak.inStriatum_nonCDS_enhancerShort_mm10_summitExtendedMin50Max2XProtect5_nonMouseBrain.bed --peakNameCol 3 --outputFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/RatAtac/M1_AllReps/peak/idr_reproducibility/idr.optimal_peak.inStriatum_nonCDS_enhancerShort_mouseTest_nonMouseBrain.bed
gzip /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/RatAtac/M1_AllReps/peak/idr_reproducibility/idr.optimal_peak.inStriatum_nonCDS_enhancerShort_mouseTest_nonMouseBrain.bed

# Get rat non-enhancers whose mouse orthologs are brain enhancers and are in the mouse test set
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/filterPeakName.py --unfilteredPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_withPeakNames_rn6Huge_summitExtendedMin50Max2Protect5_nonRatBrain.bed --peakListFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_withPeakNames_test.bed --peakNameCol 3 --outputFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_withPeakNames_rn6Huge_summitExtendedMin50Max2Protect5_nonRatBrain_mouseTest.bed
gzip /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_withPeakNames_rn6Huge_summitExtendedMin50Max2Protect5_nonRatBrain_mouseTest.bed

# Get non-restrictive euarchonta-specific brain enhancers whose mouse orthologs are in the mouse test set
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/filterPeakName.py --unfilteredPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/HumanDNase/PMC/cromwell-executions/atac/45c4bd8c-c055-4cea-80c9-5ec0eeaf8024/call-reproducibility_idr/execution/optimal_peak.inStriatum_nonCDS_enhancerShort_inMacaqueBrainLoose_nonMouseBrain_nonRatBrain.bed --peakListFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/HumanDNase/PMC/cromwell-executions/atac/45c4bd8c-c055-4cea-80c9-5ec0eeaf8024/call-reproducibility_idr/execution/optimal_peak.inStriatum_nonCDS_enhancerShort_mouseTest.bed --peakNameCol 3 --outputFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/HumanDNase/PMC/cromwell-executions/atac/45c4bd8c-c055-4cea-80c9-5ec0eeaf8024/call-reproducibility_idr/execution/optimal_peak.inStriatum_nonCDS_enhancerShort_inMacaqueBrainLoose_nonMouseBrain_nonRatBrain_mouseTest.bed
gzip /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/HumanDNase/PMC/cromwell-executions/atac/45c4bd8c-c055-4cea-80c9-5ec0eeaf8024/call-reproducibility_idr/execution/optimal_peak.inStriatum_nonCDS_enhancerShort_inMacaqueBrainLoose_nonMouseBrain_nonRatBrain_mouseTest.bed

# Get non-restrictive euarchonta-specific brain non-enhancers whose mouse orthologs are in the mouse test set
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/filterPeakName.py --unfilteredPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_withPeakNames_hg38Huge_summitExtendedMin50Max2Protect5_mouseTest.bed --unfilteredPeakNameCol 3 --peakListFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_inRatBrainLoose_nonHumanBrain_nonMacaqueBrain.bed.gz --peakNameCol 3 --outputFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_withPeakNames_hg38Huge_summitExtendedMin50Max2Protect5_mouseTest_inRatBrainLoose_nonHumanBrain_nonMacaqueBrain.bed
gzip /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_withPeakNames_hg38Huge_summitExtendedMin50Max2Protect5_mouseTest_inRatBrainLoose_nonHumanBrain_nonMacaqueBrain.bed

# Get non-restrictive human-specific enhancers whose mouse orthologs are in the mouse test set
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/filterPeakName.py --unfilteredPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/HumanDNase/PMC/cromwell-executions/atac/45c4bd8c-c055-4cea-80c9-5ec0eeaf8024/call-reproducibility_idr/execution/optimal_peak.inStriatum_nonCDS_enhancerShort_mouseTest.bed --peakListFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/HumanDNase/PMC/cromwell-executions/atac/45c4bd8c-c055-4cea-80c9-5ec0eeaf8024/call-reproducibility_idr/execution/optimal_peak.inStriatum_nonCDS_enhancerShort_rheMac8_summitExtendedMin50Max2XProtect5_UCSCNames_nonMacaqueBrain.bed --peakNameCol 3 --outputFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/HumanDNase/PMC/cromwell-executions/atac/45c4bd8c-c055-4cea-80c9-5ec0eeaf8024/call-reproducibility_idr/execution/optimal_peak.inStriatum_nonCDS_enhancerShort_nonMacaqueBrain_mouseTest.bed
gzip /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/HumanDNase/PMC/cromwell-executions/atac/45c4bd8c-c055-4cea-80c9-5ec0eeaf8024/call-reproducibility_idr/execution/optimal_peak.inStriatum_nonCDS_enhancerShort_nonMacaqueBrain_mouseTest.bed

# Get non-restrictive human-specific non-enhancers whose mouse orthologs are in the mouse test set
grep -P 'chr1\t|chr2\t' /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MacaqueAtac/OfM/peak/idr_reproducibility/idr.optimal_peak.inM1AndStriatum_nonCDS_enhancerShort_mm10_summitExtendedMin50Max2XProtect5.bed > /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MacaqueAtac/OfM/peak/idr_reproducibility/idr.optimal_peak.inM1AndStriatum_nonCDS_enhancerShort_mm10_summitExtendedMin50Max2XProtect5_test.bed
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/filterPeakName.py --unfilteredPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MacaqueAtac/OfM/peak/idr_reproducibility/idr.optimal_peak.inM1AndStriatum_nonCDS_enhancerShort_GenBankNames_hg38_summitExtendedMin50Max2XProtect5_nonHumanBrain.bed --peakListFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MacaqueAtac/OfM/peak/idr_reproducibility/idr.optimal_peak.inM1AndStriatum_nonCDS_enhancerShort_mm10_summitExtendedMin50Max2XProtect5_test.bed --peakNameCol 3 --outputFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MacaqueAtac/OfM/peak/idr_reproducibility/idr.optimal_peak.inM1AndStriatum_nonCDS_enhancerShort_hg38_summitExtendedMin50Max2XProtect5_nonHumanBrain_mouseTest.bed
gzip /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MacaqueAtac/OfM/peak/idr_reproducibility/idr.optimal_peak.inM1AndStriatum_nonCDS_enhancerShort_hg38_summitExtendedMin50Max2XProtect5_nonHumanBrain_mouseTest.bed

# Get non-restrictive macaque-specific enhancers whose mouse orthologs are in the mouse test set
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/filterPeakName.py --unfilteredPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MacaqueAtac/OfM/peak/idr_reproducibility/idr.optimal_peak.inM1AndStriatum_nonCDS_enhancerShort_mouseTest.bed --peakListFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MacaqueAtac/OfM/peak/idr_reproducibility/idr.optimal_peak.inM1AndStriatum_nonCDS_enhancerShort_hg38_summitExtendedMin50Max2XProtect5_nonHumanBrain_mouseTest.bed.gz --peakNameCol 3 --outputFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MacaqueAtac/OfM/peak/idr_reproducibility/idr.optimal_peak.inM1AndStriatum_nonCDS_enhancerShort_nonHumanBrain_mouseTest.bed
gzip /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MacaqueAtac/OfM/peak/idr_reproducibility/idr.optimal_peak.inM1AndStriatum_nonCDS_enhancerShort_nonHumanBrain_mouseTest.bed

# Get non-restrictive macaque-specific non-enhancers whose mouse orthologs are in the mouse test set
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/filterPeakName.py --unfilteredPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/HumanDNase/PMC/cromwell-executions/atac/45c4bd8c-c055-4cea-80c9-5ec0eeaf8024/call-reproducibility_idr/execution/optimal_peak.inStriatum_nonCDS_enhancerShort_rheMac8_summitExtendedMin50Max2XProtect5_UCSCNames_nonMacaqueBrain.bed --peakListFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/HumanDNase/PMC/cromwell-executions/atac/45c4bd8c-c055-4cea-80c9-5ec0eeaf8024/call-reproducibility_idr/execution/optimal_peak.inStriatum_nonCDS_enhancerShort_mouseTest.bed --peakNameCol 3 --outputFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/HumanDNase/PMC/cromwell-executions/atac/45c4bd8c-c055-4cea-80c9-5ec0eeaf8024/call-reproducibility_idr/execution/optimal_peak.inStriatum_nonCDS_enhancerShort_rheMac8_summitExtendedMin50Max2XProtect5_UCSCNames_nonMacaqueBrain_mouseTest.bed
gzip /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/HumanDNase/PMC/cromwell-executions/atac/45c4bd8c-c055-4cea-80c9-5ec0eeaf8024/call-reproducibility_idr/execution/optimal_peak.inStriatum_nonCDS_enhancerShort_rheMac8_summitExtendedMin50Max2XProtect5_UCSCNames_nonMacaqueBrain_mouseTest.bed

# Get macaque shared brain and liver enhancers whose mouse orthologs are in the mouse test set
grep -P 'chr1\t|chr2\t' /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MacaqueAtac/Liver/cromwell-executions/atac/1f5fd9f0-d389-4184-99cb-c1125fd7f064/call-reproducibility_idr/execution/optimal_peak_withPeakNames_GenBankNames_mm10Huge_summitExtendedMin50Max2Protect5.bed > /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MacaqueAtac/Liver/cromwell-executions/atac/1f5fd9f0-d389-4184-99cb-c1125fd7f064/call-reproducibility_idr/execution/optimal_peak_withPeakNames_GenBankNames_mm10Huge_summitExtendedMin50Max2Protect5_test.bed
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/filterPeakName.py --unfilteredPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MacaqueAtac/Liver/cromwell-executions/atac/1f5fd9f0-d389-4184-99cb-c1125fd7f064/call-reproducibility_idr/execution/optimal_peak_withPeakNames.bed --peakListFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MacaqueAtac/Liver/cromwell-executions/atac/1f5fd9f0-d389-4184-99cb-c1125fd7f064/call-reproducibility_idr/execution/optimal_peak_withPeakNames_GenBankNames_mm10Huge_summitExtendedMin50Max2Protect5_test.bed --peakNameCol 4 --outputFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MacaqueAtac/Liver/cromwell-executions/atac/1f5fd9f0-d389-4184-99cb-c1125fd7f064/call-reproducibility_idr/execution/optimal_peak_withPeakNames_mouseTest.bed
subtractBed -A -a /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MacaqueAtac/Liver/cromwell-executions/atac/1f5fd9f0-d389-4184-99cb-c1125fd7f064/call-reproducibility_idr/execution/optimal_peak_withPeakNames_mouseTest.bed -b /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MacaqueGenome/GCF_000772875.2_Mmul_8.0.1_genomic.protCDS_geneNames_UCSCNames.bed | sort -u -k1,1 -k2,2n -k3,3n -k10,10n | closestBed -d -t first -a stdin -b /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MacaqueGenome/GCF_000772875.2_Mmul_8.0.1_genomic_andhg38.transcript.geneNames_TSSWithStrand_sorted_UCSCNames.bed | awk 'BEGIN{OFS="\t"} $17 > 20000' | cut -f1-10 > /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MacaqueAtac/Liver/cromwell-executions/atac/1f5fd9f0-d389-4184-99cb-c1125fd7f064/call-reproducibility_idr/execution/optimal_peak_nonCDS_enhancerShort_mouseTest.bed
intersectBed -wa -u -a /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MacaqueAtac/OfM/peak/idr_reproducibility/idr.optimal_peak.inM1AndStriatum_nonCDS_enhancerShort_mouseTest.bed -b /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MacaqueAtac/Liver/cromwell-executions/atac/1f5fd9f0-d389-4184-99cb-c1125fd7f064/call-reproducibility_idr/execution/optimal_peak_nonCDS_enhancerShort_mouseTest.bed | gzip > /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MacaqueAtac/OfM/peak/idr_reproducibility/idr.optimal_peak.inM1AndStriatum_nonCDS_enhancerShort_andLiver_mouseTest.bed.gz

# Get macaque liver enhancers that are not brain enhancers whose mouse orthologs are in the mouse test set
subtractBed -A -a /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MacaqueAtac/Liver/cromwell-executions/atac/1f5fd9f0-d389-4184-99cb-c1125fd7f064/call-reproducibility_idr/execution/optimal_peak_withPeakNames_mouseTest.bed -b /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MacaqueGenome/GCF_000772875.2_Mmul_8.0.1_genomic.protCDS_geneNames_UCSCNames.bed | sort -u -k1,1 -k2,2n -k3,3n -k10,10n | closestBed -d -t first -a stdin -b /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MacaqueGenome/GCF_000772875.2_Mmul_8.0.1_genomic_andhg38.transcript.geneNames_TSSWithStrand_sorted_UCSCNames.bed | awk 'BEGIN{OFS="\t"} $17 > 20000' | cut -f1-10 | awk 'BEGIN{OFS="\t"} $3-$2<=1000' | subtractBed -A -a stdin -b /projects/MPRA/Irene/macaque/atac-pipeline-output/OfM/peak/pooled-rep/basename_prefix.pooled.pval0.01.300K.bfilt.narrowPeak.gz | subtractBed -A -a stdin -b /projects/MPRA/Irene/macaque/atac-pipeline-output/M1/peak/pooled-rep/basename_prefix.pooled.pval0.01.300K.bfilt.narrowPeak.gz | subtractBed -A -a stdin -b /projects/MPRA/Irene/macaque/atac-pipeline-output/StrC/peak/pooled-rep/basename_prefix.pooled.pval0.01.300K.bfilt.narrowPeak.gz | subtractBed -A -a stdin -b /projects/MPRA/Irene/macaque/atac-pipeline-output/StrP/peak/pooled-rep/basename_prefix.pooled.pval0.01.300K.bfilt.narrowPeak.gz | subtractBed -A -a stdin -b /projects/MPRA/macaque/atac-pipeline-output/Fiona/NAcc/call-macs2_pooled/execution/Mac1_NAcc_1_PFE116A26_S11_R1_001.merged.nodup.tn5.pooled.pval0.01.300K.bfilt.narrowPeak.gz | gzip > /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MacaqueAtac/Liver/cromwell-executions/atac/1f5fd9f0-d389-4184-99cb-c1125fd7f064/call-reproducibility_idr/execution/optimal_peak_nonCDS_enhancerShort_mouseTest_nonMacaqueBrain.bed.gz

# Get rat shared brain and liver enhancers whose mouse orthologs are in the mouse test set
intersectBed -wa -u -a /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/RatAtac/M1_AllReps/peak/idr_reproducibility/idr.optimal_peak.inStriatum_nonCDS_enhancerShort_mouseTest.bed -b /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/RatAtac/Liver_GoodReps/peak/idr_reproducibility/idr.optimal_peak_nonCDS_enhancerShort.bed.gz | gzip > /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/RatAtac/M1_AllReps/peak/idr_reproducibility/idr.optimal_peak.inStriatum_nonCDS_enhancerShort_andLiver_mouseTest.bed.gz

# Get rat liver enhancers that are not brain enhancers whose mouse orthologs are in the mouse test set
grep -P 'chr1\t|chr2\t' /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/RatAtac/Liver_GoodReps/peak/idr_reproducibility/idr.optimal_peak_nonCDS_enhancerShort_nonRatBrain_mm10_summitExtendedMin50Max2XProtect5.bed > /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/RatAtac/Liver_GoodReps/peak/idr_reproducibility/idr.optimal_peak_nonCDS_enhancerShort_nonRatBrain_mm10_summitExtendedMin50Max2XProtect5_mouseTest.bed
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/filterPeakName.py --unfilteredPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/RatAtac/Liver_GoodReps/peak/idr_reproducibility/idr.optimal_peak_nonCDS_enhancerShort_nonRatBrain.bed --peakListFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/RatAtac/Liver_GoodReps/peak/idr_reproducibility/idr.optimal_peak_nonCDS_enhancerShort_nonRatBrain_mm10_summitExtendedMin50Max2XProtect5_mouseTest.bed --peakNameCol 3 --outputFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/RatAtac/Liver_GoodReps/peak/idr_reproducibility/idr.optimal_peak_nonCDS_enhancerShort_nonRatBrain_mouseTest.bed
gzip /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/RatAtac/Liver_GoodReps/peak/idr_reproducibility/idr.optimal_peak_nonCDS_enhancerShort_nonRatBrain_mouseTest.bed

# Get mouse brain test set orthologs across Zoonomia species
zcat /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_orLiver_withPeakNames_brainOnly.bed.gz | grep -P 'chr1\t|chr2\t' > /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_orLiver_withPeakNames_brainOnly_test.bed
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/makeFilterPeakNameScript.py --unfilteredPeakFileNameListFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_orLiver_LiftedPeaks/liftedPeakMouseBrainNarrowPeakFileNamesFiltConverted.txt --peakListFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_orLiver_withPeakNames_brainOnly_test.bed --peakNameCol 3 --outputFileNameSuffix mouseTest.bed --numFileNameElementsToRemove 3 --scriptFileName /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/filterPeakNameMouseBrainTestHugeCactusScript.sh --codePath /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode
sh /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/filterPeakNameMouseBrainTestHugeCactusScript.sh
gzip /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_orLiver_LiftedPeaks/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_orLiver_withPeakNames_*_summitExtendedMin50Max1000Protect5*.narrowPeak.mouseTest.bed
gzip /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_orLiver_withPeakNames_brainOnly_test.bed

# Get macaque liver enhancer orthologs that do not overlap mouse enhancers and that are in the mouse test set
grep -P 'chr1\t|chr2\t' /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MacaqueAtac/Liver/cromwell-executions/atac/1f5fd9f0-d389-4184-99cb-c1125fd7f064/call-reproducibility_idr/execution/optimal_peak_nonCDS_enhancerShort_mm10Huge_summitExtendedMin50Max2Protect5_nonMouseLiver.bed > /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MacaqueAtac/Liver/cromwell-executions/atac/1f5fd9f0-d389-4184-99cb-c1125fd7f064/call-reproducibility_idr/execution/optimal_peak_nonCDS_enhancerShort_mm10Huge_summitExtendedMin50Max2Protect5_nonMouseLiver_mouseTest.bed
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/filterPeakName.py --unfilteredPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MacaqueAtac/Liver/cromwell-executions/atac/1f5fd9f0-d389-4184-99cb-c1125fd7f064/call-reproducibility_idr/execution/optimal_peak_nonCDS_enhancerShort.bed --peakListFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MacaqueAtac/Liver/cromwell-executions/atac/1f5fd9f0-d389-4184-99cb-c1125fd7f064/call-reproducibility_idr/execution/optimal_peak_nonCDS_enhancerShort_mm10Huge_summitExtendedMin50Max2Protect5_nonMouseLiver_mouseTest.bed --peakNameCol 3 --outputFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MacaqueAtac/Liver/cromwell-executions/atac/1f5fd9f0-d389-4184-99cb-c1125fd7f064/call-reproducibility_idr/execution/optimal_peak_nonCDS_enhancerShort_nonMouseLiver_mouseTest.bed
gzip /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MacaqueAtac/Liver/cromwell-executions/atac/1f5fd9f0-d389-4184-99cb-c1125fd7f064/call-reproducibility_idr/execution/optimal_peak_nonCDS_enhancerShort_nonMouseLiver_mouseTest.bed

# Get macaque non-enhancers whose mouse orthologs are liver enhancers and are in the mouse test set
zcat /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/LiverPfenning_ATAC_out/atac/68eb27d1-23c7-4d42-80f9-b3dd46f5e18c/call-reproducibility_idr/execution/idr.optimal_peak.narrowPeak_inLiuAll_nonCDS_enhancerShort.narrowPeak.gz | grep -P 'chr1\t|chr2\t' > /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/LiverPfenning_ATAC_out/atac/68eb27d1-23c7-4d42-80f9-b3dd46f5e18c/call-reproducibility_idr/execution/idr.optimal_peak.narrowPeak_inLiuAll_nonCDS_enhancerShort_test.bed
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/filterPeakName.py --unfilteredPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/LiverPfenning_ATAC_out/atac/68eb27d1-23c7-4d42-80f9-b3dd46f5e18c/call-reproducibility_idr/execution/idr.optimal_peak.narrowPeak_inLiuAll_nonCDS_enhancerShort_rheMac8_summitExtendedMin50Max2XProtect5_UCSCNames_nonMacaqueLiver.bed --peakListFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/LiverPfenning_ATAC_out/atac/68eb27d1-23c7-4d42-80f9-b3dd46f5e18c/call-reproducibility_idr/execution/idr.optimal_peak.narrowPeak_inLiuAll_nonCDS_enhancerShort_test.bed --peakNameCol 3 --outputFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/LiverPfenning_ATAC_out/atac/68eb27d1-23c7-4d42-80f9-b3dd46f5e18c/call-reproducibility_idr/execution/idr.optimal_peak.narrowPeak_inLiuAll_nonCDS_enhancerShort_rheMac8_summitExtendedMin50Max2XProtect5_UCSCNames_nonMacaqueLiver_mouseTest.bed
gzip /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/LiverPfenning_ATAC_out/atac/68eb27d1-23c7-4d42-80f9-b3dd46f5e18c/call-reproducibility_idr/execution/idr.optimal_peak.narrowPeak_inLiuAll_nonCDS_enhancerShort_rheMac8_summitExtendedMin50Max2XProtect5_UCSCNames_nonMacaqueLiver_mouseTest.bed

# Get rat liver enhancer orthologs that do not overlap mouse enhancers and that are in the mouse test set
zcat /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/RatAtac/Liver_GoodReps/peak/idr_reproducibility/idr.optimal_peak_nonCDS_enhancerShort_mm10_summitExtendedMin50Max2XProtect5_nonMouseLiver.bed.gz | grep -P 'chr1\t|chr2\t' > /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/RatAtac/Liver_GoodReps/peak/idr_reproducibility/idr.optimal_peak_nonCDS_enhancerShort_mm10_summitExtendedMin50Max2XProtect5_nonMouseLiver_mouseTest.bed
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/filterPeakName.py --unfilteredPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/RatAtac/Liver_GoodReps/peak/idr_reproducibility/idr.optimal_peak_nonCDS_enhancerShort.bed.gz --peakListFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/RatAtac/Liver_GoodReps/peak/idr_reproducibility/idr.optimal_peak_nonCDS_enhancerShort_mm10_summitExtendedMin50Max2XProtect5_nonMouseLiver_mouseTest.bed --peakNameCol 3 --outputFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/RatAtac/Liver_GoodReps/peak/idr_reproducibility/idr.optimal_peak_nonCDS_enhancerShort_nonMouseLiver_mouseTest.bed
gzip /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/RatAtac/Liver_GoodReps/peak/idr_reproducibility/idr.optimal_peak_nonCDS_enhancerShort_nonMouseLiver_mouseTest.bed

# Get rat non-enhancers whose mouse orthologs are liver enhancers and are in the mouse test set
zcat /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/LiverPfenning_ATAC_out/atac/68eb27d1-23c7-4d42-80f9-b3dd46f5e18c/call-reproducibility_idr/execution/idr.optimal_peak.narrowPeak_inLiuAll_nonCDS_enhancerShort.narrowPeak.gz | grep -P 'chr1\t|chr2\t' > /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/LiverPfenning_ATAC_out/atac/68eb27d1-23c7-4d42-80f9-b3dd46f5e18c/call-reproducibility_idr/execution/idr.optimal_peak.narrowPeak_inLiuAll_nonCDS_enhancerShort_test.bed
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/filterPeakName.py --unfilteredPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/LiverPfenning_ATAC_out/atac/68eb27d1-23c7-4d42-80f9-b3dd46f5e18c/call-reproducibility_idr/execution/idr.optimal_peak.narrowPeak_inLiuAll_nonCDS_enhancerShort_rn6_summitExtendedMin50Max2XProtect5_nonRatLiver.bed --peakListFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/LiverPfenning_ATAC_out/atac/68eb27d1-23c7-4d42-80f9-b3dd46f5e18c/call-reproducibility_idr/execution/idr.optimal_peak.narrowPeak_inLiuAll_nonCDS_enhancerShort_test.bed --peakNameCol 3 --outputFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/LiverPfenning_ATAC_out/atac/68eb27d1-23c7-4d42-80f9-b3dd46f5e18c/call-reproducibility_idr/execution/idr.optimal_peak.narrowPeak_inLiuAll_nonCDS_enhancerShort_rn6_summitExtendedMin50Max2XProtect5_nonRatLiver_mouseTest.bed
gzip /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/LiverPfenning_ATAC_out/atac/68eb27d1-23c7-4d42-80f9-b3dd46f5e18c/call-reproducibility_idr/execution/idr.optimal_peak.narrowPeak_inLiuAll_nonCDS_enhancerShort_rn6_summitExtendedMin50Max2XProtect5_nonRatLiver_mouseTest.bed

# Get non-restrictive glires-specific liver non-enhancers
python /home/ikaplow/RegulatoryElementEvolutionProject/src/halLiftover-postprocessing/orthologFind.py -max_frac 2 -min_len 50 -protect_dist 5 -qFile /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MacaqueAtac/Liver/cromwell-executions/atac/1f5fd9f0-d389-4184-99cb-c1125fd7f064/call-reproducibility_idr/execution/optimal_peak_nonCDS_enhancerShort.bed -tFile /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MacaqueAtac/Liver/cromwell-executions/atac/1f5fd9f0-d389-4184-99cb-c1125fd7f064/call-reproducibility_idr/execution/optimal_peak_nonCDS_enhancerShort_rn6.bed -sFile /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MacaqueAtac/Liver/cromwell-executions/atac/1f5fd9f0-d389-4184-99cb-c1125fd7f064/call-reproducibility_idr/execution/optimal_peak_nonCDS_enhancerShort_summits_rn6.bed -oFile /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MacaqueAtac/Liver/cromwell-executions/atac/1f5fd9f0-d389-4184-99cb-c1125fd7f064/call-reproducibility_idr/execution/optimal_peak_nonCDS_enhancerShort_rn6_summitExtendedMin50Max2XProtect5.bed -narrowPeak
subtractBed -A -a /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MacaqueAtac/Liver/cromwell-executions/atac/1f5fd9f0-d389-4184-99cb-c1125fd7f064/call-reproducibility_idr/execution/optimal_peak_nonCDS_enhancerShort_rn6_summitExtendedMin50Max2XProtect5.bed -b /projects/MPRA/Irene/rats/atac-pipeline-output/Liver/peak/pooled-rep/R2-Liv_R1_001.trim.merged.nodup.no_chrM.tn5.pooled.pval0.01.300K.bfilt.narrowPeak.gz > /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MacaqueAtac/Liver/cromwell-executions/atac/1f5fd9f0-d389-4184-99cb-c1125fd7f064/call-reproducibility_idr/execution/optimal_peak_nonCDS_enhancerShort_rn6_summitExtendedMin50Max2XProtect5_nonRatLiver.bed
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/filterPeakName.py --unfilteredPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MacaqueAtac/Liver/cromwell-executions/atac/1f5fd9f0-d389-4184-99cb-c1125fd7f064/call-reproducibility_idr/execution/optimal_peak_nonCDS_enhancerShort_mm10Huge_summitExtendedMin50Max2Protect5_nonMouseLiver.bed --peakListFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MacaqueAtac/Liver/cromwell-executions/atac/1f5fd9f0-d389-4184-99cb-c1125fd7f064/call-reproducibility_idr/execution/optimal_peak_nonCDS_enhancerShort_rn6_summitExtendedMin50Max2XProtect5_nonRatLiver.bed --peakNameCol 3 --outputFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MacaqueAtac/Liver/cromwell-executions/atac/1f5fd9f0-d389-4184-99cb-c1125fd7f064/call-reproducibility_idr/execution/optimal_peak_nonCDS_enhancerShort_mm10Huge_summitExtendedMin50Max2Protect5_nonMouseLiver_nonRatLiver.bed
gzip /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MacaqueAtac/Liver/cromwell-executions/atac/1f5fd9f0-d389-4184-99cb-c1125fd7f064/call-reproducibility_idr/execution/optimal_peak_nonCDS_enhancerShort_mm10Huge_summitExtendedMin50Max2Protect5_nonMouseLiver_nonRatLiver.bed

# Get non-restrictive euarchonta-specific liver enhancers whose mouse orthologs are in the mouse test set
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/filterPeakName.py --unfilteredPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MacaqueAtac/Liver/cromwell-executions/atac/1f5fd9f0-d389-4184-99cb-c1125fd7f064/call-reproducibility_idr/execution/optimal_peak_nonCDS_enhancerShort_mm10Huge_summitExtendedMin50Max2Protect5_nonMouseLiver_mouseTest.bed --peakListFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MacaqueAtac/Liver/cromwell-executions/atac/1f5fd9f0-d389-4184-99cb-c1125fd7f064/call-reproducibility_idr/execution/optimal_peak_nonCDS_enhancerShort_rn6_summitExtendedMin50Max2XProtect5_nonRatLiver.bed --peakNameCol 3 --outputFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MacaqueAtac/Liver/cromwell-executions/atac/1f5fd9f0-d389-4184-99cb-c1125fd7f064/call-reproducibility_idr/execution/optimal_peak_nonCDS_enhancerShort_mm10Huge_summitExtendedMin50Max2Protect5_nonMouseLiver_nonRatLiver_mouseTest.bed
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/filterPeakName.py --unfilteredPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MacaqueAtac/Liver/cromwell-executions/atac/1f5fd9f0-d389-4184-99cb-c1125fd7f064/call-reproducibility_idr/execution/optimal_peak_nonCDS_enhancerShort.bed --peakListFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MacaqueAtac/Liver/cromwell-executions/atac/1f5fd9f0-d389-4184-99cb-c1125fd7f064/call-reproducibility_idr/execution/optimal_peak_nonCDS_enhancerShort_mm10Huge_summitExtendedMin50Max2Protect5_nonMouseLiver_nonRatLiver_mouseTest.bed --peakNameCol 3 --outputFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MacaqueAtac/Liver/cromwell-executions/atac/1f5fd9f0-d389-4184-99cb-c1125fd7f064/call-reproducibility_idr/execution/optimal_peak_nonCDS_enhancerShort_nonMouseLiver_nonRatLiver_mouseTest.bed
gzip /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MacaqueAtac/Liver/cromwell-executions/atac/1f5fd9f0-d389-4184-99cb-c1125fd7f064/call-reproducibility_idr/execution/optimal_peak_nonCDS_enhancerShort_nonMouseLiver_nonRatLiver_mouseTest.bed

# Get non-restrictive euarchonta-specific liver non-enhancers whose mouse orthologs are in the mouse test set
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/filterPeakName.py --unfilteredPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/LiverPfenning_ATAC_out/atac/68eb27d1-23c7-4d42-80f9-b3dd46f5e18c/call-reproducibility_idr/execution/idr.optimal_peak.narrowPeak_inLiuAll_nonCDS_enhancerShort_rheMac8_summitExtendedMin50Max2XProtect5_UCSCNames_nonMacaqueLiver_mouseTest.bed.gz --peakListFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/LiverPfenning_ATAC_out/atac/68eb27d1-23c7-4d42-80f9-b3dd46f5e18c/call-reproducibility_idr/execution/idr.optimal_peak.narrowPeak_inLiuAll_nonCDS_enhancerShort_rn6_summitExtendedMin50Max2XProtect5_inRatLiverLoose.bed --peakNameCol 3 --outputFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/LiverPfenning_ATAC_out/atac/68eb27d1-23c7-4d42-80f9-b3dd46f5e18c/call-reproducibility_idr/execution/idr.optimal_peak.narrowPeak_inLiuAll_nonCDS_enhancerShort_rheMac8_summitExtendedMin50Max2XProtect5_UCSCNames_nonMacaqueLiver_inRatLiverLoose_mouseTest.bed
gzip /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/LiverPfenning_ATAC_out/atac/68eb27d1-23c7-4d42-80f9-b3dd46f5e18c/call-reproducibility_idr/execution/idr.optimal_peak.narrowPeak_inLiuAll_nonCDS_enhancerShort_rheMac8_summitExtendedMin50Max2XProtect5_UCSCNames_nonMacaqueLiver_inRatLiverLoose_mouseTest.bed

# Get macaque shared liver and brain enhancers whose mouse orthologs are in the mouse test set
intersectBed -wa -u -a /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MacaqueAtac/Liver/cromwell-executions/atac/1f5fd9f0-d389-4184-99cb-c1125fd7f064/call-reproducibility_idr/execution/optimal_peak_nonCDS_enhancerShort_mouseTest.bed -b /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MacaqueAtac/OfM/peak/idr_reproducibility/idr.optimal_peak.inM1AndStriatum_nonCDS_enhancerShort_mouseTest.bed | gzip > /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MacaqueAtac/Liver/cromwell-executions/atac/1f5fd9f0-d389-4184-99cb-c1125fd7f064/call-reproducibility_idr/execution/optimal_peak_nonCDS_enhancerShort_andBrain_mouseTest.bed.gz

# Get macaque brain enhancers that are not liver enhancers whose mouse orthologs are in the mouse test set
subtractBed -A -a /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MacaqueAtac/OfM/peak/idr_reproducibility/idr.optimal_peak.inM1AndStriatum_nonCDS_enhancerShort_mouseTest.bed -b /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MacaqueAtac/Liver/cromwell-executions/atac/1f5fd9f0-d389-4184-99cb-c1125fd7f064/call-macs2_pooled/execution/Mac1_Lv_1_PFE116A29_S8_R1_001.merged.nodup.tn5.pooled.pval0.01.300K.bfilt.narrowPeak.gz | gzip > /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MacaqueAtac/OfM/peak/idr_reproducibility/idr.optimal_peak.inM1AndStriatum_nonCDS_enhancerShort_nonLiver_mouseTest.bed.gz

# Get rat shared liver and brain enhancers whose mouse orthologs are in the mouse test set
grep -P 'chr1\t|chr2\t' /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/RatAtac/Liver_GoodReps/peak/idr_reproducibility/idr.optimal_peak_nonCDS_enhancerShort_mm10_summitExtendedMin50Max2XProtect5.bed > /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/RatAtac/Liver_GoodReps/peak/idr_reproducibility/idr.optimal_peak_nonCDS_enhancerShort_mm10_summitExtendedMin50Max2XProtect5_mouseTest.bed
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/filterPeakName.py --unfilteredPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/RatAtac/Liver_GoodReps/peak/idr_reproducibility/idr.optimal_peak_nonCDS_enhancerShort.bed.gz --peakListFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/RatAtac/Liver_GoodReps/peak/idr_reproducibility/idr.optimal_peak_nonCDS_enhancerShort_mm10_summitExtendedMin50Max2XProtect5_mouseTest.bed --peakNameCol 3 --outputFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/RatAtac/Liver_GoodReps/peak/idr_reproducibility/idr.optimal_peak_nonCDS_enhancerShort_mouseTest.bed
intersectBed -wa -u -a /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/RatAtac/Liver_GoodReps/peak/idr_reproducibility/idr.optimal_peak_nonCDS_enhancerShort_mouseTest.bed -b /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/RatAtac/M1_AllReps/peak/idr_reproducibility/idr.optimal_peak.inStriatum_nonCDS_enhancerShort.bed | gzip > /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/RatAtac/Liver_GoodReps/peak/idr_reproducibility/idr.optimal_peak_nonCDS_enhancerShort_inBrain_mouseTest.bed.gz

# Get rat brain enhancers that are not liver enhancers whose mouse orthologs are in the mouse test set
subtractBed -A -a /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/RatAtac/M1_AllReps/peak/idr_reproducibility/idr.optimal_peak.inStriatum_nonCDS_enhancerShort_mouseTest.bed -b /projects/MPRA/Irene/rats/atac-pipeline-output/Liver/peak/pooled-rep/R2-Liv_R1_001.trim.merged.nodup.no_chrM.tn5.pooled.pval0.01.300K.bfilt.narrowPeak.gz | gzip > /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/RatAtac/M1_AllReps/peak/idr_reproducibility/idr.optimal_peak.inStriatum_nonCDS_enhancerShort_nonBrain_mouseTest.bed.gz

# Get mouse liver test set orthologs across Zoonomia species
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/makeFilterPeakNameScript.py --unfilteredPeakFileNameListFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/LiverPfenning_ATAC_out/atac/68eb27d1-23c7-4d42-80f9-b3dd46f5e18c/call-reproducibility_idr/execution/LiftedPeaks/liftedPeakAllFileNamesConverted.txt --peakListFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/LiverPfenning_ATAC_out/atac/68eb27d1-23c7-4d42-80f9-b3dd46f5e18c/call-reproducibility_idr/execution/idr.optimal_peak.narrowPeak_inLiuAll_nonCDS_enhancerShort_test.bed --peakNameCol 3 --outputFileNameSuffix mouseTest.bed --numFileNameElementsToRemove 2 --scriptFileName /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/filterPeakNameMouseBrainTestHugeCactusScript.sh --codePath /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode
sh /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/filterPeakNameMouseBrainTestHugeCactusScript.sh
gzip /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/LiverPfenning_ATAC_out/atac/68eb27d1-23c7-4d42-80f9-b3dd46f5e18c/call-reproducibility_idr/execution/LiftedPeaks/idr.optimal_peak.narrowPeak_inLiuAll_nonCDS_enhancerShort_*_summitExtendedMin50Max2XProtect5*.mouseTest.bed
gzip /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/LiverPfenning_ATAC_out/atac/68eb27d1-23c7-4d42-80f9-b3dd46f5e18c/call-reproducibility_idr/execution/idr.optimal_peak.narrowPeak_inLiuAll_nonCDS_enhancerShort_test.bed

# Get non-restrictive laurastheria-specific liver enhancers whose mouse orthologs are in the mouse test set
grep -P 'chr1\t|chr2\t' /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/CowDNase/Liver/atac/888ae501-7469-4afa-8890-e36f14f6c83b/peak/idr_reproducibility/idr.optimal_peak_nonCDS_enhancerShort_GenBankNames_mm10Fixed_summitExtendedMin50Max2Protect5.bed > /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/CowDNase/Liver/atac/888ae501-7469-4afa-8890-e36f14f6c83b/peak/idr_reproducibility/idr.optimal_peak_nonCDS_enhancerShort_GenBankNames_mm10Fixed_summitExtendedMin50Max2Protect5_test.bed
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/filterPeakName.py --unfilteredPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/CowDNase/Liver/atac/888ae501-7469-4afa-8890-e36f14f6c83b/peak/idr_reproducibility/idr.optimal_peak_nonCDS_enhancerShort_inPigLiverLoose_nonMouseLiver_nonRatLiver_nonMacaqueLiver.bed --peakListFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/CowDNase/Liver/atac/888ae501-7469-4afa-8890-e36f14f6c83b/peak/idr_reproducibility/idr.optimal_peak_nonCDS_enhancerShort_GenBankNames_mm10Fixed_summitExtendedMin50Max2Protect5_test.bed --peakNameCol 3 --outputFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/CowDNase/Liver/atac/888ae501-7469-4afa-8890-e36f14f6c83b/peak/idr_reproducibility/idr.optimal_peak_nonCDS_enhancerShort_inPigLiverLoose_nonMouseLiver_nonRatLiver_nonMacaqueLiver_test.bed
gzip /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/CowDNase/Liver/atac/888ae501-7469-4afa-8890-e36f14f6c83b/peak/idr_reproducibility/idr.optimal_peak_nonCDS_enhancerShort_inPigLiverLoose_nonMouseLiver_nonRatLiver_nonMacaqueLiver_test.bed

# Get non-restrictive laurastheria-specific liver non-enhancers whose mouse orthologs are in the mouse test set
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/filterPeakName.py --unfilteredPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/LiverPfenning_ATAC_out/atac/68eb27d1-23c7-4d42-80f9-b3dd46f5e18c/call-reproducibility_idr/execution/idr.optimal_peak.inLiuAllNonLoose_nonCDS_enhancerShort_bosTau2015Fixed_summitExtendedMin50Max2XProtect5_RefSeqNames_nonCowLiver_inRatLiverLoose_inMacaqueLiverLoose_nonPigLiver.bed --peakListFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/LiverPfenning_ATAC_out/atac/68eb27d1-23c7-4d42-80f9-b3dd46f5e18c/call-reproducibility_idr/execution/idr.optimal_peak.inLiuAllLoose_nonCDS_enhancerShort_test.bed --peakNameCol 3 --outputFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/LiverPfenning_ATAC_out/atac/68eb27d1-23c7-4d42-80f9-b3dd46f5e18c/call-reproducibility_idr/execution/idr.optimal_peak.inLiuAllNonLoose_nonCDS_enhancerShort_bosTau2015Fixed_summitExtendedMin50Max2XProtect5_RefSeqNames_nonCowLiver_inRatLiverLoose_inMacaqueLiverLoose_nonPigLiver_test.bed
gzip /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/LiverPfenning_ATAC_out/atac/68eb27d1-23c7-4d42-80f9-b3dd46f5e18c/call-reproducibility_idr/execution/idr.optimal_peak.inLiuAllNonLoose_nonCDS_enhancerShort_bosTau2015Fixed_summitExtendedMin50Max2XProtect5_RefSeqNames_nonCowLiver_inRatLiverLoose_inMacaqueLiverLoose_nonPigLiver_test.bed

# Performance of brain model with flanking negatives on test set
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/predictNewSequences.py --architectureFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsFlankPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_flankNeg_500bp_conv5MuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal_architecture.json --weightsFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsFlankPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_flankNeg_500bp_conv5MuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal.hdf5 --narrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort.narrowPeak.gz --negNarrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_withPeakNames_flank500bpSep500bp.bed.gz --sequenceLength 500 --genomeFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseGenome/mm10.fa --chroms chr1 --chroms chr2 --predictedProbaFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_test_andNeg_flankNeg_conv5MuchMuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal_predictedProba.txt --RC --classification --path /home/ikaplow/RegulatoryElementEvolutionProject/src/deeplearning
# Performance: loss: nan, accuracy: 0.903129, sensitivity: 0.879808, specificity: 0.917473, auc: 0.962734, precision: 0.867672, NPV: 0.925434, auprc: 0.945182, auprg: 0.963104, F1: 0.873698, recallAt0.05FDR: 0.735358, recallAt0.1FDR: 0.845935
# Performance on mouse enhancers whose orthologs are not brain enhancers in at least one other species and negative set
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/predictNewSequences.py --architectureFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsFlankPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_flankNeg_500bp_conv5MuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal_architecture.json --weightsFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsFlankPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_flankNeg_500bp_conv5MuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal.hdf5 --narrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_withPeakNames_nonBrainOtherSpecies.narrowPeak.gz --negNarrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/HumanDNase/PMC/cromwell-executions/atac/45c4bd8c-c055-4cea-80c9-5ec0eeaf8024/call-reproducibility_idr/execution/optimal_peak.inStriatum_nonCDS_enhancerShort_mm10_summitExtendedMin50Max2XProtect5_nonMouseBrain_andMacaque_andRat.narrowPeak.gz --sequenceLength 500 --genomeFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseGenome/mm10.fa --chroms chr1 --chroms chr2 --RC --classification --path /home/ikaplow/RegulatoryElementEvolutionProject/src/deeplearning
# Performance: loss: nan, accuracy: 0.759361, sensitivity: 0.885350, specificity: 0.663434, auc: 0.862875, precision: 0.666987, NPV: 0.883721, auprc: 0.800016, auprg: 0.778374, F1: 0.760810, recallAt0.05FDR: 0.000000, recallAt0.1FDR: 0.149045
# Performance on human enhancers whose orthologs are not brain enhancers in mouse and negative set corresponding to non-enhancer orthologs of mouse enhancers
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/predictNewSequences.py --architectureFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsFlankPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_flankNeg_500bp_conv5MuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal_architecture.json --weightsFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsFlankPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_flankNeg_500bp_conv5MuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal.hdf5 --narrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/HumanDNase/PMC/cromwell-executions/atac/45c4bd8c-c055-4cea-80c9-5ec0eeaf8024/call-reproducibility_idr/execution/optimal_peak.inStriatum_nonCDS_enhancerShort_nonMouseBrain_mouseTest.bed.gz --negNarrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_withPeakNames_hg38Huge_summitExtendedMin50Max2Protect5_nonHumanBrain_mouseTest.narrowPeak.gz --sequenceLength 500 --genomeFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/HumanGenome/hg38.fa --RC --classification --path /home/ikaplow/RegulatoryElementEvolutionProject/src/deeplearning
# Peformance: loss: nan, accuracy: 0.650915, sensitivity: 0.769231, specificity: 0.595982, auc: 0.749168, precision: 0.469208, NPV: 0.847619, auprc: 0.578826, auprg: 0.628489, F1: 0.582878, recallAt0.05FDR: 0.000000, recallAt0.1FDR: 0.000000
# Performance on macaque enhancers whose orthologs are not brain enhancers in mouse and negative set corresponding to non-enhancer orthologs of mouse enhancers
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/predictNewSequences.py --architectureFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsFlankPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_flankNeg_500bp_conv5MuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal_architecture.json --weightsFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsFlankPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_flankNeg_500bp_conv5MuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal.hdf5 --narrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MacaqueAtac/OfM/peak/idr_reproducibility/idr.optimal_peak.inM1AndStriatum_nonCDS_enhancerShort_nonMouseBrain_mouseTest.bed.gz --negNarrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_withPeakNames_rheMac8Huge_summitExtendedMin50Max2Protect5_nonMacaqueBrain_mouseTest.bed.gz --sequenceLength 500 --genomeFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MacaqueGenome/rheMac8.fa --chroms chr1 --chroms chr2 --chroms chr8 --chroms chr9 --chroms chr3 --chroms chr4 --chroms chr5 --chroms chr6 --chroms chr7 --chroms chr10 --chroms chr11 --chroms chr12 --chroms chr13 --chroms chr14 --chroms chr15 --chroms chr16 --chroms chr17 --chroms chr18 --chroms chr19 --chroms chr20 --chroms chr21 --chroms chr22 --chroms chrX --chroms chrY --RC --classification --path /home/ikaplow/RegulatoryElementEvolutionProject/src/deeplearning
# Performance: loss: nan, accuracy: 0.787879, sensitivity: 0.908719, specificity: 0.674745, auc: 0.895135, precision: 0.723427, NPV: 0.887584, auprc: 0.879766, auprg: 0.825629, F1: 0.805556, recallAt0.05FDR: 0.284741, recallAt0.1FDR: 0.633515
# Performance on rat enhancers whose orthologs are not brain enhancers in mouse and negative set corresponding to non-enhancer orthologs of mouse enhancers
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/predictNewSequences.py --architectureFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsFlankPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_flankNeg_500bp_conv5MuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal_architecture.json --weightsFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsFlankPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_flankNeg_500bp_conv5MuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal.hdf5 --narrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/RatAtac/M1_AllReps/peak/idr_reproducibility/idr.optimal_peak.inStriatum_nonCDS_enhancerShort_mouseTest_nonMouseBrain.bed.gz --negNarrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_withPeakNames_rn6Huge_summitExtendedMin50Max2Protect5_nonRatBrain_mouseTest.bed.gz --sequenceLength 500 --genomeFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/RatGenome/rn6.fa --chroms chr1 --chroms chr2 --chroms chr8 --chroms chr9 --chroms chr3 --chroms chr4 --chroms chr5 --chroms chr6 --chroms chr7 --chroms chr10 --chroms chr11 --chroms chr12 --chroms chr13 --chroms chr14 --chroms chr15 --chroms chr16 --chroms chr17 --chroms chr18 --chroms chr19 --chroms chr20 --chroms chr21 --chroms chr22 --chroms chrX --chroms chrY --RC --classification --path /home/ikaplow/RegulatoryElementEvolutionProject/src/deeplearning
# Performance: loss: nan, accuracy: 0.654731, sensitivity: 0.812360, specificity: 0.446588, auc: 0.706145, precision: 0.659672, NPV: 0.643162, auprc: 0.638580, auprg: 0.439078, F1: 0.728097, recallAt0.05FDR: 0.013353, recallAt0.1FDR: 0.013353
# Performance on non-restrictive glires-specific enhancers and non-enhancers
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/predictNewSequences.py --architectureFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsFlankPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_flankNeg_500bp_conv5MuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal_architecture.json --weightsFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsFlankPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_flankNeg_500bp_conv5MuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal.hdf5 --narrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_inRatBrainLoose_nonHumanBrain_nonMacaqueBrain.bed.gz --negNarrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/HumanDNase/PMC/cromwell-executions/atac/45c4bd8c-c055-4cea-80c9-5ec0eeaf8024/call-reproducibility_idr/execution/optimal_peak.inStriatum_nonCDS_enhancerShort_mm10_summitExtendedMin50Max2XProtect5_inMacaqueBrainLoose_nonMouseBrain_nonRatBrain.bed.gz --sequenceLength 500 --genomeFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseGenome/mm10.fa --chroms chr1 --chroms chr2 --RC --classification --path /home/ikaplow/RegulatoryElementEvolutionProject/src/deeplearning
# Performance: loss: 0.367839, accuracy: 0.829670, sensitivity: 0.839130, specificity: 0.813433, auc: 0.909961, precision: 0.885321, NPV: 0.746575, auprc: 0.860386, auprg: 0.873939, F1: 0.861607, recallAt0.05FDR: 0.313433, recallAt0.1FDR: 0.417910
# Performance on non-restrictive glires-specific enhancers and non-enhancers, calibrated
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/predictNewSequences.py --architectureFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsFlankPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_flankNeg_500bp_conv5MuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal_architecture.json --weightsFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsFlankPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_flankNeg_500bp_conv5MuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal.hdf5 --narrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_inRatBrainLoose_nonHumanBrain_nonMacaqueBrain.bed.gz --negNarrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/HumanDNase/PMC/cromwell-executions/atac/45c4bd8c-c055-4cea-80c9-5ec0eeaf8024/call-reproducibility_idr/execution/optimal_peak.inStriatum_nonCDS_enhancerShort_mm10_summitExtendedMin50Max2XProtect5_inMacaqueBrainLoose_nonMouseBrain_nonRatBrain.bed.gz --sequenceLength 500 --genomeFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseGenome/mm10.fa --chroms chr1 --chroms chr2 --RC --classification --path /home/ikaplow/RegulatoryElementEvolutionProject/src/deeplearning --recalibrateModel --calibrationXTrFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerVsOtherEuarchontaglireEnhLoosePeakLessThan1kb500bp_data_train.npy --calibrationYTrFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerVsOtherEuarchontaglireEnhLoosePeakLessThan1kb500bp_labels_train.npy
# Performance: loss: 0.513716, accuracy: 0.799451, sensitivity: 0.743478, specificity: 0.895522, auc: 0.909961, precision: 0.924324, NPV: 0.670391, auprc: 0.860386, auprg: 0.873939, F1: 0.824096, recallAt0.05FDR: 0.313433, recallAt0.1FDR: 0.417910
# Performance on non-restrictive euarchonta-specific enhancers and non-enhancers
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/predictNewSequences.py --architectureFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsFlankPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_flankNeg_500bp_conv5MuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal_architecture.json --weightsFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsFlankPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_flankNeg_500bp_conv5MuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal.hdf5 --narrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/HumanDNase/PMC/cromwell-executions/atac/45c4bd8c-c055-4cea-80c9-5ec0eeaf8024/call-reproducibility_idr/execution/optimal_peak.inStriatum_nonCDS_enhancerShort_inMacaqueBrainLoose_nonMouseBrain_nonRatBrain_mouseTest.bed.gz --negNarrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_withPeakNames_hg38Huge_summitExtendedMin50Max2Protect5_mouseTest_inRatBrainLoose_nonHumanBrain_nonMacaqueBrain.bed.gz --sequenceLength 500 --genomeFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/HumanGenome/hg38.fa --RC --classification --path /home/ikaplow/RegulatoryElementEvolutionProject/src/deeplearning
# Performance: loss: nan, accuracy: 0.678571, sensitivity: 0.738806, specificity: 0.643478, auc: 0.781927, precision: 0.546961, NPV: 0.808743, auprc: 0.730989, auprg: 0.689050, F1: 0.628571, recallAt0.05FDR: 0.216418, recallAt0.1FDR: 0.253731
# Performance on non-restrictive euarchonta-specific enhancers and non-enhancers, calibrated
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/predictNewSequences.py --architectureFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsFlankPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_flankNeg_500bp_conv5MuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal_architecture.json --weightsFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsFlankPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_flankNeg_500bp_conv5MuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal.hdf5 --narrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/HumanDNase/PMC/cromwell-executions/atac/45c4bd8c-c055-4cea-80c9-5ec0eeaf8024/call-reproducibility_idr/execution/optimal_peak.inStriatum_nonCDS_enhancerShort_inMacaqueBrainLoose_nonMouseBrain_nonRatBrain_mouseTest.bed.gz --negNarrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_withPeakNames_hg38Huge_summitExtendedMin50Max2Protect5_mouseTest_inRatBrainLoose_nonHumanBrain_nonMacaqueBrain.bed.gz --sequenceLength 500 --genomeFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/HumanGenome/hg38.fa --RC --classification --path /home/ikaplow/RegulatoryElementEvolutionProject/src/deeplearning --recalibrateModel --calibrationXTrFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerVsOtherEuarchontaglireEnhLoosePeakLessThan1kb500bp_data_train.npy --calibrationYTrFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerVsOtherEuarchontaglireEnhLoosePeakLessThan1kb500bp_labels_train.npy
# Performance: loss: 0.612441, accuracy: 0.744505, sensitivity: 0.649254, specificity: 0.800000, auc: 0.781927, precision: 0.654135, NPV: 0.796537, auprc: 0.730989, auprg: 0.689050, F1: 0.651685, recallAt0.05FDR: 0.216418, recallAt0.1FDR: 0.253731
# Performance on non-restrictive mouse-specific enhancers and non-enhancers
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/predictNewSequences.py --architectureFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsFlankPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_flankNeg_500bp_conv5MuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal_architecture.json --weightsFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsFlankPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_flankNeg_500bp_conv5MuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal.hdf5 --narrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_withPeakNames_nonRatBrain.bed.gz --negNarrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/RatAtac/M1_AllReps/peak/idr_reproducibility/idr.optimal_peak.inStriatum_nonCDS_enhancerShort_mm10_summitExtendedMin50Max2XProtect5_nonMouseBrain.bed.gz --sequenceLength 500 --genomeFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseGenome/mm10.fa --chroms chr1 --chroms chr2 --RC --classification --path /home/ikaplow/RegulatoryElementEvolutionProject/src/deeplearning
# Performance: loss: nan, accuracy: 0.727621, sensitivity: 0.893175, specificity: 0.602247, auc: 0.832790, precision: 0.629707, NPV: 0.881579, auprc: 0.767521, auprg: 0.710684, F1: 0.738650, recallAt0.05FDR: 0.007418, recallAt0.1FDR: 0.204748
# Performance on non-restrictive human-specific enhancers and non-enhancers
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/predictNewSequences.py --architectureFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsFlankPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_flankNeg_500bp_conv5MuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal_architecture.json --weightsFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsFlankPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_flankNeg_500bp_conv5MuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal.hdf5 --narrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/HumanDNase/PMC/cromwell-executions/atac/45c4bd8c-c055-4cea-80c9-5ec0eeaf8024/call-reproducibility_idr/execution/optimal_peak.inStriatum_nonCDS_enhancerShort_nonMacaqueBrain_mouseTest.bed.gz --negNarrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MacaqueAtac/OfM/peak/idr_reproducibility/idr.optimal_peak.inM1AndStriatum_nonCDS_enhancerShort_hg38_summitExtendedMin50Max2XProtect5_nonHumanBrain_mouseTest.bed.gz --sequenceLength 500 --genomeFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/HumanGenome/hg38.fa --RC --classification --path /home/ikaplow/RegulatoryElementEvolutionProject/src/deeplearning
# Performance: loss: inf, accuracy: 0.356061, sensitivity: 0.772727, specificity: 0.217172, auc: 0.575911, precision: 0.247573, NPV: 0.741379, auprc: 0.311429, auprg: 0.327950, F1: 0.375000, recallAt0.05FDR: 0.000000, recallAt0.1FDR: 0.000000
# Performance on non-restrictive macaque-specific enhancers and non-enhancers
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/predictNewSequences.py --architectureFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsFlankPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_flankNeg_500bp_conv5MuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal_architecture.json --weightsFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsFlankPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_flankNeg_500bp_conv5MuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal.hdf5 --narrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MacaqueAtac/OfM/peak/idr_reproducibility/idr.optimal_peak.inM1AndStriatum_nonCDS_enhancerShort_nonHumanBrain_mouseTest.bed.gz --negNarrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/HumanDNase/PMC/cromwell-executions/atac/45c4bd8c-c055-4cea-80c9-5ec0eeaf8024/call-reproducibility_idr/execution/optimal_peak.inStriatum_nonCDS_enhancerShort_rheMac8_summitExtendedMin50Max2XProtect5_UCSCNames_nonMacaqueBrain_mouseTest.bed.gz --sequenceLength 500 --genomeFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MacaqueGenome/rheMac8.fa --RC --classification --path /home/ikaplow/RegulatoryElementEvolutionProject/src/deeplearning
# Performance: loss: 0.515634, accuracy: 0.780303, sensitivity: 0.893939, specificity: 0.439394, auc: 0.779538, precision: 0.827103, NPV: 0.580000, auprc: 0.602004, auprg: 0.750061, F1: 0.859223, recallAt0.05FDR: 0.090909, recallAt0.1FDR: 0.136364
# Performance on shared brain and liver enhancers and liver enhancers that are not brain enhancers
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/predictNewSequences.py --architectureFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsFlankPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_flankNeg_500bp_conv5MuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal_architecture.json --weightsFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsFlankPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_flankNeg_500bp_conv5MuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal.hdf5 --narrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_andLiverLiuPfenning.narrowPeak.gz --negNarrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/LiverPfenning_ATAC_out/atac/68eb27d1-23c7-4d42-80f9-b3dd46f5e18c/call-reproducibility_idr/execution/idr.optimal_peak.narrowPeak_inLiuAll_nonCDS_enhancerShort_nonBrain.narrowPeak.gz --sequenceLength 500 --genomeFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseGenome/mm10.fa  --chroms chr1 --chroms chr2 --RC --classification --path /home/ikaplow/RegulatoryElementEvolutionProject/src/deeplearning
# Performance: loss: nan, accuracy: 0.812302, sensitivity: 0.898077, specificity: 0.785925, auc: 0.934877, precision: 0.563329, NPV: 0.961650, auprc: 0.888877, auprg: 0.971336, F1: 0.692365, recallAt0.05FDR: 0.674038, recallAt0.1FDR: 0.722115
# Performance on shared brain and liver enhancers and liver enhancers that are not brain enhancers, calibrated
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/predictNewSequences.py --architectureFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsFlankPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_flankNeg_500bp_conv5MuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal_architecture.json --weightsFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsFlankPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_flankNeg_500bp_conv5MuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal.hdf5 --narrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_andLiverLiuPfenning.narrowPeak.gz --negNarrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/LiverPfenning_ATAC_out/atac/68eb27d1-23c7-4d42-80f9-b3dd46f5e18c/call-reproducibility_idr/execution/idr.optimal_peak.narrowPeak_inLiuAll_nonCDS_enhancerShort_nonBrain.narrowPeak.gz --sequenceLength 500 --genomeFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseGenome/mm10.fa  --chroms chr1 --chroms chr2 --RC --classification --path /home/ikaplow/RegulatoryElementEvolutionProject/src/deeplearning --recalibrateModel --calibrationXTrFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerVsOtherEuarchontaglireEnhLoosePeakLessThan1kb500bp_data_train.npy --calibrationYTrFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerVsOtherEuarchontaglireEnhLoosePeakLessThan1kb500bp_labels_train.npy
# Performance: loss: 0.289512, accuracy: 0.880371, sensitivity: 0.851923, specificity: 0.889119, auc: 0.934877, precision: 0.702617, NPV: 0.951281, auprc: 0.888877, auprg: 0.971336, F1: 0.770100, recallAt0.05FDR: 0.674038, recallAt0.1FDR: 0.722115
# Predictions for brain only enhancers, shared brain and liver enhancers, liver only enhancers, and negative set
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/predictNewSequencesNoEvaluation.py --architectureFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsFlankPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_flankNeg_500bp_conv5MuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal_architecture.json --weightsFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsFlankPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_flankNeg_500bp_conv5MuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal.hdf5 --narrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_nonLiver.narrowPeak.gz --sequenceLength 500 --predictedProbaFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_nonLiver_test_flankNeg_predictedProba.txt --genomeFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseGenome/mm10.fa --chromSizesFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseGenome/mm10.chrom.sizes --chroms chr1 --chroms chr2 --classification
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/predictNewSequencesNoEvaluation.py --architectureFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsFlankPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_flankNeg_500bp_conv5MuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal_architecture.json --weightsFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsFlankPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_flankNeg_500bp_conv5MuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal.hdf5 --narrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_andLiverLiuPfenning.narrowPeak.gz --sequenceLength 500 --predictedProbaFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_andLiverLiuPfenning_test_flankNeg_predictedProba.txt --genomeFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseGenome/mm10.fa --chromSizesFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseGenome/mm10.chrom.sizes --chroms chr1 --chroms chr2 --classification
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/predictNewSequencesNoEvaluation.py --architectureFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsFlankPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_flankNeg_500bp_conv5MuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal_architecture.json --weightsFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsFlankPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_flankNeg_500bp_conv5MuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal.hdf5 --narrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/LiverPfenning_ATAC_out/atac/68eb27d1-23c7-4d42-80f9-b3dd46f5e18c/call-reproducibility_idr/execution/idr.optimal_peak.narrowPeak_inLiuAll_nonCDS_enhancerShort_nonBrain.narrowPeak.gz --sequenceLength 500 --predictedProbaFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/LiverPfenning_ATAC_out/atac/68eb27d1-23c7-4d42-80f9-b3dd46f5e18c/call-reproducibility_idr/execution/idr.optimal_peak.narrowPeak_inLiuAll_nonCDS_enhancerShort_nonBrain_test_flankNeg_predictedProba.txt --genomeFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseGenome/mm10.fa --chromSizesFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseGenome/mm10.chrom.sizes --chroms chr1 --chroms chr2 --classification
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/predictNewSequencesNoEvaluation.py --architectureFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsFlankPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_flankNeg_500bp_conv5MuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal_architecture.json --weightsFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsFlankPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_flankNeg_500bp_conv5MuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal.hdf5 --narrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_withPeakNames_flank500bpSep500bp.bed.gz --sequenceLength 500 --predictedProbaFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_withPeakNames_flank500bpSep500bp_test_predictedProba.txt --genomeFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseGenome/mm10.fa --chromSizesFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseGenome/mm10.chrom.sizes --chroms chr1 --chroms chr2 --classification
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/makeViolinPlotTissueComparison.py --tissueOnlyPredFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_nonLiver_test_flankNeg_predictedProba.txt --tissueSharedPredFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_andLiverLiuPfenning_test_flankNeg_predictedProba.txt --tissueOtherPredFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/LiverPfenning_ATAC_out/atac/68eb27d1-23c7-4d42-80f9-b3dd46f5e18c/call-reproducibility_idr/execution/idr.optimal_peak.narrowPeak_inLiuAll_nonCDS_enhancerShort_nonBrain_test_flankNeg_predictedProba.txt --negSetPredFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_withPeakNames_flank500bpSep500bp_test_predictedProba.txt --outputFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_brainNonLiverVsBrainLiverVsLiverNonBrainVsFlankNeg_predictedProba.svg
# Performance on macaque shared brain and liver enhancers and liver enhancers that are not brain enhancers
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/predictNewSequences.py --architectureFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsFlankPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_flankNeg_500bp_conv5MuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal_architecture.json --weightsFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsFlankPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_flankNeg_500bp_conv5MuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal.hdf5 --narrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MacaqueAtac/OfM/peak/idr_reproducibility/idr.optimal_peak.inM1AndStriatum_nonCDS_enhancerShort_andLiver_mouseTest.bed.gz --negNarrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MacaqueAtac/Liver/cromwell-executions/atac/1f5fd9f0-d389-4184-99cb-c1125fd7f064/call-reproducibility_idr/execution/optimal_peak_nonCDS_enhancerShort_mouseTest_nonMacaqueBrain.bed.gz --sequenceLength 500 --genomeFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MacaqueGenome/rheMac8.fa --chroms chr1 --chroms chr2 --chroms chr8 --chroms chr9 --chroms chr3 --chroms chr4 --chroms chr5 --chroms chr6 --chroms chr7 --chroms chr10 --chroms chr11 --chroms chr12 --chroms chr13 --chroms chr14 --chroms chr15 --chroms chr16 --chroms chr17 --chroms chr18 --chroms chr19 --chroms chr20 --chroms chrX --chroms chrY --RC --classification --path /home/ikaplow/RegulatoryElementEvolutionProject/src/deeplearning
# Performance: loss: nan, accuracy: 0.800511, sensitivity: 0.946809, specificity: 0.735232, auc: 0.953791, precision: 0.614735, NPV: 0.968728, auprc: 0.912940, auprg: 0.963954, F1: 0.745463, recallAt0.05FDR: 0.659574, recallAt0.1FDR: 0.795508
# Performance on macaque shared brain and liver enhancers and liver enhancers that are not brain enhancers, calibrated
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/predictNewSequences.py --architectureFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsFlankPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_flankNeg_500bp_conv5MuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal_architecture.json --weightsFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsFlankPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_flankNeg_500bp_conv5MuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal.hdf5 --narrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MacaqueAtac/OfM/peak/idr_reproducibility/idr.optimal_peak.inM1AndStriatum_nonCDS_enhancerShort_andLiver_mouseTest.bed.gz --negNarrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MacaqueAtac/Liver/cromwell-executions/atac/1f5fd9f0-d389-4184-99cb-c1125fd7f064/call-reproducibility_idr/execution/optimal_peak_nonCDS_enhancerShort_mouseTest_nonMacaqueBrain.bed.gz --sequenceLength 500 --genomeFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MacaqueGenome/rheMac8.fa --chroms chr1 --chroms chr2 --chroms chr8 --chroms chr9 --chroms chr3 --chroms chr4 --chroms chr5 --chroms chr6 --chroms chr7 --chroms chr10 --chroms chr11 --chroms chr12 --chroms chr13 --chroms chr14 --chroms chr15 --chroms chr16 --chroms chr17 --chroms chr18 --chroms chr19 --chroms chr20 --chroms chrX --chroms chrY --RC --classification --path /home/ikaplow/RegulatoryElementEvolutionProject/src/deeplearning --recalibrateModel --calibrationXTrFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerVsOtherEuarchontaglireEnhLoosePeakLessThan1kb500bp_data_train.npy --calibrationYTrFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerVsOtherEuarchontaglireEnhLoosePeakLessThan1kb500bp_labels_train.npy
# Performance: loss: 0.299479, accuracy: 0.870168, sensitivity: 0.907801, specificity: 0.853376, auc: 0.953791, precision: 0.734226, NPV: 0.954009, auprc: 0.912940, auprg: 0.963954, F1: 0.811839, recallAt0.05FDR: 0.659574, recallAt0.1FDR: 0.795508
# Performance on rat shared brain and liver enhancers and liver enhancers that are not brain enhancers
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/predictNewSequences.py --architectureFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsFlankPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_flankNeg_500bp_conv5MuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal_architecture.json --weightsFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsFlankPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_flankNeg_500bp_conv5MuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal.hdf5 --narrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/RatAtac/M1_AllReps/peak/idr_reproducibility/idr.optimal_peak.inStriatum_nonCDS_enhancerShort_andLiver_mouseTest.bed.gz --negNarrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/RatAtac/Liver_GoodReps/peak/idr_reproducibility/idr.optimal_peak_nonCDS_enhancerShort_nonRatBrain_mouseTest.bed.gz --sequenceLength 500 --genomeFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/RatGenome/rn6.fa --chroms chr1 --chroms chr2 --chroms chr8 --chroms chr9 --chroms chr3 --chroms chr4 --chroms chr5 --chroms chr6 --chroms chr7 --chroms chr10 --chroms chr11 --chroms chr12 --chroms chr13 --chroms chr14 --chroms chr15 --chroms chr16 --chroms chr17 --chroms chr18 --chroms chr19 --chroms chr20 --chroms chrX --chroms chrY --RC --classification
# Performance: loss: nan, accuracy: 0.791036, sensitivity: 0.859887, specificity: 0.756396, auc: 0.903113, precision: 0.639765, NPV: 0.914747, auprc: 0.863384, auprg: 0.916941, F1: 0.733671, recallAt0.05FDR: 0.520339, recallAt0.1FDR: 0.611864
# Performance on rat shared brain and liver enhancers and liver enhancers that are not brain enhancers, calibrated
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/predictNewSequences.py --architectureFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsFlankPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_flankNeg_500bp_conv5MuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal_architecture.json --weightsFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsFlankPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_flankNeg_500bp_conv5MuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal.hdf5 --narrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/RatAtac/M1_AllReps/peak/idr_reproducibility/idr.optimal_peak.inStriatum_nonCDS_enhancerShort_andLiver_mouseTest.bed.gz --negNarrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/RatAtac/Liver_GoodReps/peak/idr_reproducibility/idr.optimal_peak_nonCDS_enhancerShort_nonRatBrain_mouseTest.bed.gz --sequenceLength 500 --genomeFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/RatGenome/rn6.fa --chroms chr1 --chroms chr2 --chroms chr8 --chroms chr9 --chroms chr3 --chroms chr4 --chroms chr5 --chroms chr6 --chroms chr7 --chroms chr10 --chroms chr11 --chroms chr12 --chroms chr13 --chroms chr14 --chroms chr15 --chroms chr16 --chroms chr17 --chroms chr18 --chroms chr19 --chroms chr20 --chroms chrX --chroms chrY --RC --classification --recalibrateModel --calibrationXTrFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerVsOtherEuarchontaglireEnhLoosePeakLessThan1kb500bp_data_train.npy --calibrationYTrFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerVsOtherEuarchontaglireEnhLoosePeakLessThan1kb500bp_labels_train.npy
# Performance: loss: 0.390411, accuracy: 0.834153, sensitivity: 0.791525, specificity: 0.855600, auc: 0.903113, precision: 0.733892, NPV: 0.890796, auprc: 0.863384, auprg: 0.916941, F1: 0.761620, recallAt0.05FDR: 0.520339, recallAt0.1FDR: 0.611864
# Get brain test set predictions across Zoonomia mammals
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/makePredictNewSequencesNoEvaluationScript.py --architectureFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsFlankPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_flankNeg_500bp_conv5MuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal_architecture.json --weightsFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsFlankPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_flankNeg_500bp_conv5MuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal.hdf5 --narrowPeakFileNameListFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_orLiver_LiftedPeaks/liftedPeakMouseTestNarrowPeakFileNamesFiltConverted.txt --genomeFileNameListFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/200MammalsFastas/200MammalsAssembliesFileNamesFilt.txt --chromSizesFileNameListFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/200MammalsFastas/chromSizesFileNames.txt --classification --scriptFileName /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/predictMouseBrainTestHugeCactusFlankScript.sh --predictedClassesFileNameSuffix flankNeg_predictedClasses.txt --predictedProbaFileNameSuffix flankNeg_predictedProba.txt
sh /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/predictMouseBrainTestHugeCactusFlankScript.sh > /home/ikaplow/RegulatoryElementEvolutionProject/src/predictMouseBrainTestHugeCactusFlankScript.e
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/gatherPeakPredictionsAcrossSpecies.py --predictionsFileNameListFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/testPredictionsFlankGliresFileNames.txt --baseSpeciesLine 35 --outputFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_test_flankNeg_peakPredictionsGlires.txt # Filtered correlation (Pearson, Spearman): -0.8935*, -0.6455*

# Performance of brain model with non-cerebrum enhancer negatives on test set
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/predictNewSequences.py --architectureFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsNonCerebrumMouseTissuePeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_nonCerebrumMouseTissueNeg_500bp_conv5MuchMoreFiltNoInterPoolVeryHighMomL2SmallBatchPretrainBal_architecture.json --weightsFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsNonCerebrumMouseTissuePeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_nonCerebrumMouseTissueNeg_500bp_conv5MuchMoreFiltNoInterPoolVeryHighMomL2SmallBatchPretrainBal.hdf5 --narrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort.narrowPeak.gz --negNarrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/LiverPfenning_ATAC_out/atac/68eb27d1-23c7-4d42-80f9-b3dd46f5e18c/call-reproducibility_idr/execution/idr.optimal_peak_nonCDS_enhancerShort_andAllLiuAndAllEncode_nonMouseCerebrum.narrowPeak.gz --sequenceLength 500 --genomeFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseGenome/mm10.fa --chroms chr1 --chroms chr2 --predictedProbaFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_test_andNeg_nonCerebrumMouseTissueNeg_conv5MuchMuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal_predictedProba.txt --RC --classification --path /home/ikaplow/RegulatoryElementEvolutionProject/src/deeplearning
# Performance: loss: 0.146615, accuracy: 0.956967, sensitivity: 0.739510, specificity: 0.969077, auc: 0.956544, precision: 0.571139, NPV: 0.985252, auprc: 0.738974, auprg: 0.995276, F1: 0.644510, recallAt0.05FDR: 0.312719, recallAt0.1FDR: 0.451923
# Performance on mouse enhancers whose orthologs are not brain enhancers in at least one other species and negative set
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/predictNewSequences.py --architectureFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsNonCerebrumMouseTissuePeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_nonCerebrumMouseTissueNeg_500bp_conv5MuchMoreFiltNoInterPoolVeryHighMomL2SmallBatchPretrainBal_architecture.json --weightsFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsNonCerebrumMouseTissuePeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_nonCerebrumMouseTissueNeg_500bp_conv5MuchMoreFiltNoInterPoolVeryHighMomL2SmallBatchPretrainBal.hdf5 --narrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_withPeakNames_nonBrainOtherSpecies.narrowPeak.gz --negNarrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/HumanDNase/PMC/cromwell-executions/atac/45c4bd8c-c055-4cea-80c9-5ec0eeaf8024/call-reproducibility_idr/execution/optimal_peak.inStriatum_nonCDS_enhancerShort_mm10_summitExtendedMin50Max2XProtect5_nonMouseBrain_andMacaque_andRat.narrowPeak.gz --sequenceLength 500 --genomeFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseGenome/mm10.fa --chroms chr1 --chroms chr2 --RC --classification --path /home/ikaplow/RegulatoryElementEvolutionProject/src/deeplearning
# Performance: loss: 0.505354, accuracy: 0.755782, sensitivity: 0.757962, specificity: 0.754122, auc: 0.834725, precision: 0.701237, NPV: 0.803618, auprc: 0.793417, auprg: 0.728067, F1: 0.728497, recallAt0.05FDR: 0.003822, recallAt0.1FDR: 0.311465
# Performance on human enhancers whose orthologs are not brain enhancers in mouse and negative set corresponding to non-enhancer orthologs of mouse enhancers
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/predictNewSequences.py --architectureFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsNonCerebrumMouseTissuePeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_nonCerebrumMouseTissueNeg_500bp_conv5MuchMoreFiltNoInterPoolVeryHighMomL2SmallBatchPretrainBal_architecture.json --weightsFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsNonCerebrumMouseTissuePeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_nonCerebrumMouseTissueNeg_500bp_conv5MuchMoreFiltNoInterPoolVeryHighMomL2SmallBatchPretrainBal.hdf5 --narrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/HumanDNase/PMC/cromwell-executions/atac/45c4bd8c-c055-4cea-80c9-5ec0eeaf8024/call-reproducibility_idr/execution/optimal_peak.inStriatum_nonCDS_enhancerShort_nonMouseBrain_mouseTest.bed.gz --negNarrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_withPeakNames_hg38Huge_summitExtendedMin50Max2Protect5_nonHumanBrain_mouseTest.narrowPeak.gz --sequenceLength 500 --genomeFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/HumanGenome/hg38.fa --RC --classification --path /home/ikaplow/RegulatoryElementEvolutionProject/src/deeplearning
# Performance: loss: 0.704504, accuracy: 0.637195, sensitivity: 0.730769, specificity: 0.593750, auc: 0.732272, precision: 0.455090, NPV: 0.826087, auprc: 0.568682, auprg: 0.578054, F1: 0.560886, recallAt0.05FDR: 0.004808, recallAt0.1FDR: 0.004808
# Performance on macaque enhancers whose orthologs are not brain enhancers in mouse and negative set corresponding to non-enhancer orthologs of mouse enhancers
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/predictNewSequences.py --architectureFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsNonCerebrumMouseTissuePeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_nonCerebrumMouseTissueNeg_500bp_conv5MuchMoreFiltNoInterPoolVeryHighMomL2SmallBatchPretrainBal_architecture.json --weightsFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsNonCerebrumMouseTissuePeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_nonCerebrumMouseTissueNeg_500bp_conv5MuchMoreFiltNoInterPoolVeryHighMomL2SmallBatchPretrainBal.hdf5 --narrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MacaqueAtac/OfM/peak/idr_reproducibility/idr.optimal_peak.inM1AndStriatum_nonCDS_enhancerShort_nonMouseBrain_mouseTest.bed.gz --negNarrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_withPeakNames_rheMac8Huge_summitExtendedMin50Max2Protect5_nonMacaqueBrain_mouseTest.bed.gz --sequenceLength 500 --genomeFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MacaqueGenome/rheMac8.fa --chroms chr1 --chroms chr2 --chroms chr8 --chroms chr9 --chroms chr3 --chroms chr4 --chroms chr5 --chroms chr6 --chroms chr7 --chroms chr10 --chroms chr11 --chroms chr12 --chroms chr13 --chroms chr14 --chroms chr15 --chroms chr16 --chroms chr17 --chroms chr18 --chroms chr19 --chroms chr20 --chroms chr21 --chroms chr22 --chroms chrX --chroms chrY --RC --classification --path /home/ikaplow/RegulatoryElementEvolutionProject/src/deeplearning
# Performance: loss: 0.646001, accuracy: 0.675889, sensitivity: 0.731608, specificity: 0.623724, auc: 0.744001, precision: 0.645433, NPV: 0.712828, auprc: 0.722995, auprg: 0.499869, F1: 0.685824, recallAt0.05FDR: 0.006812, recallAt0.1FDR: 0.029973
# Performance on rat enhancers whose orthologs are not brain enhancers in mouse and negative set corresponding to non-enhancer orthologs of mouse enhancers
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/predictNewSequences.py --architectureFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsNonCerebrumMouseTissuePeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_nonCerebrumMouseTissueNeg_500bp_conv5MuchMoreFiltNoInterPoolVeryHighMomL2SmallBatchPretrainBal_architecture.json --weightsFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsNonCerebrumMouseTissuePeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_nonCerebrumMouseTissueNeg_500bp_conv5MuchMoreFiltNoInterPoolVeryHighMomL2SmallBatchPretrainBal.hdf5 --narrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/RatAtac/M1_AllReps/peak/idr_reproducibility/idr.optimal_peak.inStriatum_nonCDS_enhancerShort_mouseTest_nonMouseBrain.bed.gz --negNarrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_withPeakNames_rn6Huge_summitExtendedMin50Max2Protect5_nonRatBrain_mouseTest.bed.gz --sequenceLength 500 --genomeFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/RatGenome/rn6.fa --chroms chr1 --chroms chr2 --chroms chr8 --chroms chr9 --chroms chr3 --chroms chr4 --chroms chr5 --chroms chr6 --chroms chr7 --chroms chr10 --chroms chr11 --chroms chr12 --chroms chr13 --chroms chr14 --chroms chr15 --chroms chr16 --chroms chr17 --chroms chr18 --chroms chr19 --chroms chr20 --chroms chr21 --chroms chr22 --chroms chrX --chroms chrY --RC --classification --path /home/ikaplow/RegulatoryElementEvolutionProject/src/deeplearning
# Performance: loss: 0.802951, accuracy: 0.586317, sensitivity: 0.569663, specificity: 0.608309, auc: 0.621867, precision: 0.657588, NPV: 0.517024, auprc: 0.548135, auprg: 0.266775, F1: 0.610476, recallAt0.05FDR: 0.007418, recallAt0.1FDR: 0.014837
# Performance on non-restrictive glires-specific enhancers and non-enhancers
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/predictNewSequences.py --architectureFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsNonCerebrumMouseTissuePeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_nonCerebrumMouseTissueNeg_500bp_conv5MuchMoreFiltNoInterPoolVeryHighMomL2SmallBatchPretrainBal_architecture.json --weightsFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsNonCerebrumMouseTissuePeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_nonCerebrumMouseTissueNeg_500bp_conv5MuchMoreFiltNoInterPoolVeryHighMomL2SmallBatchPretrainBal.hdf5 --narrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_inRatBrainLoose_nonHumanBrain_nonMacaqueBrain.bed.gz --negNarrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/HumanDNase/PMC/cromwell-executions/atac/45c4bd8c-c055-4cea-80c9-5ec0eeaf8024/call-reproducibility_idr/execution/optimal_peak.inStriatum_nonCDS_enhancerShort_mm10_summitExtendedMin50Max2XProtect5_inMacaqueBrainLoose_nonMouseBrain_nonRatBrain.bed.gz --sequenceLength 500 --genomeFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseGenome/mm10.fa --chroms chr1 --chroms chr2 --RC --classification --path /home/ikaplow/RegulatoryElementEvolutionProject/src/deeplearning
# Performance: loss: 0.513861, accuracy: 0.755495, sensitivity: 0.734783, specificity: 0.791045, auc: 0.854705, precision: 0.857868, NPV: 0.634731, auprc: 0.753566, auprg: 0.794533, F1: 0.791569, recallAt0.05FDR: 0.007463, recallAt0.1FDR: 0.268657
# Performance on non-restrictive glires-specific enhancers and non-enhancers, calibrated
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/predictNewSequences.py --architectureFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsNonCerebrumMouseTissuePeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_nonCerebrumMouseTissueNeg_500bp_conv5MuchMoreFiltNoInterPoolVeryHighMomL2SmallBatchPretrainBal_architecture.json --weightsFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsNonCerebrumMouseTissuePeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_nonCerebrumMouseTissueNeg_500bp_conv5MuchMoreFiltNoInterPoolVeryHighMomL2SmallBatchPretrainBal.hdf5 --narrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_inRatBrainLoose_nonHumanBrain_nonMacaqueBrain.bed.gz --negNarrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/HumanDNase/PMC/cromwell-executions/atac/45c4bd8c-c055-4cea-80c9-5ec0eeaf8024/call-reproducibility_idr/execution/optimal_peak.inStriatum_nonCDS_enhancerShort_mm10_summitExtendedMin50Max2XProtect5_inMacaqueBrainLoose_nonMouseBrain_nonRatBrain.bed.gz --sequenceLength 500 --genomeFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseGenome/mm10.fa --chroms chr1 --chroms chr2 --RC --classification --path /home/ikaplow/RegulatoryElementEvolutionProject/src/deeplearning --recalibrateModel --calibrationXTrFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerVsOtherEuarchontaglireEnhLoosePeakLessThan1kb500bp_data_train.npy --calibrationYTrFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerVsOtherEuarchontaglireEnhLoosePeakLessThan1kb500bp_labels_train.npy
# Performance: loss: 0.529398, accuracy: 0.752747, sensitivity: 0.713043, specificity: 0.820896, auc: 0.854705, precision: 0.872340, NPV: 0.625000, auprc: 0.753566, auprg: 0.794533, F1: 0.784689, recallAt0.05FDR: 0.007463, recallAt0.1FDR: 0.268657
# Performance on non-restrictive euarchonta-specific enhancers and non-enhancers
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/predictNewSequences.py --architectureFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsNonCerebrumMouseTissuePeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_nonCerebrumMouseTissueNeg_500bp_conv5MuchMoreFiltNoInterPoolVeryHighMomL2SmallBatchPretrainBal_architecture.json --weightsFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsNonCerebrumMouseTissuePeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_nonCerebrumMouseTissueNeg_500bp_conv5MuchMoreFiltNoInterPoolVeryHighMomL2SmallBatchPretrainBal.hdf5 --narrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/HumanDNase/PMC/cromwell-executions/atac/45c4bd8c-c055-4cea-80c9-5ec0eeaf8024/call-reproducibility_idr/execution/optimal_peak.inStriatum_nonCDS_enhancerShort_inMacaqueBrainLoose_nonMouseBrain_nonRatBrain_mouseTest.bed.gz --negNarrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_withPeakNames_hg38Huge_summitExtendedMin50Max2Protect5_mouseTest_inRatBrainLoose_nonHumanBrain_nonMacaqueBrain.bed.gz --sequenceLength 500 --genomeFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/HumanGenome/hg38.fa --RC --classification --path /home/ikaplow/RegulatoryElementEvolutionProject/src/deeplearning
# Performance: loss: 0.655268, accuracy: 0.692308, sensitivity: 0.753731, specificity: 0.656522, auc: 0.751103, precision: 0.561111, NPV: 0.820652, auprc: 0.606271, auprg: 0.591302, F1: 0.643312, recallAt0.05FDR: 0.014925, recallAt0.1FDR: 0.014925
# Performance on non-restrictive euarchonta-specific enhancers and non-enhancers, calibrated
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/predictNewSequences.py --architectureFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsNonCerebrumMouseTissuePeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_nonCerebrumMouseTissueNeg_500bp_conv5MuchMoreFiltNoInterPoolVeryHighMomL2SmallBatchPretrainBal_architecture.json --weightsFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsNonCerebrumMouseTissuePeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_nonCerebrumMouseTissueNeg_500bp_conv5MuchMoreFiltNoInterPoolVeryHighMomL2SmallBatchPretrainBal.hdf5 --narrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/HumanDNase/PMC/cromwell-executions/atac/45c4bd8c-c055-4cea-80c9-5ec0eeaf8024/call-reproducibility_idr/execution/optimal_peak.inStriatum_nonCDS_enhancerShort_inMacaqueBrainLoose_nonMouseBrain_nonRatBrain_mouseTest.bed.gz --negNarrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_withPeakNames_hg38Huge_summitExtendedMin50Max2Protect5_mouseTest_inRatBrainLoose_nonHumanBrain_nonMacaqueBrain.bed.gz --sequenceLength 500 --genomeFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/HumanGenome/hg38.fa --RC --classification --path /home/ikaplow/RegulatoryElementEvolutionProject/src/deeplearning --recalibrateModel --calibrationXTrFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerVsOtherEuarchontaglireEnhLoosePeakLessThan1kb500bp_data_train.npy --calibrationYTrFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerVsOtherEuarchontaglireEnhLoosePeakLessThan1kb500bp_labels_train.npy
# Performance: loss: 0.610292, accuracy: 0.695055, sensitivity: 0.716418, specificity: 0.682609, auc: 0.751103, precision: 0.568047, NPV: 0.805128, auprc: 0.606271, auprg: 0.591302, F1: 0.633663, recallAt0.05FDR: 0.014925, recallAt0.1FDR: 0.014925
# Performance on non-restrictive mouse-specific enhancers and non-enhancers
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/predictNewSequences.py --architectureFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsNonCerebrumMouseTissuePeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_nonCerebrumMouseTissueNeg_500bp_conv5MuchMoreFiltNoInterPoolVeryHighMomL2SmallBatchPretrainBal_architecture.json --weightsFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsNonCerebrumMouseTissuePeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_nonCerebrumMouseTissueNeg_500bp_conv5MuchMoreFiltNoInterPoolVeryHighMomL2SmallBatchPretrainBal.hdf5 --narrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_withPeakNames_nonRatBrain.bed.gz --negNarrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/RatAtac/M1_AllReps/peak/idr_reproducibility/idr.optimal_peak.inStriatum_nonCDS_enhancerShort_mm10_summitExtendedMin50Max2XProtect5_nonMouseBrain.bed.gz --sequenceLength 500 --genomeFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseGenome/mm10.fa --chroms chr1 --chroms chr2 --RC --classification --path /home/ikaplow/RegulatoryElementEvolutionProject/src/deeplearning
# Performance: loss: 0.519144, accuracy: 0.743606, sensitivity: 0.758160, specificity: 0.732584, auc: 0.824174, precision: 0.682243, NPV: 0.800000, auprc: 0.795274, auprg: 0.703628, F1: 0.718201, recallAt0.05FDR: 0.222552, recallAt0.1FDR: 0.323442
# Performance on non-restrictive human-specific enhancers and non-enhancers
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/predictNewSequences.py --architectureFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsNonCerebrumMouseTissuePeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_nonCerebrumMouseTissueNeg_500bp_conv5MuchMoreFiltNoInterPoolVeryHighMomL2SmallBatchPretrainBal_architecture.json --weightsFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsNonCerebrumMouseTissuePeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_nonCerebrumMouseTissueNeg_500bp_conv5MuchMoreFiltNoInterPoolVeryHighMomL2SmallBatchPretrainBal.hdf5 --narrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/HumanDNase/PMC/cromwell-executions/atac/45c4bd8c-c055-4cea-80c9-5ec0eeaf8024/call-reproducibility_idr/execution/optimal_peak.inStriatum_nonCDS_enhancerShort_nonMacaqueBrain_mouseTest.bed.gz --negNarrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MacaqueAtac/OfM/peak/idr_reproducibility/idr.optimal_peak.inM1AndStriatum_nonCDS_enhancerShort_hg38_summitExtendedMin50Max2XProtect5_nonHumanBrain_mouseTest.bed.gz --sequenceLength 500 --genomeFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/HumanGenome/hg38.fa --RC --classification --path /home/ikaplow/RegulatoryElementEvolutionProject/src/deeplearning
# Performance: loss: 0.843973, accuracy: 0.511364, sensitivity: 0.772727, specificity: 0.424242, auc: 0.705234, precision: 0.309091, NPV: 0.848485, auprc: 0.399547, auprg: 0.555997, F1: 0.441558, recallAt0.05FDR: 0.000000, recallAt0.1FDR: 0.000000
# Performance on non-restrictive macaque-specific enhancers and non-enhancers
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/predictNewSequences.py --architectureFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsNonCerebrumMouseTissuePeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_nonCerebrumMouseTissueNeg_500bp_conv5MuchMoreFiltNoInterPoolVeryHighMomL2SmallBatchPretrainBal_architecture.json --weightsFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsNonCerebrumMouseTissuePeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_nonCerebrumMouseTissueNeg_500bp_conv5MuchMoreFiltNoInterPoolVeryHighMomL2SmallBatchPretrainBal.hdf5 --narrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MacaqueAtac/OfM/peak/idr_reproducibility/idr.optimal_peak.inM1AndStriatum_nonCDS_enhancerShort_nonHumanBrain_mouseTest.bed.gz --negNarrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/HumanDNase/PMC/cromwell-executions/atac/45c4bd8c-c055-4cea-80c9-5ec0eeaf8024/call-reproducibility_idr/execution/optimal_peak.inStriatum_nonCDS_enhancerShort_rheMac8_summitExtendedMin50Max2XProtect5_UCSCNames_nonMacaqueBrain_mouseTest.bed.gz --sequenceLength 500 --genomeFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MacaqueGenome/rheMac8.fa --RC --classification --path /home/ikaplow/RegulatoryElementEvolutionProject/src/deeplearning
# Performance: loss: 0.767077, accuracy: 0.594697, sensitivity: 0.646465, specificity: 0.439394, auc: 0.570095, precision: 0.775758, NPV: 0.292929, auprc: 0.318964, auprg: 0.179591, F1: 0.705234, recallAt0.05FDR: 0.015152, recallAt0.1FDR: 0.015152
# Performance on shared brain and liver enhancers and liver enhancers that are not brain enhancers
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/predictNewSequences.py --architectureFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsNonCerebrumMouseTissuePeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_nonCerebrumMouseTissueNeg_500bp_conv5MuchMoreFiltNoInterPoolVeryHighMomL2SmallBatchPretrainBal_architecture.json --weightsFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsNonCerebrumMouseTissuePeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_nonCerebrumMouseTissueNeg_500bp_conv5MuchMoreFiltNoInterPoolVeryHighMomL2SmallBatchPretrainBal.hdf5 --narrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_andLiverLiuPfenning.narrowPeak.gz --negNarrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/LiverPfenning_ATAC_out/atac/68eb27d1-23c7-4d42-80f9-b3dd46f5e18c/call-reproducibility_idr/execution/idr.optimal_peak.narrowPeak_inLiuAll_nonCDS_enhancerShort_nonBrain.narrowPeak.gz --sequenceLength 500 --genomeFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseGenome/mm10.fa  --chroms chr1 --chroms chr2 --RC --classification --path /home/ikaplow/RegulatoryElementEvolutionProject/src/deeplearning
# Performance: loss: 0.224683, accuracy: 0.910674, sensitivity: 0.713462, specificity: 0.971319, auc: 0.957793, precision: 0.884386, NPV: 0.916829, auprc: 0.897616, auprg: 0.975242, F1: 0.789782, recallAt0.05FDR: 0.494231, recallAt0.1FDR: 0.666346
# Performance on shared brain and liver enhancers and liver enhancers that are not brain enhancers, calibrated
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/predictNewSequences.py --architectureFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsNonCerebrumMouseTissuePeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_nonCerebrumMouseTissueNeg_500bp_conv5MuchMoreFiltNoInterPoolVeryHighMomL2SmallBatchPretrainBal_architecture.json --weightsFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsNonCerebrumMouseTissuePeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_nonCerebrumMouseTissueNeg_500bp_conv5MuchMoreFiltNoInterPoolVeryHighMomL2SmallBatchPretrainBal.hdf5 --narrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_andLiverLiuPfenning.narrowPeak.gz --negNarrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/LiverPfenning_ATAC_out/atac/68eb27d1-23c7-4d42-80f9-b3dd46f5e18c/call-reproducibility_idr/execution/idr.optimal_peak.narrowPeak_inLiuAll_nonCDS_enhancerShort_nonBrain.narrowPeak.gz --sequenceLength 500 --genomeFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseGenome/mm10.fa  --chroms chr1 --chroms chr2 --RC --classification --path /home/ikaplow/RegulatoryElementEvolutionProject/src/deeplearning --recalibrateModel --calibrationXTrFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerVsOtherEuarchontaglireEnhLoosePeakLessThan1kb500bp_data_train.npy --calibrationYTrFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerVsOtherEuarchontaglireEnhLoosePeakLessThan1kb500bp_labels_train.npy
# Performance: loss: 0.258791, accuracy: 0.907734, sensitivity: 0.686538, specificity: 0.975754, auc: 0.957793, precision: 0.896985, NPV: 0.910094, auprc: 0.897616, auprg: 0.975242, F1: 0.777778, recallAt0.05FDR: 0.494231, recallAt0.1FDR: 0.666346
# Predictions for brain only enhancers, shared brain and liver enhancers, liver only enhancers, and negative set
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/predictNewSequencesNoEvaluation.py --architectureFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsNonCerebrumMouseTissuePeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_nonCerebrumMouseTissueNeg_500bp_conv5MuchMoreFiltNoInterPoolVeryHighMomL2SmallBatchPretrainBal_architecture.json --weightsFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsNonCerebrumMouseTissuePeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_nonCerebrumMouseTissueNeg_500bp_conv5MuchMoreFiltNoInterPoolVeryHighMomL2SmallBatchPretrainBal.hdf5 --narrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_nonLiver.narrowPeak.gz --sequenceLength 500 --predictedProbaFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_nonLiver_test_nonCerebrumMouseTissueNeg_predictedProba.txt --genomeFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseGenome/mm10.fa --chromSizesFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseGenome/mm10.chrom.sizes --chroms chr1 --chroms chr2 --classification
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/predictNewSequencesNoEvaluation.py --architectureFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsNonCerebrumMouseTissuePeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_nonCerebrumMouseTissueNeg_500bp_conv5MuchMoreFiltNoInterPoolVeryHighMomL2SmallBatchPretrainBal_architecture.json --weightsFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsNonCerebrumMouseTissuePeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_nonCerebrumMouseTissueNeg_500bp_conv5MuchMoreFiltNoInterPoolVeryHighMomL2SmallBatchPretrainBal.hdf5 --narrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_andLiverLiuPfenning.narrowPeak.gz --sequenceLength 500 --predictedProbaFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_andLiverLiuPfenning_test_nonCerebrumMouseTissueNeg_predictedProba.txt --genomeFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseGenome/mm10.fa --chromSizesFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseGenome/mm10.chrom.sizes --chroms chr1 --chroms chr2 --classification
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/predictNewSequencesNoEvaluation.py --architectureFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsNonCerebrumMouseTissuePeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_nonCerebrumMouseTissueNeg_500bp_conv5MuchMoreFiltNoInterPoolVeryHighMomL2SmallBatchPretrainBal_architecture.json --weightsFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsNonCerebrumMouseTissuePeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_nonCerebrumMouseTissueNeg_500bp_conv5MuchMoreFiltNoInterPoolVeryHighMomL2SmallBatchPretrainBal.hdf5 --narrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/LiverPfenning_ATAC_out/atac/68eb27d1-23c7-4d42-80f9-b3dd46f5e18c/call-reproducibility_idr/execution/idr.optimal_peak.narrowPeak_inLiuAll_nonCDS_enhancerShort_nonBrain.narrowPeak.gz --sequenceLength 500 --predictedProbaFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/LiverPfenning_ATAC_out/atac/68eb27d1-23c7-4d42-80f9-b3dd46f5e18c/call-reproducibility_idr/execution/idr.optimal_peak.narrowPeak_inLiuAll_nonCDS_enhancerShort_nonBrain_test_nonCerebrumMouseTissueNeg_predictedProba.txt --genomeFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseGenome/mm10.fa --chromSizesFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseGenome/mm10.chrom.sizes --chroms chr1 --chroms chr2 --classification
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/predictNewSequencesNoEvaluation.py --architectureFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsNonCerebrumMouseTissuePeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_nonCerebrumMouseTissueNeg_500bp_conv5MuchMoreFiltNoInterPoolVeryHighMomL2SmallBatchPretrainBal_architecture.json --weightsFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsNonCerebrumMouseTissuePeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_nonCerebrumMouseTissueNeg_500bp_conv5MuchMoreFiltNoInterPoolVeryHighMomL2SmallBatchPretrainBal.hdf5 --narrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/LiverPfenning_ATAC_out/atac/68eb27d1-23c7-4d42-80f9-b3dd46f5e18c/call-reproducibility_idr/execution/idr.optimal_peak_nonCDS_enhancerShort_andAllLiuAndAllEncode_nonMouseCerebrum.narrowPeak.gz --sequenceLength 500 --predictedProbaFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/LiverPfenning_ATAC_out/atac/68eb27d1-23c7-4d42-80f9-b3dd46f5e18c/call-reproducibility_idr/execution/idr.optimal_peak_nonCDS_enhancerShort_andAllLiuAndAllEncode_nonMouseCerebrum_test_predictedProba.txt --genomeFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseGenome/mm10.fa --chromSizesFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseGenome/mm10.chrom.sizes --chroms chr1 --chroms chr2 --classification
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/makeViolinPlotTissueComparison.py --tissueOnlyPredFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_nonLiver_test_flankNeg_predictedProba.txt --tissueSharedPredFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_andLiverLiuPfenning_test_flankNeg_predictedProba.txt --tissueOtherPredFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/LiverPfenning_ATAC_out/atac/68eb27d1-23c7-4d42-80f9-b3dd46f5e18c/call-reproducibility_idr/execution/idr.optimal_peak.narrowPeak_inLiuAll_nonCDS_enhancerShort_nonBrain_test_flankNeg_predictedProba.txt --negSetPredFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/LiverPfenning_ATAC_out/atac/68eb27d1-23c7-4d42-80f9-b3dd46f5e18c/call-reproducibility_idr/execution/idr.optimal_peak_nonCDS_enhancerShort_andAllLiuAndAllEncode_nonMouseCerebrum_test_predictedProba.txt --outputFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_brainNonLiverVsBrainLiverVsLiverNonBrainVsNonBrainVsNonCerebrumMouseTissueNeg_predictedProba.svg
# Performance on macaque shared brain and liver enhancers and liver enhancers that are not brain enhancers
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/predictNewSequences.py --architectureFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsNonCerebrumMouseTissuePeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_nonCerebrumMouseTissueNeg_500bp_conv5MuchMoreFiltNoInterPoolVeryHighMomL2SmallBatchPretrainBal_architecture.json --weightsFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsNonCerebrumMouseTissuePeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_nonCerebrumMouseTissueNeg_500bp_conv5MuchMoreFiltNoInterPoolVeryHighMomL2SmallBatchPretrainBal.hdf5 --narrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MacaqueAtac/OfM/peak/idr_reproducibility/idr.optimal_peak.inM1AndStriatum_nonCDS_enhancerShort_andLiver_mouseTest.bed.gz --negNarrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MacaqueAtac/Liver/cromwell-executions/atac/1f5fd9f0-d389-4184-99cb-c1125fd7f064/call-reproducibility_idr/execution/optimal_peak_nonCDS_enhancerShort_mouseTest_nonMacaqueBrain.bed.gz --sequenceLength 500 --genomeFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MacaqueGenome/rheMac8.fa --chroms chr1 --chroms chr2 --chroms chr8 --chroms chr9 --chroms chr3 --chroms chr4 --chroms chr5 --chroms chr6 --chroms chr7 --chroms chr10 --chroms chr11 --chroms chr12 --chroms chr13 --chroms chr14 --chroms chr15 --chroms chr16 --chroms chr17 --chroms chr18 --chroms chr19 --chroms chr20 --chroms chrX --chroms chrY --RC --classification --path /home/ikaplow/RegulatoryElementEvolutionProject/src/deeplearning
# Performance: loss: 0.281126, accuracy: 0.876732, sensitivity: 0.695035, specificity: 0.957806, auc: 0.949701, precision: 0.880240, NPV: 0.875603, auprc: 0.891824, auprg: 0.950689, F1: 0.776750, recallAt0.05FDR: 0.408983, recallAt0.1FDR: 0.661939
# Performance on macaque shared brain and liver enhancers and liver enhancers that are not brain enhancers, calibrated
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/predictNewSequences.py --architectureFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsNonCerebrumMouseTissuePeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_nonCerebrumMouseTissueNeg_500bp_conv5MuchMoreFiltNoInterPoolVeryHighMomL2SmallBatchPretrainBal_architecture.json --weightsFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsNonCerebrumMouseTissuePeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_nonCerebrumMouseTissueNeg_500bp_conv5MuchMoreFiltNoInterPoolVeryHighMomL2SmallBatchPretrainBal.hdf5 --narrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MacaqueAtac/OfM/peak/idr_reproducibility/idr.optimal_peak.inM1AndStriatum_nonCDS_enhancerShort_andLiver_mouseTest.bed.gz --negNarrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MacaqueAtac/Liver/cromwell-executions/atac/1f5fd9f0-d389-4184-99cb-c1125fd7f064/call-reproducibility_idr/execution/optimal_peak_nonCDS_enhancerShort_mouseTest_nonMacaqueBrain.bed.gz --sequenceLength 500 --genomeFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MacaqueGenome/rheMac8.fa --chroms chr1 --chroms chr2 --chroms chr8 --chroms chr9 --chroms chr3 --chroms chr4 --chroms chr5 --chroms chr6 --chroms chr7 --chroms chr10 --chroms chr11 --chroms chr12 --chroms chr13 --chroms chr14 --chroms chr15 --chroms chr16 --chroms chr17 --chroms chr18 --chroms chr19 --chroms chr20 --chroms chrX --chroms chrY --RC --classification --path /home/ikaplow/RegulatoryElementEvolutionProject/src/deeplearning --recalibrateModel --calibrationXTrFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerVsOtherEuarchontaglireEnhLoosePeakLessThan1kb500bp_data_train.npy --calibrationYTrFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerVsOtherEuarchontaglireEnhLoosePeakLessThan1kb500bp_labels_train.npy
# Performance: loss: 0.316572, accuracy: 0.875274, sensitivity: 0.671395, specificity: 0.966245, auc: 0.949701, precision: 0.898734, NPV: 0.868246, auprc: 0.891824, auprg: 0.950689, F1: 0.768606, recallAt0.05FDR: 0.408983, recallAt0.1FDR: 0.661939
# Performance on rat shared brain and liver enhancers and liver enhancers that are not brain enhancers
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/predictNewSequences.py --architectureFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsNonCerebrumMouseTissuePeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_nonCerebrumMouseTissueNeg_500bp_conv5MuchMoreFiltNoInterPoolVeryHighMomL2SmallBatchPretrainBal_architecture.json --weightsFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsNonCerebrumMouseTissuePeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_nonCerebrumMouseTissueNeg_500bp_conv5MuchMoreFiltNoInterPoolVeryHighMomL2SmallBatchPretrainBal.hdf5 --narrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/RatAtac/M1_AllReps/peak/idr_reproducibility/idr.optimal_peak.inStriatum_nonCDS_enhancerShort_andLiver_mouseTest.bed.gz --negNarrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/RatAtac/Liver_GoodReps/peak/idr_reproducibility/idr.optimal_peak_nonCDS_enhancerShort_nonRatBrain_mouseTest.bed.gz --sequenceLength 500 --genomeFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/RatGenome/rn6.fa --chroms chr1 --chroms chr2 --chroms chr8 --chroms chr9 --chroms chr3 --chroms chr4 --chroms chr5 --chroms chr6 --chroms chr7 --chroms chr10 --chroms chr11 --chroms chr12 --chroms chr13 --chroms chr14 --chroms chr15 --chroms chr16 --chroms chr17 --chroms chr18 --chroms chr19 --chroms chr20 --chroms chrX --chroms chrY --RC --classification
# Performance: loss: 0.350298, accuracy: 0.843230, sensitivity: 0.607345, specificity: 0.961910, auc: 0.926294, precision: 0.889165, NPV: 0.829615, auprc: 0.874879, auprg: 0.924350, F1: 0.721719, recallAt0.05FDR: 0.388136, recallAt0.1FDR: 0.583051
# Performance on rat shared brain and liver enhancers and liver enhancers that are not brain enhancers, calibrated
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/predictNewSequences.py --architectureFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsNonCerebrumMouseTissuePeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_nonCerebrumMouseTissueNeg_500bp_conv5MuchMoreFiltNoInterPoolVeryHighMomL2SmallBatchPretrainBal_architecture.json --weightsFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsNonCerebrumMouseTissuePeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_nonCerebrumMouseTissueNeg_500bp_conv5MuchMoreFiltNoInterPoolVeryHighMomL2SmallBatchPretrainBal.hdf5 --narrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/RatAtac/M1_AllReps/peak/idr_reproducibility/idr.optimal_peak.inStriatum_nonCDS_enhancerShort_andLiver_mouseTest.bed.gz --negNarrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/RatAtac/Liver_GoodReps/peak/idr_reproducibility/idr.optimal_peak_nonCDS_enhancerShort_nonRatBrain_mouseTest.bed.gz --sequenceLength 500 --genomeFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/RatGenome/rn6.fa --chroms chr1 --chroms chr2 --chroms chr8 --chroms chr9 --chroms chr3 --chroms chr4 --chroms chr5 --chroms chr6 --chroms chr7 --chroms chr10 --chroms chr11 --chroms chr12 --chroms chr13 --chroms chr14 --chroms chr15 --chroms chr16 --chroms chr17 --chroms chr18 --chroms chr19 --chroms chr20 --chroms chrX --chroms chrY --RC --classification --recalibrateModel --calibrationXTrFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerVsOtherEuarchontaglireEnhLoosePeakLessThan1kb500bp_data_train.npy --calibrationYTrFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerVsOtherEuarchontaglireEnhLoosePeakLessThan1kb500bp_labels_train.npy
# Performance: loss: 0.379812, accuracy: 0.837368, sensitivity: 0.577966, specificity: 0.967879, auc: 0.926294, precision: 0.900528, NPV: 0.820087, auprc: 0.874879, auprg: 0.924350, F1: 0.704061, recallAt0.05FDR: 0.388136, recallAt0.1FDR: 0.583051
# Get brain test set predictions across Zoonomia mammals
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/makePredictNewSequencesNoEvaluationScript.py --architectureFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsNonCerebrumMouseTissuePeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_nonCerebrumMouseTissueNeg_500bp_conv5MuchMoreFiltNoInterPoolVeryHighMomL2SmallBatchPretrainBal_architecture.json --weightsFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsNonCerebrumMouseTissuePeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_nonCerebrumMouseTissueNeg_500bp_conv5MuchMoreFiltNoInterPoolVeryHighMomL2SmallBatchPretrainBal.hdf5 --narrowPeakFileNameListFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_orLiver_LiftedPeaks/liftedPeakMouseTestNarrowPeakFileNamesFiltConverted.txt --genomeFileNameListFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/200MammalsFastas/200MammalsAssembliesFileNamesFilt.txt --chromSizesFileNameListFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/200MammalsFastas/chromSizesFileNames.txt --classification --scriptFileName /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/predictMouseBrainTestHugeCactusNonCerebrumTissueScript.sh --predictedClassesFileNameSuffix nonCerebrumMouseTissueNeg_predictedClasses.txt --predictedProbaFileNameSuffix nonCerebrumMouseTissueNeg_predictedProba.txt
sh /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/predictMouseBrainTestHugeCactusNonCerebrumTissueScript.sh 2> /home/ikaplow/RegulatoryElementEvolutionProject/src/predictMouseBrainTestHugeCactusNonCerebrumTissueScript.e
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/gatherPeakPredictionsAcrossSpecies.py --predictionsFileNameListFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/testPredictionsNonCerebrumTissueGliresFileNames.txt --baseSpeciesLine 35 --outputFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_test_nonCerebrumMouseTissueNeg_peakPredictionsGlires.txt # Filtered correlation (Pearson, Spearman): -0.9213*, -0.7875*

# Performance of brain model with 10X G/C, repeat-matched negatives on test set
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/predictNewSequences.py --architectureFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsRandomGCRepeat10XPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_RandomGCRepeat10XNeg_500bp_conv5MostFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal_architecture.json --weightsFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsRandomGCRepeat10XPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_RandomGCRepeat10XNeg_500bp_conv5MostFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal.hdf5 --narrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort.narrowPeak.gz --negNarrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_summitPlusMinus250bp_randomGCRepeatMatched10x_nonMouseBrain.narrowPeak.gz --sequenceLength 500 --genomeFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseGenome/mm10.fa --chroms chr1 --chroms chr2 --predictedProbaFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_test_andNeg_RandomGCRepeat10XNeg_conv5MuchMuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal_predictedProba.txt --RC --classification --path /home/ikaplow/RegulatoryElementEvolutionProject/src/deeplearning
# Performance: loss: nan, accuracy: 0.953204, sensitivity: 0.735358, specificity: 0.984348, auc: 0.969346, precision: 0.870409, NPV: 0.962987, auprc: 0.884410, auprg: 0.993019, F1: 0.797204, recallAt0.05FDR: 0.604458, recallAt0.1FDR: 0.693619
# Performance on mouse enhancers whose orthologs are not brain enhancers in at least one other species and negative set
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/predictNewSequences.py --architectureFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsRandomGCRepeat10XPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_RandomGCRepeat10XNeg_500bp_conv5MostFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal_architecture.json --weightsFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsRandomGCRepeat10XPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_RandomGCRepeat10XNeg_500bp_conv5MostFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal.hdf5 --narrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_withPeakNames_nonBrainOtherSpecies.narrowPeak.gz --negNarrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/HumanDNase/PMC/cromwell-executions/atac/45c4bd8c-c055-4cea-80c9-5ec0eeaf8024/call-reproducibility_idr/execution/optimal_peak.inStriatum_nonCDS_enhancerShort_mm10_summitExtendedMin50Max2XProtect5_nonMouseBrain_andMacaque_andRat.narrowPeak.gz --sequenceLength 500 --genomeFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseGenome/mm10.fa --chroms chr1 --chroms chr2 --RC --classification --path /home/ikaplow/RegulatoryElementEvolutionProject/src/deeplearning
# Performance: loss: nan, accuracy: 0.804791, sensitivity: 0.720382, specificity: 0.869059, auc: 0.879997, precision: 0.807281, NPV: 0.803227, auprc: 0.838553, auprg: 0.821076, F1: 0.761360, recallAt0.05FDR: 0.001274, recallAt0.1FDR: 0.481529
# Performance on human enhancers whose orthologs are not brain enhancers in mouse and negative set corresponding to non-enhancer orthologs of mouse enhancers
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/predictNewSequences.py --architectureFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsRandomGCRepeat10XPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_RandomGCRepeat10XNeg_500bp_conv5MostFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal_architecture.json --weightsFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsRandomGCRepeat10XPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_RandomGCRepeat10XNeg_500bp_conv5MostFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal.hdf5 --narrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/HumanDNase/PMC/cromwell-executions/atac/45c4bd8c-c055-4cea-80c9-5ec0eeaf8024/call-reproducibility_idr/execution/optimal_peak.inStriatum_nonCDS_enhancerShort_nonMouseBrain_mouseTest.bed.gz --negNarrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_withPeakNames_hg38Huge_summitExtendedMin50Max2Protect5_nonHumanBrain_mouseTest.narrowPeak.gz --sequenceLength 500 --genomeFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/HumanGenome/hg38.fa --RC --classification --path /home/ikaplow/RegulatoryElementEvolutionProject/src/deeplearning
# Performance: loss: inf, accuracy: 0.740091, sensitivity: 0.608173, specificity: 0.801339, auc: 0.781365, precision: 0.587007, NPV: 0.814983, auprc: 0.662617, auprg: 0.705152, F1: 0.597403, recallAt0.05FDR: 0.108173, recallAt0.1FDR: 0.252404
# Performance on macaque enhancers whose orthologs are not brain enhancers in mouse and negative set corresponding to non-enhancer orthologs of mouse enhancers
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/predictNewSequences.py --architectureFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsRandomGCRepeat10XPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_RandomGCRepeat10XNeg_500bp_conv5MostFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal_architecture.json --weightsFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsRandomGCRepeat10XPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_RandomGCRepeat10XNeg_500bp_conv5MostFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal.hdf5 --narrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MacaqueAtac/OfM/peak/idr_reproducibility/idr.optimal_peak.inM1AndStriatum_nonCDS_enhancerShort_nonMouseBrain_mouseTest.bed.gz --negNarrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_withPeakNames_rheMac8Huge_summitExtendedMin50Max2Protect5_nonMacaqueBrain_mouseTest.bed.gz --sequenceLength 500 --genomeFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MacaqueGenome/rheMac8.fa --chroms chr1 --chroms chr2 --chroms chr8 --chroms chr9 --chroms chr3 --chroms chr4 --chroms chr5 --chroms chr6 --chroms chr7 --chroms chr10 --chroms chr11 --chroms chr12 --chroms chr13 --chroms chr14 --chroms chr15 --chroms chr16 --chroms chr17 --chroms chr18 --chroms chr19 --chroms chr20 --chroms chr21 --chroms chr22 --chroms chrX --chroms chrY --RC --classification --path /home/ikaplow/RegulatoryElementEvolutionProject/src/deeplearning
# Performance: loss: nan, accuracy: 0.807642, sensitivity: 0.762943, specificity: 0.849490, auc: 0.887774, precision: 0.825959, NPV: 0.792857, auprc: 0.878157, auprg: 0.802029, F1: 0.793201, recallAt0.05FDR: 0.363760, recallAt0.1FDR: 0.543597
# Performance on rat enhancers whose orthologs are not brain enhancers in mouse and negative set corresponding to non-enhancer orthologs of mouse enhancers
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/predictNewSequences.py --architectureFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsRandomGCRepeat10XPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_RandomGCRepeat10XNeg_500bp_conv5MostFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal_architecture.json --weightsFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsRandomGCRepeat10XPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_RandomGCRepeat10XNeg_500bp_conv5MostFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal.hdf5 --narrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/RatAtac/M1_AllReps/peak/idr_reproducibility/idr.optimal_peak.inStriatum_nonCDS_enhancerShort_mouseTest_nonMouseBrain.bed.gz --negNarrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_withPeakNames_rn6Huge_summitExtendedMin50Max2Protect5_nonRatBrain_mouseTest.bed.gz --sequenceLength 500 --genomeFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/RatGenome/rn6.fa --chroms chr1 --chroms chr2 --chroms chr8 --chroms chr9 --chroms chr3 --chroms chr4 --chroms chr5 --chroms chr6 --chroms chr7 --chroms chr10 --chroms chr11 --chroms chr12 --chroms chr13 --chroms chr14 --chroms chr15 --chroms chr16 --chroms chr17 --chroms chr18 --chroms chr19 --chroms chr20 --chroms chr21 --chroms chr22 --chroms chrX --chroms chrY --RC --classification --path /home/ikaplow/RegulatoryElementEvolutionProject/src/deeplearning
# Performance: loss: nan, accuracy: 0.622762, sensitivity: 0.562921, specificity: 0.701780, auc: 0.705905, precision: 0.713675, NPV: 0.548724, auprc: 0.659630, auprg: 0.430909, F1: 0.629397, recallAt0.05FDR: 0.028190, recallAt0.1FDR: 0.094955
# Performance on non-restrictive glires-specific enhancers and non-enhancers
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/predictNewSequences.py --architectureFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsRandomGCRepeat10XPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_RandomGCRepeat10XNeg_500bp_conv5MostFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal_architecture.json --weightsFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsRandomGCRepeat10XPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_RandomGCRepeat10XNeg_500bp_conv5MostFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal.hdf5 --narrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_inRatBrainLoose_nonHumanBrain_nonMacaqueBrain.bed.gz --negNarrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/HumanDNase/PMC/cromwell-executions/atac/45c4bd8c-c055-4cea-80c9-5ec0eeaf8024/call-reproducibility_idr/execution/optimal_peak.inStriatum_nonCDS_enhancerShort_mm10_summitExtendedMin50Max2XProtect5_inMacaqueBrainLoose_nonMouseBrain_nonRatBrain.bed.gz --sequenceLength 500 --genomeFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseGenome/mm10.fa --chroms chr1 --chroms chr2 --RC --classification --path /home/ikaplow/RegulatoryElementEvolutionProject/src/deeplearning
# Performance: loss: 0.504666, accuracy: 0.796703, sensitivity: 0.708696, specificity: 0.947761, auc: 0.921350, precision: 0.958824, NPV: 0.654639, auprc: 0.868279, auprg: 0.885648, F1: 0.815000, recallAt0.05FDR: 0.320896, recallAt0.1FDR: 0.447761
# Performance on non-restrictive glires-specific enhancers and non-enhancers, calibrated
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/predictNewSequences.py --architectureFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsRandomGCRepeat10XPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_RandomGCRepeat10XNeg_500bp_conv5MostFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal_architecture.json --weightsFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsRandomGCRepeat10XPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_RandomGCRepeat10XNeg_500bp_conv5MostFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal.hdf5 --narrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_inRatBrainLoose_nonHumanBrain_nonMacaqueBrain.bed.gz --negNarrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/HumanDNase/PMC/cromwell-executions/atac/45c4bd8c-c055-4cea-80c9-5ec0eeaf8024/call-reproducibility_idr/execution/optimal_peak.inStriatum_nonCDS_enhancerShort_mm10_summitExtendedMin50Max2XProtect5_inMacaqueBrainLoose_nonMouseBrain_nonRatBrain.bed.gz --sequenceLength 500 --genomeFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseGenome/mm10.fa --chroms chr1 --chroms chr2 --RC --classification --path /home/ikaplow/RegulatoryElementEvolutionProject/src/deeplearning --recalibrateModel --calibrationXTrFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerVsOtherEuarchontaglireEnhLoosePeakLessThan1kb500bp_data_train.npy --calibrationYTrFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerVsOtherEuarchontaglireEnhLoosePeakLessThan1kb500bp_labels_train.npy
# Performance: loss: 0.500048, accuracy: 0.796703, sensitivity: 0.700000, specificity: 0.962687, auc: 0.921350, precision: 0.969880, NPV: 0.651515, auprc: 0.868279, auprg: 0.885648, F1: 0.813131, recallAt0.05FDR: 0.320896, recallAt0.1FDR: 0.447761
# Performance on non-restrictive euarchonta-specific enhancers and non-enhancers
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/predictNewSequences.py --architectureFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsRandomGCRepeat10XPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_RandomGCRepeat10XNeg_500bp_conv5MostFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal_architecture.json --weightsFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsRandomGCRepeat10XPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_RandomGCRepeat10XNeg_500bp_conv5MostFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal.hdf5 --narrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/HumanDNase/PMC/cromwell-executions/atac/45c4bd8c-c055-4cea-80c9-5ec0eeaf8024/call-reproducibility_idr/execution/optimal_peak.inStriatum_nonCDS_enhancerShort_inMacaqueBrainLoose_nonMouseBrain_nonRatBrain_mouseTest.bed.gz --negNarrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_withPeakNames_hg38Huge_summitExtendedMin50Max2Protect5_mouseTest_inRatBrainLoose_nonHumanBrain_nonMacaqueBrain.bed.gz --sequenceLength 500 --genomeFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/HumanGenome/hg38.fa --RC --classification --path /home/ikaplow/RegulatoryElementEvolutionProject/src/deeplearning
# Performance: loss: 0.602982, accuracy: 0.755495, sensitivity: 0.604478, specificity: 0.843478, auc: 0.797923, precision: 0.692308, NPV: 0.785425, auprc: 0.754950, auprg: 0.716105, F1: 0.645418, recallAt0.05FDR: 0.291045, recallAt0.1FDR: 0.343284
# Performance on non-restrictive euarchonta-specific enhancers and non-enhancers, calibrated
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/predictNewSequences.py --architectureFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsRandomGCRepeat10XPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_RandomGCRepeat10XNeg_500bp_conv5MostFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal_architecture.json --weightsFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsRandomGCRepeat10XPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_RandomGCRepeat10XNeg_500bp_conv5MostFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal.hdf5 --narrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/HumanDNase/PMC/cromwell-executions/atac/45c4bd8c-c055-4cea-80c9-5ec0eeaf8024/call-reproducibility_idr/execution/optimal_peak.inStriatum_nonCDS_enhancerShort_inMacaqueBrainLoose_nonMouseBrain_nonRatBrain_mouseTest.bed.gz --negNarrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_withPeakNames_hg38Huge_summitExtendedMin50Max2Protect5_mouseTest_inRatBrainLoose_nonHumanBrain_nonMacaqueBrain.bed.gz --sequenceLength 500 --genomeFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/HumanGenome/hg38.fa --RC --classification --path /home/ikaplow/RegulatoryElementEvolutionProject/src/deeplearning --recalibrateModel --calibrationXTrFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerVsOtherEuarchontaglireEnhLoosePeakLessThan1kb500bp_data_train.npy --calibrationYTrFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerVsOtherEuarchontaglireEnhLoosePeakLessThan1kb500bp_labels_train.npy
# Performance: loss: 0.569240, accuracy: 0.752747, sensitivity: 0.574627, specificity: 0.856522, auc: 0.797923, precision: 0.700000, NPV: 0.775591, auprc: 0.754950, auprg: 0.716105, F1: 0.631148, recallAt0.05FDR: 0.291045, recallAt0.1FDR: 0.343284
# Performance on non-restrictive mouse-specific enhancers and non-enhancers
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/predictNewSequences.py --architectureFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsRandomGCRepeat10XPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_RandomGCRepeat10XNeg_500bp_conv5MostFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal_architecture.json --weightsFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsRandomGCRepeat10XPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_RandomGCRepeat10XNeg_500bp_conv5MostFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal.hdf5 --narrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_withPeakNames_nonRatBrain.bed.gz --negNarrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/RatAtac/M1_AllReps/peak/idr_reproducibility/idr.optimal_peak.inStriatum_nonCDS_enhancerShort_mm10_summitExtendedMin50Max2XProtect5_nonMouseBrain.bed.gz --sequenceLength 500 --genomeFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseGenome/mm10.fa --chroms chr1 --chroms chr2 --RC --classification --path /home/ikaplow/RegulatoryElementEvolutionProject/src/deeplearning
# Performance: loss: 0.516246, accuracy: 0.783248, sensitivity: 0.716617, specificity: 0.833708, auc: 0.858299, precision: 0.765452, NPV: 0.795284, auprc: 0.817378, auprg: 0.778494, F1: 0.740230, recallAt0.05FDR: 0.114243, recallAt0.1FDR: 0.391691
# Performance on non-restrictive human-specific enhancers and non-enhancers
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/predictNewSequences.py --architectureFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsRandomGCRepeat10XPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_RandomGCRepeat10XNeg_500bp_conv5MostFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal_architecture.json --weightsFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsRandomGCRepeat10XPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_RandomGCRepeat10XNeg_500bp_conv5MostFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal.hdf5 --narrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/HumanDNase/PMC/cromwell-executions/atac/45c4bd8c-c055-4cea-80c9-5ec0eeaf8024/call-reproducibility_idr/execution/optimal_peak.inStriatum_nonCDS_enhancerShort_nonMacaqueBrain_mouseTest.bed.gz --negNarrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MacaqueAtac/OfM/peak/idr_reproducibility/idr.optimal_peak.inM1AndStriatum_nonCDS_enhancerShort_hg38_summitExtendedMin50Max2XProtect5_nonHumanBrain_mouseTest.bed.gz --sequenceLength 500 --genomeFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/HumanGenome/hg38.fa --RC --classification --path /home/ikaplow/RegulatoryElementEvolutionProject/src/deeplearning
# Performance: loss: inf, accuracy: 0.609848, sensitivity: 0.651515, specificity: 0.595960, auc: 0.647000, precision: 0.349593, NPV: 0.836879, auprc: 0.363393, auprg: 0.475261, F1: 0.455026, recallAt0.05FDR: 0.000000, recallAt0.1FDR: 0.000000
# Performance on non-restrictive macaque-specific enhancers and non-enhancers
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/predictNewSequences.py --architectureFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsRandomGCRepeat10XPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_RandomGCRepeat10XNeg_500bp_conv5MostFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal_architecture.json --weightsFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsRandomGCRepeat10XPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_RandomGCRepeat10XNeg_500bp_conv5MostFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal.hdf5 --narrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MacaqueAtac/OfM/peak/idr_reproducibility/idr.optimal_peak.inM1AndStriatum_nonCDS_enhancerShort_nonHumanBrain_mouseTest.bed.gz --negNarrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/HumanDNase/PMC/cromwell-executions/atac/45c4bd8c-c055-4cea-80c9-5ec0eeaf8024/call-reproducibility_idr/execution/optimal_peak.inStriatum_nonCDS_enhancerShort_rheMac8_summitExtendedMin50Max2XProtect5_UCSCNames_nonMacaqueBrain_mouseTest.bed.gz --sequenceLength 500 --genomeFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MacaqueGenome/rheMac8.fa --RC --classification --path /home/ikaplow/RegulatoryElementEvolutionProject/src/deeplearning
# Performance: loss: 0.764282, accuracy: 0.678030, sensitivity: 0.676768, specificity: 0.681818, auc: 0.740205, precision: 0.864516, NPV: 0.412844, auprc: 0.551899, auprg: 0.665096, F1: 0.759207, recallAt0.05FDR: 0.090909, recallAt0.1FDR: 0.090909
# Performance on shared brain and liver enhancers and liver enhancers that are not brain enhancers
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/predictNewSequences.py --architectureFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsRandomGCRepeat10XPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_RandomGCRepeat10XNeg_500bp_conv5MostFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal_architecture.json --weightsFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsRandomGCRepeat10XPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_RandomGCRepeat10XNeg_500bp_conv5MostFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal.hdf5 --narrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_andLiverLiuPfenning.narrowPeak.gz --negNarrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/LiverPfenning_ATAC_out/atac/68eb27d1-23c7-4d42-80f9-b3dd46f5e18c/call-reproducibility_idr/execution/idr.optimal_peak.narrowPeak_inLiuAll_nonCDS_enhancerShort_nonBrain.narrowPeak.gz --sequenceLength 500 --genomeFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseGenome/mm10.fa  --chroms chr1 --chroms chr2 --RC --classification --path /home/ikaplow/RegulatoryElementEvolutionProject/src/deeplearning
# Performance: loss: nan, accuracy: 0.912483, sensitivity: 0.797115, specificity: 0.947960, auc: 0.940664, precision: 0.824876, NPV: 0.938250, auprc: 0.895788, auprg: 0.973997, F1: 0.810758, recallAt0.05FDR: 0.684615, recallAt0.1FDR: 0.724038
# Performance on shared brain and liver enhancers and liver enhancers that are not brain enhancers, calibrated
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/predictNewSequences.py --architectureFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsRandomGCRepeat10XPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_RandomGCRepeat10XNeg_500bp_conv5MostFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal_architecture.json --weightsFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsRandomGCRepeat10XPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_RandomGCRepeat10XNeg_500bp_conv5MostFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal.hdf5 --narrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_andLiverLiuPfenning.narrowPeak.gz --negNarrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/LiverPfenning_ATAC_out/atac/68eb27d1-23c7-4d42-80f9-b3dd46f5e18c/call-reproducibility_idr/execution/idr.optimal_peak.narrowPeak_inLiuAll_nonCDS_enhancerShort_nonBrain.narrowPeak.gz --sequenceLength 500 --genomeFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseGenome/mm10.fa  --chroms chr1 --chroms chr2 --RC --classification --path /home/ikaplow/RegulatoryElementEvolutionProject/src/deeplearning --recalibrateModel --calibrationXTrFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerVsOtherEuarchontaglireEnhLoosePeakLessThan1kb500bp_data_train.npy --calibrationYTrFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerVsOtherEuarchontaglireEnhLoosePeakLessThan1kb500bp_labels_train.npy
# Performance: loss: 0.238057, accuracy: 0.912935, sensitivity: 0.791346, specificity: 0.950325, auc: 0.940664, precision: 0.830474, NPV: 0.936753, auprc: 0.895787, auprg: 0.973995, F1: 0.810438, recallAt0.05FDR: 0.684615, recallAt0.1FDR: 0.724038
# Predictions for brain only enhancers, shared brain and liver enhancers, liver only enhancers, and negative set
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/predictNewSequencesNoEvaluation.py --architectureFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsRandomGCRepeat10XPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_RandomGCRepeat10XNeg_500bp_conv5MostFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal_architecture.json --weightsFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsRandomGCRepeat10XPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_RandomGCRepeat10XNeg_500bp_conv5MostFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal.hdf5 --narrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_nonLiver.narrowPeak.gz --sequenceLength 500 --predictedProbaFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_nonLiver_test_RandomGCRepeat10XNeg_predictedProba.txt --genomeFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseGenome/mm10.fa --chromSizesFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseGenome/mm10.chrom.sizes --chroms chr1 --chroms chr2 --classification
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/predictNewSequencesNoEvaluation.py --architectureFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsRandomGCRepeat10XPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_RandomGCRepeat10XNeg_500bp_conv5MostFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal_architecture.json --weightsFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsRandomGCRepeat10XPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_RandomGCRepeat10XNeg_500bp_conv5MostFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal.hdf5 --narrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_andLiverLiuPfenning.narrowPeak.gz --sequenceLength 500 --predictedProbaFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_andLiverLiuPfenning_test_RandomGCRepeat10XNeg_predictedProba.txt --genomeFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseGenome/mm10.fa --chromSizesFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseGenome/mm10.chrom.sizes --chroms chr1 --chroms chr2 --classification
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/predictNewSequencesNoEvaluation.py --architectureFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsRandomGCRepeat10XPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_RandomGCRepeat10XNeg_500bp_conv5MostFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal_architecture.json --weightsFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsRandomGCRepeat10XPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_RandomGCRepeat10XNeg_500bp_conv5MostFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal.hdf5 --narrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/LiverPfenning_ATAC_out/atac/68eb27d1-23c7-4d42-80f9-b3dd46f5e18c/call-reproducibility_idr/execution/idr.optimal_peak.narrowPeak_inLiuAll_nonCDS_enhancerShort_nonBrain.narrowPeak.gz --sequenceLength 500 --predictedProbaFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/LiverPfenning_ATAC_out/atac/68eb27d1-23c7-4d42-80f9-b3dd46f5e18c/call-reproducibility_idr/execution/idr.optimal_peak.narrowPeak_inLiuAll_nonCDS_enhancerShort_nonBrain_test_RandomGCRepeat10XNeg_predictedProba.txt --genomeFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseGenome/mm10.fa --chromSizesFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseGenome/mm10.chrom.sizes --chroms chr1 --chroms chr2 --classification
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/predictNewSequencesNoEvaluation.py --architectureFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsRandomGCRepeat10XPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_RandomGCRepeat10XNeg_500bp_conv5MostFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal_architecture.json --weightsFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsRandomGCRepeat10XPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_RandomGCRepeat10XNeg_500bp_conv5MostFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal.hdf5 --narrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_summitPlusMinus250bp_randomGCRepeatMatched10x_nonMouseBrain.narrowPeak.gz --sequenceLength 500 --predictedProbaFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_summitPlusMinus250bp_randomGCRepeatMatched10x_nonMouseBrain_test_predictedProba.txt --genomeFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseGenome/mm10.fa --chromSizesFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseGenome/mm10.chrom.sizes --chroms chr1 --chroms chr2 --classification
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/makeViolinPlotTissueComparison.py --tissueOnlyPredFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_nonLiver_test_flankNeg_predictedProba.txt --tissueSharedPredFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_andLiverLiuPfenning_test_flankNeg_predictedProba.txt --tissueOtherPredFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/LiverPfenning_ATAC_out/atac/68eb27d1-23c7-4d42-80f9-b3dd46f5e18c/call-reproducibility_idr/execution/idr.optimal_peak.narrowPeak_inLiuAll_nonCDS_enhancerShort_nonBrain_test_flankNeg_predictedProba.txt --negSetPredFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_summitPlusMinus250bp_randomGCRepeatMatched10x_nonMouseBrain_test_predictedProba.txt --outputFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_brainNonLiverVsBrainLiverVsLiverNonBrainVsRandomGCRepeat10XNeg_predictedProba.svg
# Performance on macaque shared brain and liver enhancers and liver enhancers that are not brain enhancers
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/predictNewSequences.py --architectureFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsRandomGCRepeat10XPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_RandomGCRepeat10XNeg_500bp_conv5MostFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal_architecture.json --weightsFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsRandomGCRepeat10XPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_RandomGCRepeat10XNeg_500bp_conv5MostFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal.hdf5 --narrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MacaqueAtac/OfM/peak/idr_reproducibility/idr.optimal_peak.inM1AndStriatum_nonCDS_enhancerShort_andLiver_mouseTest.bed.gz --negNarrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MacaqueAtac/Liver/cromwell-executions/atac/1f5fd9f0-d389-4184-99cb-c1125fd7f064/call-reproducibility_idr/execution/optimal_peak_nonCDS_enhancerShort_mouseTest_nonMacaqueBrain.bed.gz --sequenceLength 500 --genomeFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MacaqueGenome/rheMac8.fa --chroms chr1 --chroms chr2 --chroms chr8 --chroms chr9 --chroms chr3 --chroms chr4 --chroms chr5 --chroms chr6 --chroms chr7 --chroms chr10 --chroms chr11 --chroms chr12 --chroms chr13 --chroms chr14 --chroms chr15 --chroms chr16 --chroms chr17 --chroms chr18 --chroms chr19 --chroms chr20 --chroms chrX --chroms chrY --RC --classification --path /home/ikaplow/RegulatoryElementEvolutionProject/src/deeplearning
# Performance: loss: nan, accuracy: 0.901167, sensitivity: 0.849882, specificity: 0.924051, auc: 0.954634, precision: 0.833140, NPV: 0.932411, auprc: 0.913337, auprg: 0.963474, F1: 0.841428, recallAt0.05FDR: 0.635934, recallAt0.1FDR: 0.778960
# Performance on macaque shared brain and liver enhancers and liver enhancers that are not brain enhancers, calibrated
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/predictNewSequences.py --architectureFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsRandomGCRepeat10XPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_RandomGCRepeat10XNeg_500bp_conv5MostFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal_architecture.json --weightsFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsRandomGCRepeat10XPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_RandomGCRepeat10XNeg_500bp_conv5MostFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal.hdf5 --narrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MacaqueAtac/OfM/peak/idr_reproducibility/idr.optimal_peak.inM1AndStriatum_nonCDS_enhancerShort_andLiver_mouseTest.bed.gz --negNarrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MacaqueAtac/Liver/cromwell-executions/atac/1f5fd9f0-d389-4184-99cb-c1125fd7f064/call-reproducibility_idr/execution/optimal_peak_nonCDS_enhancerShort_mouseTest_nonMacaqueBrain.bed.gz --sequenceLength 500 --genomeFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MacaqueGenome/rheMac8.fa --chroms chr1 --chroms chr2 --chroms chr8 --chroms chr9 --chroms chr3 --chroms chr4 --chroms chr5 --chroms chr6 --chroms chr7 --chroms chr10 --chroms chr11 --chroms chr12 --chroms chr13 --chroms chr14 --chroms chr15 --chroms chr16 --chroms chr17 --chroms chr18 --chroms chr19 --chroms chr20 --chroms chrX --chroms chrY --RC --classification --path /home/ikaplow/RegulatoryElementEvolutionProject/src/deeplearning --recalibrateModel --calibrationXTrFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerVsOtherEuarchontaglireEnhLoosePeakLessThan1kb500bp_data_train.npy --calibrationYTrFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerVsOtherEuarchontaglireEnhLoosePeakLessThan1kb500bp_labels_train.npy
# Performance: loss: 0.264169, accuracy: 0.901896, sensitivity: 0.840426, specificity: 0.929325, auc: 0.954634, precision: 0.841420, NPV: 0.928835, auprc: 0.913343, auprg: 0.963474, F1: 0.840923, recallAt0.05FDR: 0.635934, recallAt0.1FDR: 0.778960
# Performance on rat shared brain and liver enhancers and liver enhancers that are not brain enhancers
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/predictNewSequences.py --architectureFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsRandomGCRepeat10XPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_RandomGCRepeat10XNeg_500bp_conv5MostFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal_architecture.json --weightsFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsRandomGCRepeat10XPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_RandomGCRepeat10XNeg_500bp_conv5MostFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal.hdf5 --narrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/RatAtac/M1_AllReps/peak/idr_reproducibility/idr.optimal_peak.inStriatum_nonCDS_enhancerShort_andLiver_mouseTest.bed.gz --negNarrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/RatAtac/Liver_GoodReps/peak/idr_reproducibility/idr.optimal_peak_nonCDS_enhancerShort_nonRatBrain_mouseTest.bed.gz --sequenceLength 500 --genomeFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/RatGenome/rn6.fa --chroms chr1 --chroms chr2 --chroms chr8 --chroms chr9 --chroms chr3 --chroms chr4 --chroms chr5 --chroms chr6 --chroms chr7 --chroms chr10 --chroms chr11 --chroms chr12 --chroms chr13 --chroms chr14 --chroms chr15 --chroms chr16 --chroms chr17 --chroms chr18 --chroms chr19 --chroms chr20 --chroms chrX --chroms chrY --RC --classification
# Performance: loss: nan, accuracy: 0.852874, sensitivity: 0.714124, specificity: 0.922683, auc: 0.904155, precision: 0.822917, NPV: 0.865139, auprc: 0.864354, auprg: 0.917510, F1: 0.764670, recallAt0.05FDR: 0.527684, recallAt0.1FDR: 0.603955
# Performance on rat shared brain and liver enhancers and liver enhancers that are not brain enhancers, calibrated
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/predictNewSequences.py --architectureFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsRandomGCRepeat10XPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_RandomGCRepeat10XNeg_500bp_conv5MostFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal_architecture.json --weightsFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsRandomGCRepeat10XPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_RandomGCRepeat10XNeg_500bp_conv5MostFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal.hdf5 --narrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/RatAtac/M1_AllReps/peak/idr_reproducibility/idr.optimal_peak.inStriatum_nonCDS_enhancerShort_andLiver_mouseTest.bed.gz --negNarrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/RatAtac/Liver_GoodReps/peak/idr_reproducibility/idr.optimal_peak_nonCDS_enhancerShort_nonRatBrain_mouseTest.bed.gz --sequenceLength 500 --genomeFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/RatGenome/rn6.fa --chroms chr1 --chroms chr2 --chroms chr8 --chroms chr9 --chroms chr3 --chroms chr4 --chroms chr5 --chroms chr6 --chroms chr7 --chroms chr10 --chroms chr11 --chroms chr12 --chroms chr13 --chroms chr14 --chroms chr15 --chroms chr16 --chroms chr17 --chroms chr18 --chroms chr19 --chroms chr20 --chroms chrX --chroms chrY --RC --classification --recalibrateModel --calibrationXTrFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerVsOtherEuarchontaglireEnhLoosePeakLessThan1kb500bp_data_train.npy --calibrationYTrFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerVsOtherEuarchontaglireEnhLoosePeakLessThan1kb500bp_labels_train.npy
# Performance: loss: 0.371344, accuracy: 0.851740, sensitivity: 0.702825, specificity: 0.926663, auc: 0.904155, precision: 0.828229, NPV: 0.861067, auprc: 0.864354, auprg: 0.917510, F1: 0.760391, recallAt0.05FDR: 0.527684, recallAt0.1FDR: 0.603955
# Get brain test set predictions across Zoonomia mammals
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/makePredictNewSequencesNoEvaluationScript.py --architectureFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsRandomGCRepeat10XPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_RandomGCRepeat10XNeg_500bp_conv5MostFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal_architecture.json --weightsFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsRandomGCRepeat10XPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_RandomGCRepeat10XNeg_500bp_conv5MostFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal.hdf5 --narrowPeakFileNameListFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_orLiver_LiftedPeaks/liftedPeakMouseTestNarrowPeakFileNamesFiltConverted.txt --genomeFileNameListFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/200MammalsFastas/200MammalsAssembliesFileNamesFilt.txt --chromSizesFileNameListFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/200MammalsFastas/chromSizesFileNames.txt --classification --scriptFileName /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/predictMouseBrainTestHugeCactusRandomGC10XScript.sh --predictedClassesFileNameSuffix RandomGCRepeat10XNeg_predictedClasses.txt --predictedProbaFileNameSuffix RandomGCRepeat10XNeg_predictedProba.txt
sh /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/predictMouseBrainTestHugeCactusRandomGC10XScript.sh 2> /home/ikaplow/RegulatoryElementEvolutionProject/src/predictMouseBrainTestHugeCactusRandomGC10XScript.e
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/gatherPeakPredictionsAcrossSpecies.py --predictionsFileNameListFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/testPredictionsRandomGC10XGliresFileNames.txt --baseSpeciesLine 35 --outputFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_test_RandomGCRepeat10XNeg_peakPredictionsGlires.txt # Filtered correlation (Pearson, Spearman): -0.9487*, -0.7372

# Performance of brain model with 2X G/C, repeat-matched negatives on test set
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/predictNewSequences.py --architectureFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsRandomGCRepeat2XPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_RandomGCRepeatNonEnhRatioXNeg_500bp_conv5MuchMuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal_architecture.json --weightsFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsRandomGCRepeat2XPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_RandomGCRepeatNonEnhRatioXNeg_500bp_conv5MuchMuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal.hdf5 --narrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort.narrowPeak.gz --negNarrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_summitPlusMinus250bp_randomGCRepeatMatched2x_nonMouseBrain.narrowPeak.gz --sequenceLength 500 --genomeFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseGenome/mm10.fa --chroms chr1 --chroms chr2 --predictedProbaFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_test_andNeg_RandomGCRepeat2XNeg_conv5MuchMuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal_predictedProba.txt --RC --classification --path /home/ikaplow/RegulatoryElementEvolutionProject/src/deeplearning
# Performance: loss: nan, accuracy: 0.899377, sensitivity: 0.892264, specificity: 0.904268, auc: 0.964784, precision: 0.865042, NPV: 0.924270, auprc: 0.954690, auprg: 0.963244, F1: 0.878442, recallAt0.05FDR: 0.768575, recallAt0.1FDR: 0.850087
# Performance on mouse enhancers whose orthologs are not brain enhancers in at least one other species and negative set
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/predictNewSequences.py --architectureFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsRandomGCRepeat2XPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_RandomGCRepeatNonEnhRatioXNeg_500bp_conv5MuchMuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal_architecture.json --weightsFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsRandomGCRepeat2XPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_RandomGCRepeatNonEnhRatioXNeg_500bp_conv5MuchMuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal.hdf5 --narrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_withPeakNames_nonBrainOtherSpecies.narrowPeak.gz --negNarrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/HumanDNase/PMC/cromwell-executions/atac/45c4bd8c-c055-4cea-80c9-5ec0eeaf8024/call-reproducibility_idr/execution/optimal_peak.inStriatum_nonCDS_enhancerShort_mm10_summitExtendedMin50Max2XProtect5_nonMouseBrain_andMacaque_andRat.narrowPeak.gz --sequenceLength 500 --genomeFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseGenome/mm10.fa --chroms chr1 --chroms chr2 --RC --classification --path /home/ikaplow/RegulatoryElementEvolutionProject/src/deeplearning
# Performance: loss: nan, accuracy: 0.727148, sensitivity: 0.900000, specificity: 0.595538, auc: 0.855972, precision: 0.628838, NPV: 0.886643, auprc: 0.794482, auprg: 0.770383, F1: 0.740372, recallAt0.05FDR: 0.003185, recallAt0.1FDR: 0.149045
# Performance on human enhancers whose orthologs are not brain enhancers in mouse and negative set corresponding to non-enhancer orthologs of mouse enhancers
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/predictNewSequences.py --architectureFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsRandomGCRepeat2XPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_RandomGCRepeatNonEnhRatioXNeg_500bp_conv5MuchMuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal_architecture.json --weightsFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsRandomGCRepeat2XPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_RandomGCRepeatNonEnhRatioXNeg_500bp_conv5MuchMuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal.hdf5 --narrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/HumanDNase/PMC/cromwell-executions/atac/45c4bd8c-c055-4cea-80c9-5ec0eeaf8024/call-reproducibility_idr/execution/optimal_peak.inStriatum_nonCDS_enhancerShort_nonMouseBrain_mouseTest.bed.gz --negNarrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_withPeakNames_hg38Huge_summitExtendedMin50Max2Protect5_nonHumanBrain_mouseTest.narrowPeak.gz --sequenceLength 500 --genomeFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/HumanGenome/hg38.fa --RC --classification --path /home/ikaplow/RegulatoryElementEvolutionProject/src/deeplearning
# Performance: loss: inf, accuracy: 0.623476, sensitivity: 0.831731, specificity: 0.526786, auc: 0.762143, precision: 0.449351, NPV: 0.870849, auprc: 0.598298, auprg: 0.648986, F1: 0.583474, recallAt0.05FDR: 0.000000, recallAt0.1FDR: 0.000000
# Performance on macaque enhancers whose orthologs are not brain enhancers in mouse and negative set corresponding to non-enhancer orthologs of mouse enhancers
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/predictNewSequences.py --architectureFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsRandomGCRepeat2XPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_RandomGCRepeatNonEnhRatioXNeg_500bp_conv5MuchMuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal_architecture.json --weightsFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsRandomGCRepeat2XPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_RandomGCRepeatNonEnhRatioXNeg_500bp_conv5MuchMuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal.hdf5 --narrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MacaqueAtac/OfM/peak/idr_reproducibility/idr.optimal_peak.inM1AndStriatum_nonCDS_enhancerShort_nonMouseBrain_mouseTest.bed.gz --negNarrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_withPeakNames_rheMac8Huge_summitExtendedMin50Max2Protect5_nonMacaqueBrain_mouseTest.bed.gz --sequenceLength 500 --genomeFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MacaqueGenome/rheMac8.fa --chroms chr1 --chroms chr2 --chroms chr8 --chroms chr9 --chroms chr3 --chroms chr4 --chroms chr5 --chroms chr6 --chroms chr7 --chroms chr10 --chroms chr11 --chroms chr12 --chroms chr13 --chroms chr14 --chroms chr15 --chroms chr16 --chroms chr17 --chroms chr18 --chroms chr19 --chroms chr20 --chroms chr21 --chroms chr22 --chroms chrX --chroms chrY --RC --classification --path /home/ikaplow/RegulatoryElementEvolutionProject/src/deeplearning
# Performance: loss: nan, accuracy: 0.734519, sensitivity: 0.910082, specificity: 0.570153, auc: 0.863023, precision: 0.664677, NPV: 0.871345, auprc: 0.853627, auprg: 0.754225, F1: 0.768258, recallAt0.05FDR: 0.288828, recallAt0.1FDR: 0.438692
# Performance on rat enhancers whose orthologs are not brain enhancers in mouse and negative set corresponding to non-enhancer orthologs of mouse enhancers
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/predictNewSequences.py --architectureFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsRandomGCRepeat2XPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_RandomGCRepeatNonEnhRatioXNeg_500bp_conv5MuchMuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal_architecture.json --weightsFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsRandomGCRepeat2XPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_RandomGCRepeatNonEnhRatioXNeg_500bp_conv5MuchMuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal.hdf5 --narrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/RatAtac/M1_AllReps/peak/idr_reproducibility/idr.optimal_peak.inStriatum_nonCDS_enhancerShort_mouseTest_nonMouseBrain.bed.gz --negNarrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_withPeakNames_rn6Huge_summitExtendedMin50Max2Protect5_nonRatBrain_mouseTest.bed.gz --sequenceLength 500 --genomeFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/RatGenome/rn6.fa --chroms chr1 --chroms chr2 --chroms chr8 --chroms chr9 --chroms chr3 --chroms chr4 --chroms chr5 --chroms chr6 --chroms chr7 --chroms chr10 --chroms chr11 --chroms chr12 --chroms chr13 --chroms chr14 --chroms chr15 --chroms chr16 --chroms chr17 --chroms chr18 --chroms chr19 --chroms chr20 --chroms chr21 --chroms chr22 --chroms chrX --chroms chrY --RC --classification --path /home/ikaplow/RegulatoryElementEvolutionProject/src/deeplearning
# Performance: loss: nan, accuracy: 0.650895, sensitivity: 0.822472, specificity: 0.424332, auc: 0.694434, precision: 0.653571, NPV: 0.644144, auprc: 0.632473, auprg: 0.414778, F1: 0.728358, recallAt0.05FDR: 0.005935, recallAt0.1FDR: 0.026706
# Performance on non-restrictive glires-specific enhancers and non-enhancers
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/predictNewSequences.py --architectureFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsRandomGCRepeat2XPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_RandomGCRepeatNonEnhRatioXNeg_500bp_conv5MuchMuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal_architecture.json --weightsFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsRandomGCRepeat2XPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_RandomGCRepeatNonEnhRatioXNeg_500bp_conv5MuchMuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal.hdf5 --narrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_inRatBrainLoose_nonHumanBrain_nonMacaqueBrain.bed.gz --negNarrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/HumanDNase/PMC/cromwell-executions/atac/45c4bd8c-c055-4cea-80c9-5ec0eeaf8024/call-reproducibility_idr/execution/optimal_peak.inStriatum_nonCDS_enhancerShort_mm10_summitExtendedMin50Max2XProtect5_inMacaqueBrainLoose_nonMouseBrain_nonRatBrain.bed.gz --sequenceLength 500 --genomeFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseGenome/mm10.fa --chroms chr1 --chroms chr2 --RC --classification --path /home/ikaplow/RegulatoryElementEvolutionProject/src/deeplearning
# Performance: loss: 0.370869, accuracy: 0.813187, sensitivity: 0.856522, specificity: 0.738806, auc: 0.905094, precision: 0.849138, NPV: 0.750000, auprc: 0.851260, auprg: 0.875724, F1: 0.852814, recallAt0.05FDR: 0.313433, recallAt0.1FDR: 0.522388
# Performance on non-restrictive glires-specific enhancers and non-enhancers, calibrated
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/predictNewSequences.py --architectureFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsRandomGCRepeat2XPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_RandomGCRepeatNonEnhRatioXNeg_500bp_conv5MuchMuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal_architecture.json --weightsFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsRandomGCRepeat2XPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_RandomGCRepeatNonEnhRatioXNeg_500bp_conv5MuchMuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal.hdf5 --narrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_inRatBrainLoose_nonHumanBrain_nonMacaqueBrain.bed.gz --negNarrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/HumanDNase/PMC/cromwell-executions/atac/45c4bd8c-c055-4cea-80c9-5ec0eeaf8024/call-reproducibility_idr/execution/optimal_peak.inStriatum_nonCDS_enhancerShort_mm10_summitExtendedMin50Max2XProtect5_inMacaqueBrainLoose_nonMouseBrain_nonRatBrain.bed.gz --sequenceLength 500 --genomeFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseGenome/mm10.fa --chroms chr1 --chroms chr2 --RC --classification --path /home/ikaplow/RegulatoryElementEvolutionProject/src/deeplearning --recalibrateModel --calibrationXTrFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerVsOtherEuarchontaglireEnhLoosePeakLessThan1kb500bp_data_train.npy --calibrationYTrFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerVsOtherEuarchontaglireEnhLoosePeakLessThan1kb500bp_labels_train.npy
# Performance: loss: 0.515402, accuracy: 0.780220, sensitivity: 0.721739, specificity: 0.880597, auc: 0.905094, precision: 0.912088, NPV: 0.648352, auprc: 0.851260, auprg: 0.875724, F1: 0.805825, recallAt0.05FDR: 0.313433, recallAt0.1FDR: 0.522388
# Performance on non-restrictive euarchonta-specific enhancers and non-enhancers
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/predictNewSequences.py --architectureFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsRandomGCRepeat2XPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_RandomGCRepeatNonEnhRatioXNeg_500bp_conv5MuchMuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal_architecture.json --weightsFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsRandomGCRepeat2XPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_RandomGCRepeatNonEnhRatioXNeg_500bp_conv5MuchMuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal.hdf5 --narrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/HumanDNase/PMC/cromwell-executions/atac/45c4bd8c-c055-4cea-80c9-5ec0eeaf8024/call-reproducibility_idr/execution/optimal_peak.inStriatum_nonCDS_enhancerShort_inMacaqueBrainLoose_nonMouseBrain_nonRatBrain_mouseTest.bed.gz --negNarrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_withPeakNames_hg38Huge_summitExtendedMin50Max2Protect5_mouseTest_inRatBrainLoose_nonHumanBrain_nonMacaqueBrain.bed.gz --sequenceLength 500 --genomeFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/HumanGenome/hg38.fa --RC --classification --path /home/ikaplow/RegulatoryElementEvolutionProject/src/deeplearning
# Performance: loss: 0.681450, accuracy: 0.653846, sensitivity: 0.835821, specificity: 0.547826, auc: 0.792505, precision: 0.518519, NPV: 0.851351, auprc: 0.720030, auprg: 0.689459, F1: 0.640000, recallAt0.05FDR: 0.171642, recallAt0.1FDR: 0.179104
# Performance on non-restrictive euarchonta-specific enhancers and non-enhancers, calibrated
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/predictNewSequences.py --architectureFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsRandomGCRepeat2XPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_RandomGCRepeatNonEnhRatioXNeg_500bp_conv5MuchMuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal_architecture.json --weightsFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsRandomGCRepeat2XPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_RandomGCRepeatNonEnhRatioXNeg_500bp_conv5MuchMuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal.hdf5 --narrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/HumanDNase/PMC/cromwell-executions/atac/45c4bd8c-c055-4cea-80c9-5ec0eeaf8024/call-reproducibility_idr/execution/optimal_peak.inStriatum_nonCDS_enhancerShort_inMacaqueBrainLoose_nonMouseBrain_nonRatBrain_mouseTest.bed.gz --negNarrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_withPeakNames_hg38Huge_summitExtendedMin50Max2Protect5_mouseTest_inRatBrainLoose_nonHumanBrain_nonMacaqueBrain.bed.gz --sequenceLength 500 --genomeFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/HumanGenome/hg38.fa --RC --classification --path /home/ikaplow/RegulatoryElementEvolutionProject/src/deeplearning --recalibrateModel --calibrationXTrFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerVsOtherEuarchontaglireEnhLoosePeakLessThan1kb500bp_data_train.npy --calibrationYTrFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerVsOtherEuarchontaglireEnhLoosePeakLessThan1kb500bp_labels_train.npy
# Preformance: loss: 0.572639, accuracy: 0.719780, sensitivity: 0.694030, specificity: 0.734783, auc: 0.792505, precision: 0.603896, NPV: 0.804762, auprc: 0.720030, auprg: 0.689459, F1: 0.645833, recallAt0.05FDR: 0.171642, recallAt0.1FDR: 0.179104
# Performance on non-restrictive mouse-specific enhancers and non-enhancers
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/predictNewSequences.py --architectureFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsRandomGCRepeat2XPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_RandomGCRepeatNonEnhRatioXNeg_500bp_conv5MuchMuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal_architecture.json --weightsFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsRandomGCRepeat2XPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_RandomGCRepeatNonEnhRatioXNeg_500bp_conv5MuchMuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal.hdf5 --narrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_withPeakNames_nonRatBrain.bed.gz --negNarrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/RatAtac/M1_AllReps/peak/idr_reproducibility/idr.optimal_peak.inStriatum_nonCDS_enhancerShort_mm10_summitExtendedMin50Max2XProtect5_nonMouseBrain.bed.gz --sequenceLength 500 --genomeFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseGenome/mm10.fa --chroms chr1 --chroms chr2 --RC --classification --path /home/ikaplow/RegulatoryElementEvolutionProject/src/deeplearning
# Performance: loss: 0.663654, accuracy: 0.700128, sensitivity: 0.909496, specificity: 0.541573, auc: 0.830130, precision: 0.600392, NPV: 0.887661, auprc: 0.763177, auprg: 0.717473, F1: 0.723304, recallAt0.05FDR: 0.008902, recallAt0.1FDR: 0.008902
# Performance on non-restrictive human-specific enhancers and non-enhancers
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/predictNewSequences.py --architectureFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsRandomGCRepeat2XPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_RandomGCRepeatNonEnhRatioXNeg_500bp_conv5MuchMuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal_architecture.json --weightsFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsRandomGCRepeat2XPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_RandomGCRepeatNonEnhRatioXNeg_500bp_conv5MuchMuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal.hdf5 --narrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/HumanDNase/PMC/cromwell-executions/atac/45c4bd8c-c055-4cea-80c9-5ec0eeaf8024/call-reproducibility_idr/execution/optimal_peak.inStriatum_nonCDS_enhancerShort_nonMacaqueBrain_mouseTest.bed.gz --negNarrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MacaqueAtac/OfM/peak/idr_reproducibility/idr.optimal_peak.inM1AndStriatum_nonCDS_enhancerShort_hg38_summitExtendedMin50Max2XProtect5_nonHumanBrain_mouseTest.bed.gz --sequenceLength 500 --genomeFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/HumanGenome/hg38.fa --RC --classification --path /home/ikaplow/RegulatoryElementEvolutionProject/src/deeplearning
# Performance: loss: inf, accuracy: 0.375000, sensitivity: 0.909091, specificity: 0.196970, auc: 0.601928, precision: 0.273973, NPV: 0.866667, auprc: 0.305595, auprg: 0.310857, F1: 0.421053, recallAt0.05FDR: 0.000000, recallAt0.1FDR: 0.000000
# Performance on non-restrictive macaque-specific enhancers and non-enhancers
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/predictNewSequences.py --architectureFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsRandomGCRepeat2XPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_RandomGCRepeatNonEnhRatioXNeg_500bp_conv5MuchMuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal_architecture.json --weightsFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsRandomGCRepeat2XPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_RandomGCRepeatNonEnhRatioXNeg_500bp_conv5MuchMuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal.hdf5 --narrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MacaqueAtac/OfM/peak/idr_reproducibility/idr.optimal_peak.inM1AndStriatum_nonCDS_enhancerShort_nonHumanBrain_mouseTest.bed.gz --negNarrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/HumanDNase/PMC/cromwell-executions/atac/45c4bd8c-c055-4cea-80c9-5ec0eeaf8024/call-reproducibility_idr/execution/optimal_peak.inStriatum_nonCDS_enhancerShort_rheMac8_summitExtendedMin50Max2XProtect5_UCSCNames_nonMacaqueBrain_mouseTest.bed.gz --sequenceLength 500 --genomeFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MacaqueGenome/rheMac8.fa --RC --classification --path /home/ikaplow/RegulatoryElementEvolutionProject/src/deeplearning
# Performance: loss: 0.543792, accuracy: 0.746212, sensitivity: 0.883838, specificity: 0.333333, auc: 0.739057, precision: 0.799087, NPV: 0.488889, auprc: 0.467821, auprg: 0.591329, F1: 0.839329, recallAt0.05FDR: 0.000000, recallAt0.1FDR: 0.000000
# Performance on shared brain and liver enhancers and liver enhancers that are not brain enhancers
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/predictNewSequences.py --architectureFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsRandomGCRepeat2XPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_RandomGCRepeatNonEnhRatioXNeg_500bp_conv5MuchMuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal_architecture.json --weightsFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsRandomGCRepeat2XPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_RandomGCRepeatNonEnhRatioXNeg_500bp_conv5MuchMuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal.hdf5 --narrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_andLiverLiuPfenning.narrowPeak.gz --negNarrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/LiverPfenning_ATAC_out/atac/68eb27d1-23c7-4d42-80f9-b3dd46f5e18c/call-reproducibility_idr/execution/idr.optimal_peak.narrowPeak_inLiuAll_nonCDS_enhancerShort_nonBrain.narrowPeak.gz --sequenceLength 500 --genomeFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseGenome/mm10.fa  --chroms chr1 --chroms chr2 --RC --classification --path /home/ikaplow/RegulatoryElementEvolutionProject/src/deeplearning
# Performance: loss: nan, accuracy: 0.785165, sensitivity: 0.908654, specificity: 0.747191, auc: 0.931863, precision: 0.525000, NPV: 0.963768, auprc: 0.877841, auprg: 0.969234, F1: 0.665493, recallAt0.05FDR: 0.633654, recallAt0.1FDR: 0.714423
# Performance on shared brain and liver enhancers and liver enhancers that are not brain enhancers, calibrated
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/predictNewSequences.py --architectureFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsRandomGCRepeat2XPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_RandomGCRepeatNonEnhRatioXNeg_500bp_conv5MuchMuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal_architecture.json --weightsFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsRandomGCRepeat2XPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_RandomGCRepeatNonEnhRatioXNeg_500bp_conv5MuchMuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal.hdf5 --narrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_andLiverLiuPfenning.narrowPeak.gz --negNarrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/LiverPfenning_ATAC_out/atac/68eb27d1-23c7-4d42-80f9-b3dd46f5e18c/call-reproducibility_idr/execution/idr.optimal_peak.narrowPeak_inLiuAll_nonCDS_enhancerShort_nonBrain.narrowPeak.gz --sequenceLength 500 --genomeFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseGenome/mm10.fa  --chroms chr1 --chroms chr2 --RC --classification --path /home/ikaplow/RegulatoryElementEvolutionProject/src/deeplearning --recalibrateModel --calibrationXTrFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerVsOtherEuarchontaglireEnhLoosePeakLessThan1kb500bp_data_train.npy --calibrationYTrFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerVsOtherEuarchontaglireEnhLoosePeakLessThan1kb500bp_labels_train.npy
# Performance: loss: 0.295482, accuracy: 0.874491, sensitivity: 0.830769, specificity: 0.887936, auc: 0.931863, precision: 0.695093, NPV: 0.944637, auprc: 0.877841, auprg: 0.969234, F1: 0.756899, recallAt0.05FDR: 0.633654, recallAt0.1FDR: 0.714423
# Predictions for brain only enhancers, shared brain and liver enhancers, liver only enhancers, and negative set
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/predictNewSequencesNoEvaluation.py --architectureFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsRandomGCRepeat2XPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_RandomGCRepeatNonEnhRatioXNeg_500bp_conv5MuchMuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal_architecture.json --weightsFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsRandomGCRepeat2XPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_RandomGCRepeatNonEnhRatioXNeg_500bp_conv5MuchMuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal.hdf5 --narrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_nonLiver.narrowPeak.gz --sequenceLength 500 --predictedProbaFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_nonLiver_test_RandomGCRepeat2XNeg_predictedProba.txt --genomeFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseGenome/mm10.fa --chromSizesFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseGenome/mm10.chrom.sizes --chroms chr1 --chroms chr2 --classification
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/predictNewSequencesNoEvaluation.py --architectureFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsRandomGCRepeat2XPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_RandomGCRepeatNonEnhRatioXNeg_500bp_conv5MuchMuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal_architecture.json --weightsFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsRandomGCRepeat2XPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_RandomGCRepeatNonEnhRatioXNeg_500bp_conv5MuchMuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal.hdf5 --narrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_andLiverLiuPfenning.narrowPeak.gz --sequenceLength 500 --predictedProbaFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_andLiverLiuPfenning_test_RandomGCRepeat2XNeg_predictedProba.txt --genomeFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseGenome/mm10.fa --chromSizesFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseGenome/mm10.chrom.sizes --chroms chr1 --chroms chr2 --classification
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/predictNewSequencesNoEvaluation.py --architectureFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsRandomGCRepeat2XPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_RandomGCRepeatNonEnhRatioXNeg_500bp_conv5MuchMuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal_architecture.json --weightsFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsRandomGCRepeat2XPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_RandomGCRepeatNonEnhRatioXNeg_500bp_conv5MuchMuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal.hdf5 --narrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/LiverPfenning_ATAC_out/atac/68eb27d1-23c7-4d42-80f9-b3dd46f5e18c/call-reproducibility_idr/execution/idr.optimal_peak.narrowPeak_inLiuAll_nonCDS_enhancerShort_nonBrain.narrowPeak.gz --sequenceLength 500 --predictedProbaFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/LiverPfenning_ATAC_out/atac/68eb27d1-23c7-4d42-80f9-b3dd46f5e18c/call-reproducibility_idr/execution/idr.optimal_peak.narrowPeak_inLiuAll_nonCDS_enhancerShort_nonBrain_test_RandomGCRepeat2XNeg_predictedProba.txt --genomeFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseGenome/mm10.fa --chromSizesFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseGenome/mm10.chrom.sizes --chroms chr1 --chroms chr2 --classification
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/predictNewSequencesNoEvaluation.py --architectureFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsRandomGCRepeat2XPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_RandomGCRepeatNonEnhRatioXNeg_500bp_conv5MuchMuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal_architecture.json --weightsFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsRandomGCRepeat2XPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_RandomGCRepeatNonEnhRatioXNeg_500bp_conv5MuchMuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal.hdf5 --narrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_summitPlusMinus250bp_randomGCRepeatMatched2x_nonMouseBrain.narrowPeak.gz --sequenceLength 500 --predictedProbaFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_summitPlusMinus250bp_randomGCRepeatMatched2x_nonMouseBrain_test_predictedProba.txt --genomeFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseGenome/mm10.fa --chromSizesFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseGenome/mm10.chrom.sizes --chroms chr1 --chroms chr2 --classification
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/makeViolinPlotTissueComparison.py --tissueOnlyPredFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_nonLiver_test_flankNeg_predictedProba.txt --tissueSharedPredFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_andLiverLiuPfenning_test_flankNeg_predictedProba.txt --tissueOtherPredFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/LiverPfenning_ATAC_out/atac/68eb27d1-23c7-4d42-80f9-b3dd46f5e18c/call-reproducibility_idr/execution/idr.optimal_peak.narrowPeak_inLiuAll_nonCDS_enhancerShort_nonBrain_test_flankNeg_predictedProba.txt --negSetPredFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_summitPlusMinus250bp_randomGCRepeatMatched2x_nonMouseBrain_test_predictedProba.txt --outputFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_brainNonLiverVsBrainLiverVsLiverNonBrainVsRandomGCRepeat2XNeg_predictedProba.svg
# Performance on macaque shared brain and liver enhancers and liver enhancers that are not brain enhancers
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/predictNewSequences.py --architectureFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsRandomGCRepeat2XPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_RandomGCRepeatNonEnhRatioXNeg_500bp_conv5MuchMuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal_architecture.json --weightsFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsRandomGCRepeat2XPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_RandomGCRepeatNonEnhRatioXNeg_500bp_conv5MuchMuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal.hdf5 --narrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MacaqueAtac/OfM/peak/idr_reproducibility/idr.optimal_peak.inM1AndStriatum_nonCDS_enhancerShort_andLiver_mouseTest.bed.gz --negNarrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MacaqueAtac/Liver/cromwell-executions/atac/1f5fd9f0-d389-4184-99cb-c1125fd7f064/call-reproducibility_idr/execution/optimal_peak_nonCDS_enhancerShort_mouseTest_nonMacaqueBrain.bed.gz --sequenceLength 500 --genomeFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MacaqueGenome/rheMac8.fa --chroms chr1 --chroms chr2 --chroms chr8 --chroms chr9 --chroms chr3 --chroms chr4 --chroms chr5 --chroms chr6 --chroms chr7 --chroms chr10 --chroms chr11 --chroms chr12 --chroms chr13 --chroms chr14 --chroms chr15 --chroms chr16 --chroms chr17 --chroms chr18 --chroms chr19 --chroms chr20 --chroms chrX --chroms chrY --RC --classification --path /home/ikaplow/RegulatoryElementEvolutionProject/src/deeplearning
# Performance: loss: nan, accuracy: 0.774617, sensitivity: 0.947991, specificity: 0.697257, auc: 0.946277, precision: 0.582849, NPV: 0.967789, auprc: 0.900714, auprg: 0.959477, F1: 0.721872, recallAt0.05FDR: 0.641844, recallAt0.1FDR: 0.761229
# Performance on macaque shared brain and liver enhancers and liver enhancers that are not brain enhancers, calibrated
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/predictNewSequences.py --architectureFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsRandomGCRepeat2XPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_RandomGCRepeatNonEnhRatioXNeg_500bp_conv5MuchMuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal_architecture.json --weightsFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsRandomGCRepeat2XPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_RandomGCRepeatNonEnhRatioXNeg_500bp_conv5MuchMuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal.hdf5 --narrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MacaqueAtac/OfM/peak/idr_reproducibility/idr.optimal_peak.inM1AndStriatum_nonCDS_enhancerShort_andLiver_mouseTest.bed.gz --negNarrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MacaqueAtac/Liver/cromwell-executions/atac/1f5fd9f0-d389-4184-99cb-c1125fd7f064/call-reproducibility_idr/execution/optimal_peak_nonCDS_enhancerShort_mouseTest_nonMacaqueBrain.bed.gz --sequenceLength 500 --genomeFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MacaqueGenome/rheMac8.fa --chroms chr1 --chroms chr2 --chroms chr8 --chroms chr9 --chroms chr3 --chroms chr4 --chroms chr5 --chroms chr6 --chroms chr7 --chroms chr10 --chroms chr11 --chroms chr12 --chroms chr13 --chroms chr14 --chroms chr15 --chroms chr16 --chroms chr17 --chroms chr18 --chroms chr19 --chroms chr20 --chroms chrX --chroms chrY --RC --classification --path /home/ikaplow/RegulatoryElementEvolutionProject/src/deeplearning --recalibrateModel --calibrationXTrFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerVsOtherEuarchontaglireEnhLoosePeakLessThan1kb500bp_data_train.npy --calibrationYTrFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerVsOtherEuarchontaglireEnhLoosePeakLessThan1kb500bp_labels_train.npy
# Performance: loss: 0.312631, accuracy: 0.863968, sensitivity: 0.892435, specificity: 0.851266, auc: 0.946277, precision: 0.728062, NPV: 0.946628, auprc: 0.900714, auprg: 0.959477, F1: 0.801912, recallAt0.05FDR: 0.641844, recallAt0.1FDR: 0.761229
# Performance on rat shared brain and liver enhancers and liver enhancers that are not brain enhancers
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/predictNewSequences.py --architectureFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsRandomGCRepeat2XPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_RandomGCRepeatNonEnhRatioXNeg_500bp_conv5MuchMuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal_architecture.json --weightsFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsRandomGCRepeat2XPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_RandomGCRepeatNonEnhRatioXNeg_500bp_conv5MuchMuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal.hdf5 --narrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/RatAtac/M1_AllReps/peak/idr_reproducibility/idr.optimal_peak.inStriatum_nonCDS_enhancerShort_andLiver_mouseTest.bed.gz --negNarrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/RatAtac/Liver_GoodReps/peak/idr_reproducibility/idr.optimal_peak_nonCDS_enhancerShort_nonRatBrain_mouseTest.bed.gz --sequenceLength 500 --genomeFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/RatGenome/rn6.fa --chroms chr1 --chroms chr2 --chroms chr8 --chroms chr9 --chroms chr3 --chroms chr4 --chroms chr5 --chroms chr6 --chroms chr7 --chroms chr10 --chroms chr11 --chroms chr12 --chroms chr13 --chroms chr14 --chroms chr15 --chroms chr16 --chroms chr17 --chroms chr18 --chroms chr19 --chroms chr20 --chroms chrX --chroms chrY --RC --classification
# Performance: loss: nan, accuracy: 0.762670, sensitivity: 0.863277, specificity: 0.712052, auc: 0.894308, precision: 0.601338, NPV: 0.911904, auprc: 0.851286, auprg: 0.904993, F1: 0.708884, recallAt0.05FDR: 0.473446, recallAt0.1FDR: 0.590395
# Performance on rat shared brain and liver enhancers and liver enhancers that are not brain enhancers, calibrated
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/predictNewSequences.py --architectureFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsRandomGCRepeat2XPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_RandomGCRepeatNonEnhRatioXNeg_500bp_conv5MuchMuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal_architecture.json --weightsFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsRandomGCRepeat2XPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_RandomGCRepeatNonEnhRatioXNeg_500bp_conv5MuchMuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal.hdf5 --narrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/RatAtac/M1_AllReps/peak/idr_reproducibility/idr.optimal_peak.inStriatum_nonCDS_enhancerShort_andLiver_mouseTest.bed.gz --negNarrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/RatAtac/Liver_GoodReps/peak/idr_reproducibility/idr.optimal_peak_nonCDS_enhancerShort_nonRatBrain_mouseTest.bed.gz --sequenceLength 500 --genomeFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/RatGenome/rn6.fa --chroms chr1 --chroms chr2 --chroms chr8 --chroms chr9 --chroms chr3 --chroms chr4 --chroms chr5 --chroms chr6 --chroms chr7 --chroms chr10 --chroms chr11 --chroms chr12 --chroms chr13 --chroms chr14 --chroms chr15 --chroms chr16 --chroms chr17 --chroms chr18 --chroms chr19 --chroms chr20 --chroms chrX --chroms chrY --RC --classification --recalibrateModel --calibrationXTrFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerVsOtherEuarchontaglireEnhLoosePeakLessThan1kb500bp_data_train.npy --calibrationYTrFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerVsOtherEuarchontaglireEnhLoosePeakLessThan1kb500bp_labels_train.npy
# Performance: loss: 0.400322, accuracy: 0.827156, sensitivity: 0.776271, specificity: 0.852757, auc: 0.894308, precision: 0.726216, NPV: 0.883392, auprc: 0.851286, auprg: 0.904993, F1: 0.750410, recallAt0.05FDR: 0.473446, recallAt0.1FDR: 0.590395
# Get brain test set predictions across Zoonomia mammals
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/makePredictNewSequencesNoEvaluationScript.py --architectureFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsRandomGCRepeat2XPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_RandomGCRepeatNonEnhRatioXNeg_500bp_conv5MuchMuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal_architecture.json --weightsFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsRandomGCRepeat2XPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_RandomGCRepeatNonEnhRatioXNeg_500bp_conv5MuchMuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal.hdf5 --narrowPeakFileNameListFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_orLiver_LiftedPeaks/liftedPeakMouseTestNarrowPeakFileNamesFiltConverted.txt --genomeFileNameListFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/200MammalsFastas/200MammalsAssembliesFileNamesFilt.txt --chromSizesFileNameListFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/200MammalsFastas/chromSizesFileNames.txt --classification --scriptFileName /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/predictMouseBrainTestHugeCactusRandomGC2XScript.sh --predictedClassesFileNameSuffix RandomGCRepeat2XNeg_predictedClasses.txt --predictedProbaFileNameSuffix RandomGCRepeat2XNeg_predictedProba.txt
sh /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/predictMouseBrainTestHugeCactusRandomGC2XScript.sh 2> /home/ikaplow/RegulatoryElementEvolutionProject/src/predictMouseBrainTestHugeCactusRandomGC2XScript.e
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/gatherPeakPredictionsAcrossSpecies.py --predictionsFileNameListFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/testPredictionsRandomGC2XGliresFileNames.txt --baseSpeciesLine 35 --outputFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_test_RandomGCRepeat2XNeg_peakPredictionsGlires.txt # Filtered correlation (Pearson, Spearman): -0.9369*, -0.6972*

# Performance of brain model with 10X dinucleotide-shuffled negatives on test set
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/predictNewSequences.py --architectureFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsDiShuf10XPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_DiShuf10XNeg_500bp_conv5MuchMuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal_architecture.json --weightsFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsDiShuf10XPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_DiShuf10XNeg_500bp_conv5MuchMuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal.hdf5 --fastaFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_summitPlusMinus250bp_test.fa --negFastaFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_summitPlusMinus250bp_test_shufDiNuc10x.fa --sequenceLength 500 --genomeFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseGenome/mm10.fa --chroms chr1 --chroms chr2 --predictedProbaFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_test_andNeg_DiShuf10XNeg_conv5MuchMuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal_predictedProba.txt --RC --classification --path /home/ikaplow/RegulatoryElementEvolutionProject/src/deeplearning
# Performance: loss: nan, accuracy: 0.982537, sensitivity: 0.849869, specificity: 0.995804, auc: 0.993935, precision: 0.952953, NPV: 0.985148, auprc: 0.965455, auprg: 0.999297, F1: 0.898464, recallAt0.05FDR: 0.853584, recallAt0.1FDR: 0.907561
# Performance on mouse enhancers whose orthologs are not brain enhancers in at least one other species and negative set
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/predictNewSequences.py --architectureFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsDiShuf10XPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_DiShuf10XNeg_500bp_conv5MuchMuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal_architecture.json --weightsFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsDiShuf10XPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_DiShuf10XNeg_500bp_conv5MuchMuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal.hdf5 --narrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_withPeakNames_nonBrainOtherSpecies.narrowPeak.gz --negNarrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/HumanDNase/PMC/cromwell-executions/atac/45c4bd8c-c055-4cea-80c9-5ec0eeaf8024/call-reproducibility_idr/execution/optimal_peak.inStriatum_nonCDS_enhancerShort_mm10_summitExtendedMin50Max2XProtect5_nonMouseBrain_andMacaque_andRat.narrowPeak.gz --sequenceLength 500 --genomeFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseGenome/mm10.fa --chroms chr1 --chroms chr2 --RC --classification --path /home/ikaplow/RegulatoryElementEvolutionProject/src/deeplearning
# Performance: loss: nan, accuracy: 0.604901, sensitivity: 0.838854, specificity: 0.426770, auc: 0.696157, precision: 0.527011, NPV: 0.776699, auprc: 0.603508, auprg: 0.424662, F1: 0.647333, recallAt0.05FDR: 0.000000, recallAt0.1FDR: 0.007006
# Performance on human enhancers whose orthologs are not brain enhancers in mouse and negative set corresponding to non-enhancer orthologs of mouse enhancers
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/predictNewSequences.py --architectureFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsDiShuf10XPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_DiShuf10XNeg_500bp_conv5MuchMuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal_architecture.json --weightsFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsDiShuf10XPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_DiShuf10XNeg_500bp_conv5MuchMuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal.hdf5 --narrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/HumanDNase/PMC/cromwell-executions/atac/45c4bd8c-c055-4cea-80c9-5ec0eeaf8024/call-reproducibility_idr/execution/optimal_peak.inStriatum_nonCDS_enhancerShort_nonMouseBrain_mouseTest.bed.gz --negNarrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_withPeakNames_hg38Huge_summitExtendedMin50Max2Protect5_nonHumanBrain_mouseTest.narrowPeak.gz --sequenceLength 500 --genomeFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/HumanGenome/hg38.fa --RC --classification --path /home/ikaplow/RegulatoryElementEvolutionProject/src/deeplearning
# Performance: loss: 1.261745, accuracy: 0.572409, sensitivity: 0.709135, specificity: 0.508929, auc: 0.654047, precision: 0.401361, NPV: 0.790295, auprc: 0.443799, auprg: 0.396644, F1: 0.512598, recallAt0.05FDR: 0.004808, recallAt0.1FDR: 0.004808
# Performance on macaque enhancers whose orthologs are not brain enhancers in mouse and negative set corresponding to non-enhancer orthologs of mouse enhancers
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/predictNewSequences.py --architectureFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsDiShuf10XPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_DiShuf10XNeg_500bp_conv5MuchMuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal_architecture.json --weightsFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsDiShuf10XPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_DiShuf10XNeg_500bp_conv5MuchMuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal.hdf5 --narrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MacaqueAtac/OfM/peak/idr_reproducibility/idr.optimal_peak.inM1AndStriatum_nonCDS_enhancerShort_nonMouseBrain_mouseTest.bed.gz --negNarrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_withPeakNames_rheMac8Huge_summitExtendedMin50Max2Protect5_nonMacaqueBrain_mouseTest.bed.gz --sequenceLength 500 --genomeFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MacaqueGenome/rheMac8.fa --chroms chr1 --chroms chr2 --chroms chr8 --chroms chr9 --chroms chr3 --chroms chr4 --chroms chr5 --chroms chr6 --chroms chr7 --chroms chr10 --chroms chr11 --chroms chr12 --chroms chr13 --chroms chr14 --chroms chr15 --chroms chr16 --chroms chr17 --chroms chr18 --chroms chr19 --chroms chr20 --chroms chr21 --chroms chr22 --chroms chrX --chroms chrY --RC --classification --path /home/ikaplow/RegulatoryElementEvolutionProject/src/deeplearning
# Performance: loss: nan, accuracy: 0.683794, sensitivity: 0.846049, specificity: 0.531888, auc: 0.775482, precision: 0.628543, NPV: 0.786792, auprc: 0.763626, auprg: 0.562959, F1: 0.721254, recallAt0.05FDR: 0.089918, recallAt0.1FDR: 0.178474
# Performance on rat enhancers whose orthologs are not brain enhancers in mouse and negative set corresponding to non-enhancer orthologs of mouse enhancers
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/predictNewSequences.py --architectureFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsDiShuf10XPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_DiShuf10XNeg_500bp_conv5MuchMuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal_architecture.json --weightsFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsDiShuf10XPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_DiShuf10XNeg_500bp_conv5MuchMuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal.hdf5 --narrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/RatAtac/M1_AllReps/peak/idr_reproducibility/idr.optimal_peak.inStriatum_nonCDS_enhancerShort_mouseTest_nonMouseBrain.bed.gz --negNarrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_withPeakNames_rn6Huge_summitExtendedMin50Max2Protect5_nonRatBrain_mouseTest.bed.gz --sequenceLength 500 --genomeFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/RatGenome/rn6.fa --chroms chr1 --chroms chr2 --chroms chr8 --chroms chr9 --chroms chr3 --chroms chr4 --chroms chr5 --chroms chr6 --chroms chr7 --chroms chr10 --chroms chr11 --chroms chr12 --chroms chr13 --chroms chr14 --chroms chr15 --chroms chr16 --chroms chr17 --chroms chr18 --chroms chr19 --chroms chr20 --chroms chr21 --chroms chr22 --chroms chrX --chroms chrY --RC --classification --path /home/ikaplow/RegulatoryElementEvolutionProject/src/deeplearning
# Peformance: loss: nan, accuracy: 0.590153, sensitivity: 0.769663, specificity: 0.353116, auc: 0.628193, precision: 0.611062, NPV: 0.537246, auprc: 0.525143, auprg: 0.271283, F1: 0.681253, recallAt0.05FDR: 0.000000, recallAt0.1FDR: 0.000000
# Performance on non-restrictive glires-specific enhancers and non-enhancers
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/predictNewSequences.py --architectureFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsDiShuf10XPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_DiShuf10XNeg_500bp_conv5MuchMuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal_architecture.json --weightsFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsDiShuf10XPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_DiShuf10XNeg_500bp_conv5MuchMuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal.hdf5 --narrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_inRatBrainLoose_nonHumanBrain_nonMacaqueBrain.bed.gz --negNarrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/HumanDNase/PMC/cromwell-executions/atac/45c4bd8c-c055-4cea-80c9-5ec0eeaf8024/call-reproducibility_idr/execution/optimal_peak.inStriatum_nonCDS_enhancerShort_mm10_summitExtendedMin50Max2XProtect5_inMacaqueBrainLoose_nonMouseBrain_nonRatBrain.bed.gz --sequenceLength 500 --genomeFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseGenome/mm10.fa --chroms chr1 --chroms chr2 --RC --classification --path /home/ikaplow/RegulatoryElementEvolutionProject/src/deeplearning
# Performance: loss: nan, accuracy: 0.747253, sensitivity: 0.826087, specificity: 0.611940, auc: 0.786859, precision: 0.785124, NPV: 0.672131, auprc: 0.708801, auprg: 0.692813, F1: 0.805085, recallAt0.05FDR: 0.037313, recallAt0.1FDR: 0.104478
# Performance on non-restrictive glires-specific enhancers and non-enhancers, calibrated
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/predictNewSequences.py --architectureFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsDiShuf10XPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_DiShuf10XNeg_500bp_conv5MuchMuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal_architecture.json --weightsFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsDiShuf10XPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_DiShuf10XNeg_500bp_conv5MuchMuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal.hdf5 --narrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_inRatBrainLoose_nonHumanBrain_nonMacaqueBrain.bed.gz --negNarrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/HumanDNase/PMC/cromwell-executions/atac/45c4bd8c-c055-4cea-80c9-5ec0eeaf8024/call-reproducibility_idr/execution/optimal_peak.inStriatum_nonCDS_enhancerShort_mm10_summitExtendedMin50Max2XProtect5_inMacaqueBrainLoose_nonMouseBrain_nonRatBrain.bed.gz --sequenceLength 500 --genomeFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseGenome/mm10.fa --chroms chr1 --chroms chr2 --RC --classification --path /home/ikaplow/RegulatoryElementEvolutionProject/src/deeplearning --recalibrateModel --calibrationXTrFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerVsOtherEuarchontaglireEnhLoosePeakLessThan1kb500bp_data_train.npy --calibrationYTrFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerVsOtherEuarchontaglireEnhLoosePeakLessThan1kb500bp_labels_train.npy
# Performance: loss: 0.641773, accuracy: 0.714286, sensitivity: 0.669565, specificity: 0.791045, auc: 0.786859, precision: 0.846154, NPV: 0.582418, auprc: 0.708801, auprg: 0.692813, F1: 0.747573, recallAt0.05FDR: 0.037313, recallAt0.1FDR: 0.104478
# Performance on non-restrictive euarchonta-specific enhancers and non-enhancers
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/predictNewSequences.py --architectureFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsDiShuf10XPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_DiShuf10XNeg_500bp_conv5MuchMuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal_architecture.json --weightsFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsDiShuf10XPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_DiShuf10XNeg_500bp_conv5MuchMuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal.hdf5 --narrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/HumanDNase/PMC/cromwell-executions/atac/45c4bd8c-c055-4cea-80c9-5ec0eeaf8024/call-reproducibility_idr/execution/optimal_peak.inStriatum_nonCDS_enhancerShort_inMacaqueBrainLoose_nonMouseBrain_nonRatBrain_mouseTest.bed.gz --negNarrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_withPeakNames_hg38Huge_summitExtendedMin50Max2Protect5_mouseTest_inRatBrainLoose_nonHumanBrain_nonMacaqueBrain.bed.gz --sequenceLength 500 --genomeFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/HumanGenome/hg38.fa --RC --classification --path /home/ikaplow/RegulatoryElementEvolutionProject/src/deeplearning
# Performance: loss: 1.023134, accuracy: 0.593407, sensitivity: 0.679104, specificity: 0.543478, auc: 0.690558, precision: 0.464286, NPV: 0.744048, auprc: 0.581158, auprg: 0.419168, F1: 0.551515, recallAt0.05FDR: 0.037313, recallAt0.1FDR: 0.037313
# Performance on non-restrictive euarchonta-specific enhancers and non-enhancers, calibrated
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/predictNewSequences.py --architectureFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsDiShuf10XPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_DiShuf10XNeg_500bp_conv5MuchMuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal_architecture.json --weightsFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsDiShuf10XPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_DiShuf10XNeg_500bp_conv5MuchMuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal.hdf5 --narrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/HumanDNase/PMC/cromwell-executions/atac/45c4bd8c-c055-4cea-80c9-5ec0eeaf8024/call-reproducibility_idr/execution/optimal_peak.inStriatum_nonCDS_enhancerShort_inMacaqueBrainLoose_nonMouseBrain_nonRatBrain_mouseTest.bed.gz --negNarrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_withPeakNames_hg38Huge_summitExtendedMin50Max2Protect5_mouseTest_inRatBrainLoose_nonHumanBrain_nonMacaqueBrain.bed.gz --sequenceLength 500 --genomeFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/HumanGenome/hg38.fa --RC --classification --path /home/ikaplow/RegulatoryElementEvolutionProject/src/deeplearning --recalibrateModel --calibrationXTrFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerVsOtherEuarchontaglireEnhLoosePeakLessThan1kb500bp_data_train.npy --calibrationYTrFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerVsOtherEuarchontaglireEnhLoosePeakLessThan1kb500bp_labels_train.npy
# Performance: loss: 0.641357, accuracy: 0.642857, sensitivity: 0.514925, specificity: 0.717391, auc: 0.690558, precision: 0.514925, NPV: 0.717391, auprc: 0.581158, auprg: 0.419168, F1: 0.514925, recallAt0.05FDR: 0.037313, recallAt0.1FDR: 0.037313
# Performance on non-restrictive mouse-specific enhancers and non-enhancers
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/predictNewSequences.py --architectureFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsDiShuf10XPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_DiShuf10XNeg_500bp_conv5MuchMuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal_architecture.json --weightsFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsDiShuf10XPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_DiShuf10XNeg_500bp_conv5MuchMuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal.hdf5 --narrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_withPeakNames_nonRatBrain.bed.gz --negNarrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/RatAtac/M1_AllReps/peak/idr_reproducibility/idr.optimal_peak.inStriatum_nonCDS_enhancerShort_mm10_summitExtendedMin50Max2XProtect5_nonMouseBrain.bed.gz --sequenceLength 500 --genomeFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseGenome/mm10.fa --chroms chr1 --chroms chr2 --RC --classification --path /home/ikaplow/RegulatoryElementEvolutionProject/src/deeplearning
# Performance: loss: nan, accuracy: 0.592072, sensitivity: 0.870920, specificity: 0.380899, auc: 0.691184, precision: 0.515817, NPV: 0.795775, auprc: 0.582162, auprg: 0.420560, F1: 0.647903, recallAt0.05FDR: 0.000000, recallAt0.1FDR: 0.000000
# Performance on non-restrictive human-specific enhancers and non-enhancers
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/predictNewSequences.py --architectureFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsDiShuf10XPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_DiShuf10XNeg_500bp_conv5MuchMuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal_architecture.json --weightsFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsDiShuf10XPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_DiShuf10XNeg_500bp_conv5MuchMuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal.hdf5 --narrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/HumanDNase/PMC/cromwell-executions/atac/45c4bd8c-c055-4cea-80c9-5ec0eeaf8024/call-reproducibility_idr/execution/optimal_peak.inStriatum_nonCDS_enhancerShort_nonMacaqueBrain_mouseTest.bed.gz --negNarrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MacaqueAtac/OfM/peak/idr_reproducibility/idr.optimal_peak.inM1AndStriatum_nonCDS_enhancerShort_hg38_summitExtendedMin50Max2XProtect5_nonHumanBrain_mouseTest.bed.gz --sequenceLength 500 --genomeFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/HumanGenome/hg38.fa --RC --classification --path /home/ikaplow/RegulatoryElementEvolutionProject/src/deeplearning
# Performance: loss: 2.373343, accuracy: 0.371212, sensitivity: 0.712121, specificity: 0.257576, auc: 0.465259, precision: 0.242268, NPV: 0.728571, auprc: 0.269140, auprg: -0.152010, F1: 0.361538, recallAt0.05FDR: 0.030303, recallAt0.1FDR: 0.030303
# Performance on non-restrictive macaque-specific enhancers and non-enhancers
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/predictNewSequences.py --architectureFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsDiShuf10XPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_DiShuf10XNeg_500bp_conv5MuchMuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal_architecture.json --weightsFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsDiShuf10XPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_DiShuf10XNeg_500bp_conv5MuchMuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal.hdf5 --narrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MacaqueAtac/OfM/peak/idr_reproducibility/idr.optimal_peak.inM1AndStriatum_nonCDS_enhancerShort_nonHumanBrain_mouseTest.bed.gz --negNarrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/HumanDNase/PMC/cromwell-executions/atac/45c4bd8c-c055-4cea-80c9-5ec0eeaf8024/call-reproducibility_idr/execution/optimal_peak.inStriatum_nonCDS_enhancerShort_rheMac8_summitExtendedMin50Max2XProtect5_UCSCNames_nonMacaqueBrain_mouseTest.bed.gz --sequenceLength 500 --genomeFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MacaqueGenome/rheMac8.fa --RC --classification --path /home/ikaplow/RegulatoryElementEvolutionProject/src/deeplearning
# Performance: loss: nan, accuracy: 0.674242, sensitivity: 0.762626, specificity: 0.409091, auc: 0.621212, precision: 0.794737, NPV: 0.364865, auprc: 0.401587, auprg: 0.378420, F1: 0.778351, recallAt0.05FDR: 0.045455, recallAt0.1FDR: 0.045455
# Performance on shared brain and liver enhancers and liver enhancers that are not brain enhancers
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/predictNewSequences.py --architectureFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsDiShuf10XPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_DiShuf10XNeg_500bp_conv5MuchMuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal_architecture.json --weightsFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsDiShuf10XPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_DiShuf10XNeg_500bp_conv5MuchMuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal.hdf5 --narrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_andLiverLiuPfenning.narrowPeak.gz --negNarrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/LiverPfenning_ATAC_out/atac/68eb27d1-23c7-4d42-80f9-b3dd46f5e18c/call-reproducibility_idr/execution/idr.optimal_peak.narrowPeak_inLiuAll_nonCDS_enhancerShort_nonBrain.narrowPeak.gz --sequenceLength 500 --genomeFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseGenome/mm10.fa  --chroms chr1 --chroms chr2 --RC --classification --path /home/ikaplow/RegulatoryElementEvolutionProject/src/deeplearning
# Performance: loss: nan, accuracy: 0.607191, sensitivity: 0.862500, specificity: 0.528681, auc: 0.813873, precision: 0.360096, NPV: 0.925945, auprc: 0.608024, auprg: 0.815388, F1: 0.508071, recallAt0.05FDR: 0.000000, recallAt0.1FDR: 0.000000
# Performance on shared brain and liver enhancers and liver enhancers that are not brain enhancers, calibrated
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/predictNewSequences.py --architectureFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsDiShuf10XPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_DiShuf10XNeg_500bp_conv5MuchMuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal_architecture.json --weightsFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsDiShuf10XPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_DiShuf10XNeg_500bp_conv5MuchMuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal.hdf5 --narrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_andLiverLiuPfenning.narrowPeak.gz --negNarrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/LiverPfenning_ATAC_out/atac/68eb27d1-23c7-4d42-80f9-b3dd46f5e18c/call-reproducibility_idr/execution/idr.optimal_peak.narrowPeak_inLiuAll_nonCDS_enhancerShort_nonBrain.narrowPeak.gz --sequenceLength 500 --genomeFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseGenome/mm10.fa  --chroms chr1 --chroms chr2 --RC --classification --path /home/ikaplow/RegulatoryElementEvolutionProject/src/deeplearning --recalibrateModel --calibrationXTrFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerVsOtherEuarchontaglireEnhLoosePeakLessThan1kb500bp_data_train.npy --calibrationYTrFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerVsOtherEuarchontaglireEnhLoosePeakLessThan1kb500bp_labels_train.npy
# Performance: loss: 0.511782, accuracy: 0.710086, sensitivity: 0.789423, specificity: 0.685689, auc: 0.813873, precision: 0.435775, NPV: 0.913712, auprc: 0.608024, auprg: 0.815388, F1: 0.561560, recallAt0.05FDR: 0.000000, recallAt0.1FDR: 0.000000
# Predictions for brain only enhancers, shared brain and liver enhancers, liver only enhancers, and negative set
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/predictNewSequencesNoEvaluation.py --architectureFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsDiShuf10XPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_DiShuf10XNeg_500bp_conv5MuchMuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal_architecture.json --weightsFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsDiShuf10XPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_DiShuf10XNeg_500bp_conv5MuchMuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal.hdf5 --narrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_nonLiver.narrowPeak.gz --sequenceLength 500 --predictedProbaFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_nonLiver_test_DiShuf10XNeg_predictedProba.txt --genomeFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseGenome/mm10.fa --chromSizesFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseGenome/mm10.chrom.sizes --chroms chr1 --chroms chr2 --classification
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/predictNewSequencesNoEvaluation.py --architectureFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsDiShuf10XPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_DiShuf10XNeg_500bp_conv5MuchMuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal_architecture.json --weightsFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsDiShuf10XPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_DiShuf10XNeg_500bp_conv5MuchMuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal.hdf5 --narrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_andLiverLiuPfenning.narrowPeak.gz --sequenceLength 500 --predictedProbaFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_andLiverLiuPfenning_test_DiShuf10XNeg_predictedProba.txt --genomeFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseGenome/mm10.fa --chromSizesFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseGenome/mm10.chrom.sizes --chroms chr1 --chroms chr2 --classification
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/predictNewSequencesNoEvaluation.py --architectureFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsDiShuf10XPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_DiShuf10XNeg_500bp_conv5MuchMuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal_architecture.json --weightsFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsDiShuf10XPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_DiShuf10XNeg_500bp_conv5MuchMuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal.hdf5 --narrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/LiverPfenning_ATAC_out/atac/68eb27d1-23c7-4d42-80f9-b3dd46f5e18c/call-reproducibility_idr/execution/idr.optimal_peak.narrowPeak_inLiuAll_nonCDS_enhancerShort_nonBrain.narrowPeak.gz --sequenceLength 500 --predictedProbaFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/LiverPfenning_ATAC_out/atac/68eb27d1-23c7-4d42-80f9-b3dd46f5e18c/call-reproducibility_idr/execution/idr.optimal_peak.narrowPeak_inLiuAll_nonCDS_enhancerShort_nonBrain_test_DiShuf10XNeg_predictedProba.txt --genomeFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseGenome/mm10.fa --chromSizesFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseGenome/mm10.chrom.sizes --chroms chr1 --chroms chr2 --classification
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/predictNewSequencesNoEvaluation.py --architectureFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsDiShuf10XPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_DiShuf10XNeg_500bp_conv5MuchMuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal_architecture.json --weightsFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsDiShuf10XPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_DiShuf10XNeg_500bp_conv5MuchMuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal.hdf5 --fastaFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_summitPlusMinus250bp_test_shufDiNuc10x.fa --sequenceLength 500 --predictedProbaFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_summitPlusMinus250bp_test_shufDiNuc10X_predictedProba.txt --genomeFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseGenome/mm10.fa --chromSizesFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseGenome/mm10.chrom.sizes --chroms chr1 --chroms chr2 --classification
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/makeViolinPlotTissueComparison.py --tissueOnlyPredFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_nonLiver_test_flankNeg_predictedProba.txt --tissueSharedPredFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_andLiverLiuPfenning_test_flankNeg_predictedProba.txt --tissueOtherPredFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/LiverPfenning_ATAC_out/atac/68eb27d1-23c7-4d42-80f9-b3dd46f5e18c/call-reproducibility_idr/execution/idr.optimal_peak.narrowPeak_inLiuAll_nonCDS_enhancerShort_nonBrain_test_flankNeg_predictedProba.txt --negSetPredFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_summitPlusMinus250bp_test_shufDiNuc10X_predictedProba.txt --outputFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_brainNonLiverVsBrainLiverVsLiverNonBrainVsDiShuf10XNeg_predictedProba.svg
# Performance on macaque shared brain and liver enhancers and liver enhancers that are not brain enhancers
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/predictNewSequences.py --architectureFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsDiShuf10XPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_DiShuf10XNeg_500bp_conv5MuchMuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal_architecture.json --weightsFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsDiShuf10XPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_DiShuf10XNeg_500bp_conv5MuchMuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal.hdf5 --narrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MacaqueAtac/OfM/peak/idr_reproducibility/idr.optimal_peak.inM1AndStriatum_nonCDS_enhancerShort_andLiver_mouseTest.bed.gz --negNarrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MacaqueAtac/Liver/cromwell-executions/atac/1f5fd9f0-d389-4184-99cb-c1125fd7f064/call-reproducibility_idr/execution/optimal_peak_nonCDS_enhancerShort_mouseTest_nonMacaqueBrain.bed.gz --sequenceLength 500 --genomeFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MacaqueGenome/rheMac8.fa --chroms chr1 --chroms chr2 --chroms chr8 --chroms chr9 --chroms chr3 --chroms chr4 --chroms chr5 --chroms chr6 --chroms chr7 --chroms chr10 --chroms chr11 --chroms chr12 --chroms chr13 --chroms chr14 --chroms chr15 --chroms chr16 --chroms chr17 --chroms chr18 --chroms chr19 --chroms chr20 --chroms chrX --chroms chrY --RC --classification --path /home/ikaplow/RegulatoryElementEvolutionProject/src/deeplearning
# Performance: loss: nan, accuracy: 0.684902, sensitivity: 0.918440, specificity: 0.580696, auc: 0.867743, precision: 0.494275, NPV: 0.941026, auprc: 0.765854, auprg: 0.854958, F1: 0.642680, recallAt0.05FDR: 0.050827, recallAt0.1FDR: 0.213948
# Performance on macaque shared brain and liver enhancers and liver enhancers that are not brain enhancers, calibrated
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/predictNewSequences.py --architectureFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsDiShuf10XPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_DiShuf10XNeg_500bp_conv5MuchMuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal_architecture.json --weightsFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsDiShuf10XPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_DiShuf10XNeg_500bp_conv5MuchMuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal.hdf5 --narrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MacaqueAtac/OfM/peak/idr_reproducibility/idr.optimal_peak.inM1AndStriatum_nonCDS_enhancerShort_andLiver_mouseTest.bed.gz --negNarrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MacaqueAtac/Liver/cromwell-executions/atac/1f5fd9f0-d389-4184-99cb-c1125fd7f064/call-reproducibility_idr/execution/optimal_peak_nonCDS_enhancerShort_mouseTest_nonMacaqueBrain.bed.gz --sequenceLength 500 --genomeFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MacaqueGenome/rheMac8.fa --chroms chr1 --chroms chr2 --chroms chr8 --chroms chr9 --chroms chr3 --chroms chr4 --chroms chr5 --chroms chr6 --chroms chr7 --chroms chr10 --chroms chr11 --chroms chr12 --chroms chr13 --chroms chr14 --chroms chr15 --chroms chr16 --chroms chr17 --chroms chr18 --chroms chr19 --chroms chr20 --chroms chrX --chroms chrY --RC --classification --path /home/ikaplow/RegulatoryElementEvolutionProject/src/deeplearning --recalibrateModel --calibrationXTrFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerVsOtherEuarchontaglireEnhLoosePeakLessThan1kb500bp_data_train.npy --calibrationYTrFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerVsOtherEuarchontaglireEnhLoosePeakLessThan1kb500bp_labels_train.npy
# Performance: loss: 0.475899, accuracy: 0.753100, sensitivity: 0.828605, specificity: 0.719409, auc: 0.867743, precision: 0.568532, NPV: 0.903910, auprc: 0.765854, auprg: 0.854958, F1: 0.674363, recallAt0.05FDR: 0.050827, recallAt0.1FDR: 0.213948
# Performance on rat shared brain and liver enhancers and liver enhancers that are not brain enhancers
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/predictNewSequences.py --architectureFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsDiShuf10XPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_DiShuf10XNeg_500bp_conv5MuchMuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal_architecture.json --weightsFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsDiShuf10XPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_DiShuf10XNeg_500bp_conv5MuchMuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal.hdf5 --narrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/RatAtac/M1_AllReps/peak/idr_reproducibility/idr.optimal_peak.inStriatum_nonCDS_enhancerShort_andLiver_mouseTest.bed.gz --negNarrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/RatAtac/Liver_GoodReps/peak/idr_reproducibility/idr.optimal_peak_nonCDS_enhancerShort_nonRatBrain_mouseTest.bed.gz --sequenceLength 500 --genomeFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/RatGenome/rn6.fa --chroms chr1 --chroms chr2 --chroms chr8 --chroms chr9 --chroms chr3 --chroms chr4 --chroms chr5 --chroms chr6 --chroms chr7 --chroms chr10 --chroms chr11 --chroms chr12 --chroms chr13 --chroms chr14 --chroms chr15 --chroms chr16 --chroms chr17 --chroms chr18 --chroms chr19 --chroms chr20 --chroms chrX --chroms chrY --RC --classification
# Preformance: loss: nan, accuracy: 0.654123, sensitivity: 0.837853, specificity: 0.561683, auc: 0.792554, precision: 0.490248, NPV: 0.873177, auprc: 0.671075, auprg: 0.708455, F1: 0.618561, recallAt0.05FDR: 0.000000, recallAt0.1FDR: 0.026554
# Performance on rat shared brain and liver enhancers and liver enhancers that are not brain enhancers, calibrated
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/predictNewSequences.py --architectureFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsDiShuf10XPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_DiShuf10XNeg_500bp_conv5MuchMuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal_architecture.json --weightsFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsDiShuf10XPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_DiShuf10XNeg_500bp_conv5MuchMuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal.hdf5 --narrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/RatAtac/M1_AllReps/peak/idr_reproducibility/idr.optimal_peak.inStriatum_nonCDS_enhancerShort_andLiver_mouseTest.bed.gz --negNarrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/RatAtac/Liver_GoodReps/peak/idr_reproducibility/idr.optimal_peak_nonCDS_enhancerShort_nonRatBrain_mouseTest.bed.gz --sequenceLength 500 --genomeFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/RatGenome/rn6.fa --chroms chr1 --chroms chr2 --chroms chr8 --chroms chr9 --chroms chr3 --chroms chr4 --chroms chr5 --chroms chr6 --chroms chr7 --chroms chr10 --chroms chr11 --chroms chr12 --chroms chr13 --chroms chr14 --chroms chr15 --chroms chr16 --chroms chr17 --chroms chr18 --chroms chr19 --chroms chr20 --chroms chrX --chroms chrY --RC --classification --recalibrateModel --calibrationXTrFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerVsOtherEuarchontaglireEnhLoosePeakLessThan1kb500bp_data_train.npy --calibrationYTrFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerVsOtherEuarchontaglireEnhLoosePeakLessThan1kb500bp_labels_train.npy
# Performance: loss: 0.475899, accuracy: 0.753100, sensitivity: 0.828605, specificity: 0.719409, auc: 0.867743, precision: 0.568532, NPV: 0.903910, auprc: 0.765854, auprg: 0.854958, F1: 0.674363, recallAt0.05FDR: 0.050827, recallAt0.1FDR: 0.213948
# Get brain test set predictions across Zoonomia mammals
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/makePredictNewSequencesNoEvaluationScript.py --architectureFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsDiShuf10XPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_DiShuf10XNeg_500bp_conv5MuchMuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal_architecture.json --weightsFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerNonCDSVsDiShuf10XPeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_DiShuf10XNeg_500bp_conv5MuchMuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal.hdf5 --narrowPeakFileNameListFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_orLiver_LiftedPeaks/liftedPeakMouseTestNarrowPeakFileNamesFiltConverted.txt --genomeFileNameListFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/200MammalsFastas/200MammalsAssembliesFileNamesFilt.txt --chromSizesFileNameListFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/200MammalsFastas/chromSizesFileNames.txt --classification --scriptFileName /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/predictMouseBrainTestHugeCactusDiShuf10XScript.sh --predictedClassesFileNameSuffix DiShuf10XNeg_predictedClasses.txt --predictedProbaFileNameSuffix DiShuf10XNeg_predictedProba.txt
sh /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/predictMouseBrainTestHugeCactusDiShuf10XScript.sh 2> /home/ikaplow/RegulatoryElementEvolutionProject/src/predictMouseBrainTestHugeCactusDiShuf10XScript.e
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/gatherPeakPredictionsAcrossSpecies.py --predictionsFileNameListFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/testPredictionsDiShuf10XGliresFileNames.txt --baseSpeciesLine 35 --outputFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_test_DiShuf10XNeg_peakPredictionsGlires.txt # Filtered correlation (Pearson, Spearman): -0.9345*,-0.7832*

# Performance of brain model with non-enhancer orthologs of loose enhancers negatives on test set
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/predictNewSequences.py --architectureFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerVsOtherEuarchontaglireEnhLoosePeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_euarchontaglireEnhLooseNeg_500bp_conv5MuchMuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal_architecture.json --weightsFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerVsOtherEuarchontaglireEnhLoosePeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_euarchontaglireEnhLooseNeg_500bp_conv5MuchMuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal.hdf5 --narrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort.narrowPeak.gz --negNarrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/HumanDNase/PMC/cromwell-executions/atac/45c4bd8c-c055-4cea-80c9-5ec0eeaf8024/call-reproducibility_idr/execution/optimal_peak.inStriatumLoose_nonCDS_enhancerShort_mm10_summitExtendedMin50Max2XProtect5_nonMouseBrain_andMacaque_andRat.narrowPeak.gz --sequenceLength 500 --genomeFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseGenome/mm10.fa --chroms chr1 --chroms chr2 --predictedProbaFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_test_andNeg_conv5MuchMuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal_predictedProba.txt --RC --classification --path /home/ikaplow/RegulatoryElementEvolutionProject/src/deeplearning
# Performance: loss: 0.412016, accuracy: 0.813808, sensitivity: 0.763112, specificity: 0.863230, auc: 0.891474, precision: 0.844702, NPV: 0.788941, auprc: 0.900562, auprg: 0.823818, F1: 0.801837, recallAt0.05FDR: 0.507867, recallAt0.1FDR: 0.673514
# Performance on mouse enhancers whose orthologs are not brain enhancers in at least one other species and negative set
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/predictNewSequences.py --architectureFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerVsOtherEuarchontaglireEnhLoosePeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_euarchontaglireEnhLooseNeg_500bp_conv5MuchMuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal_architecture.json --weightsFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerVsOtherEuarchontaglireEnhLoosePeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_euarchontaglireEnhLooseNeg_500bp_conv5MuchMuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal.hdf5 --narrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_withPeakNames_nonBrainOtherSpecies.narrowPeak.gz --negNarrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/HumanDNase/PMC/cromwell-executions/atac/45c4bd8c-c055-4cea-80c9-5ec0eeaf8024/call-reproducibility_idr/execution/optimal_peak.inStriatum_nonCDS_enhancerShort_mm10_summitExtendedMin50Max2XProtect5_nonMouseBrain_andMacaque_andRat.narrowPeak.gz --sequenceLength 500 --genomeFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseGenome/mm10.fa --chroms chr1 --chroms chr2 --RC --classification --path /home/ikaplow/RegulatoryElementEvolutionProject/src/deeplearning
# Performance: loss: 0.419569, accuracy: 0.815529, sensitivity: 0.750318, specificity: 0.865179, auc: 0.886039, precision: 0.809066, NPV: 0.819853, auprc: 0.864265, auprg: 0.836770, F1: 0.778586, recallAt0.05FDR: 0.275159, recallAt0.1FDR: 0.570064
# Performance on human enhancers whose orthologs are not brain enhancers in mouse and negative set corresponding to non-enhancer orthologs of mouse enhancers
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/predictNewSequences.py --architectureFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerVsOtherEuarchontaglireEnhLoosePeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_euarchontaglireEnhLooseNeg_500bp_conv5MuchMuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal_architecture.json --weightsFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerVsOtherEuarchontaglireEnhLoosePeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_euarchontaglireEnhLooseNeg_500bp_conv5MuchMuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal.hdf5 --narrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/HumanDNase/PMC/cromwell-executions/atac/45c4bd8c-c055-4cea-80c9-5ec0eeaf8024/call-reproducibility_idr/execution/optimal_peak.inStriatum_nonCDS_enhancerShort_nonMouseBrain_mouseTest.bed.gz --negNarrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_withPeakNames_hg38Huge_summitExtendedMin50Max2Protect5_nonHumanBrain_mouseTest.narrowPeak.gz --sequenceLength 500 --genomeFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/HumanGenome/hg38.fa --RC --classification --path /home/ikaplow/RegulatoryElementEvolutionProject/src/deeplearning
# Performance: loss: inf, accuracy: 0.734756, sensitivity: 0.639423, specificity: 0.779018, auc: 0.783128, precision: 0.573276, NPV: 0.823113, auprc: 0.643005, auprg: 0.713047, F1: 0.604545, recallAt0.05FDR: 0.000000, recallAt0.1FDR: 0.115385
# Performance on macaque enhancers whose orthologs are not brain enhancers in mouse and negative set corresponding to non-enhancer orthologs of mouse enhancers
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/predictNewSequences.py --architectureFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerVsOtherEuarchontaglireEnhLoosePeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_euarchontaglireEnhLooseNeg_500bp_conv5MuchMuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal_architecture.json --weightsFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerVsOtherEuarchontaglireEnhLoosePeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_euarchontaglireEnhLooseNeg_500bp_conv5MuchMuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal.hdf5 --narrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MacaqueAtac/OfM/peak/idr_reproducibility/idr.optimal_peak.inM1AndStriatum_nonCDS_enhancerShort_nonMouseBrain_mouseTest.bed.gz --negNarrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_withPeakNames_rheMac8Huge_summitExtendedMin50Max2Protect5_nonMacaqueBrain_mouseTest.bed.gz --sequenceLength 500 --genomeFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MacaqueGenome/rheMac8.fa --chroms chr1 --chroms chr2 --chroms chr8 --chroms chr9 --chroms chr3 --chroms chr4 --chroms chr5 --chroms chr6 --chroms chr7 --chroms chr10 --chroms chr11 --chroms chr12 --chroms chr13 --chroms chr14 --chroms chr15 --chroms chr16 --chroms chr17 --chroms chr18 --chroms chr19 --chroms chr20 --chroms chr21 --chroms chr22 --chroms chrX --chroms chrY --RC --classification --path /home/ikaplow/RegulatoryElementEvolutionProject/src/deeplearning
# Performance: loss: 0.413193, accuracy: 0.806983, sensitivity: 0.761580, specificity: 0.849490, auc: 0.895300, precision: 0.825702, NPV: 0.791914, auprc: 0.890343, auprg: 0.824208, F1: 0.792346, recallAt0.05FDR: 0.404632, recallAt0.1FDR: 0.622616
# Performance on rat enhancers whose orthologs are not brain enhancers in mouse and negative set corresponding to non-enhancer orthologs of mouse enhancers
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/predictNewSequences.py --architectureFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerVsOtherEuarchontaglireEnhLoosePeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_euarchontaglireEnhLooseNeg_500bp_conv5MuchMuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal_architecture.json --weightsFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerVsOtherEuarchontaglireEnhLoosePeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_euarchontaglireEnhLooseNeg_500bp_conv5MuchMuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal.hdf5 --narrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/RatAtac/M1_AllReps/peak/idr_reproducibility/idr.optimal_peak.inStriatum_nonCDS_enhancerShort_mouseTest_nonMouseBrain.bed.gz --negNarrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_withPeakNames_rn6Huge_summitExtendedMin50Max2Protect5_nonRatBrain_mouseTest.bed.gz --sequenceLength 500 --genomeFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/RatGenome/rn6.fa --chroms chr1 --chroms chr2 --chroms chr8 --chroms chr9 --chroms chr3 --chroms chr4 --chroms chr5 --chroms chr6 --chroms chr7 --chroms chr10 --chroms chr11 --chroms chr12 --chroms chr13 --chroms chr14 --chroms chr15 --chroms chr16 --chroms chr17 --chroms chr18 --chroms chr19 --chroms chr20 --chroms chr21 --chroms chr22 --chroms chrX --chroms chrY --RC --classification --path /home/ikaplow/RegulatoryElementEvolutionProject/src/deeplearning
# Performance: loss: 0.691549, accuracy: 0.629795, sensitivity: 0.588764, specificity: 0.683976, auc: 0.699537, precision: 0.710991, NPV: 0.557437, auprc: 0.638184, auprg: 0.434351, F1: 0.644130, recallAt0.05FDR: 0.044510, recallAt0.1FDR: 0.057864
# Performance on non-restrictive glires-specific enhancers and non-enhancers
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/predictNewSequences.py --architectureFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerVsOtherEuarchontaglireEnhLoosePeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_euarchontaglireEnhLooseNeg_500bp_conv5MuchMuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal_architecture.json --weightsFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerVsOtherEuarchontaglireEnhLoosePeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_euarchontaglireEnhLooseNeg_500bp_conv5MuchMuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal.hdf5 --narrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_inRatBrainLoose_nonHumanBrain_nonMacaqueBrain.bed.gz --negNarrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/HumanDNase/PMC/cromwell-executions/atac/45c4bd8c-c055-4cea-80c9-5ec0eeaf8024/call-reproducibility_idr/execution/optimal_peak.inStriatum_nonCDS_enhancerShort_mm10_summitExtendedMin50Max2XProtect5_inMacaqueBrainLoose_nonMouseBrain_nonRatBrain.bed.gz --sequenceLength 500 --genomeFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseGenome/mm10.fa --chroms chr1 --chroms chr2 --RC --classification --path /home/ikaplow/RegulatoryElementEvolutionProject/src/deeplearning
# Performance: loss: 0.432957, accuracy: 0.807692, sensitivity: 0.743478, specificity: 0.917910, auc: 0.912979, precision: 0.939560, NPV: 0.675824, auprc: 0.869219, auprg: 0.889326, F1: 0.830097, recallAt0.05FDR: 0.350746, recallAt0.1FDR: 0.492537
# Performance on non-restrictive glires-specific enhancers and non-enhancers, calibrated
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/predictNewSequences.py --architectureFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerVsOtherEuarchontaglireEnhLoosePeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_euarchontaglireEnhLooseNeg_500bp_conv5MuchMuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal_architecture.json --weightsFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerVsOtherEuarchontaglireEnhLoosePeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_euarchontaglireEnhLooseNeg_500bp_conv5MuchMuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal.hdf5 --narrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_inRatBrainLoose_nonHumanBrain_nonMacaqueBrain.bed.gz --negNarrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/HumanDNase/PMC/cromwell-executions/atac/45c4bd8c-c055-4cea-80c9-5ec0eeaf8024/call-reproducibility_idr/execution/optimal_peak.inStriatum_nonCDS_enhancerShort_mm10_summitExtendedMin50Max2XProtect5_inMacaqueBrainLoose_nonMouseBrain_nonRatBrain.bed.gz --sequenceLength 500 --genomeFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseGenome/mm10.fa --chroms chr1 --chroms chr2 --RC --classification --path /home/ikaplow/RegulatoryElementEvolutionProject/src/deeplearning --recalibrateModel --calibrationXTrFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerVsOtherEuarchontaglireEnhLoosePeakLessThan1kb500bp_data_train.npy --calibrationYTrFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerVsOtherEuarchontaglireEnhLoosePeakLessThan1kb500bp_labels_train.npy
# Performance: loss: 0.472082, accuracy: 0.821429, sensitivity: 0.765217, specificity: 0.917910, auc: 0.912979, precision: 0.941176, NPV: 0.694915, auprc: 0.869219, auprg: 0.889326, F1: 0.844125, recallAt0.05FDR: 0.350746, recallAt0.1FDR: 0.492537
# Performance on non-restrictive euarchonta-specific enhancers and non-enhancers
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/predictNewSequences.py --architectureFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerVsOtherEuarchontaglireEnhLoosePeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_euarchontaglireEnhLooseNeg_500bp_conv5MuchMuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal_architecture.json --weightsFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerVsOtherEuarchontaglireEnhLoosePeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_euarchontaglireEnhLooseNeg_500bp_conv5MuchMuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal.hdf5 --narrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/HumanDNase/PMC/cromwell-executions/atac/45c4bd8c-c055-4cea-80c9-5ec0eeaf8024/call-reproducibility_idr/execution/optimal_peak.inStriatum_nonCDS_enhancerShort_inMacaqueBrainLoose_nonMouseBrain_nonRatBrain_mouseTest.bed.gz --negNarrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_withPeakNames_hg38Huge_summitExtendedMin50Max2Protect5_mouseTest_inRatBrainLoose_nonHumanBrain_nonMacaqueBrain.bed.gz --sequenceLength 500 --genomeFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/HumanGenome/hg38.fa --RC --classification --path /home/ikaplow/RegulatoryElementEvolutionProject/src/deeplearning
# Performance: loss: 0.489318, accuracy: 0.755495, sensitivity: 0.597015, specificity: 0.847826, auc: 0.819046, precision: 0.695652, NPV: 0.783133, auprc: 0.769924, auprg: 0.752006, F1: 0.642570, recallAt0.05FDR: 0.291045, recallAt0.1FDR: 0.373134
# Performance on non-restrictive euarchonta-specific enhancers and non-enhancers, calibrated
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/predictNewSequences.py --architectureFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerVsOtherEuarchontaglireEnhLoosePeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_euarchontaglireEnhLooseNeg_500bp_conv5MuchMuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal_architecture.json --weightsFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerVsOtherEuarchontaglireEnhLoosePeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_euarchontaglireEnhLooseNeg_500bp_conv5MuchMuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal.hdf5 --narrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/HumanDNase/PMC/cromwell-executions/atac/45c4bd8c-c055-4cea-80c9-5ec0eeaf8024/call-reproducibility_idr/execution/optimal_peak.inStriatum_nonCDS_enhancerShort_inMacaqueBrainLoose_nonMouseBrain_nonRatBrain_mouseTest.bed.gz --negNarrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_withPeakNames_hg38Huge_summitExtendedMin50Max2Protect5_mouseTest_inRatBrainLoose_nonHumanBrain_nonMacaqueBrain.bed.gz --sequenceLength 500 --genomeFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/HumanGenome/hg38.fa --RC --classification --path /home/ikaplow/RegulatoryElementEvolutionProject/src/deeplearning --recalibrateModel --calibrationXTrFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerVsOtherEuarchontaglireEnhLoosePeakLessThan1kb500bp_data_train.npy --calibrationYTrFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerVsOtherEuarchontaglireEnhLoosePeakLessThan1kb500bp_labels_train.npy
# Performance: loss: 0.558937, accuracy: 0.750000, sensitivity: 0.611940, specificity: 0.830435, auc: 0.819046, precision: 0.677686, NPV: 0.786008, auprc: 0.769924, auprg: 0.752006, F1: 0.643137, recallAt0.05FDR: 0.291045, recallAt0.1FDR: 0.373134
# Performance on non-restrictive mouse-specific enhancers and non-enhancers
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/predictNewSequences.py --architectureFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerVsOtherEuarchontaglireEnhLoosePeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_euarchontaglireEnhLooseNeg_500bp_conv5MuchMuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal_architecture.json --weightsFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerVsOtherEuarchontaglireEnhLoosePeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_euarchontaglireEnhLooseNeg_500bp_conv5MuchMuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal.hdf5 --narrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_withPeakNames_nonRatBrain.bed.gz --negNarrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/RatAtac/M1_AllReps/peak/idr_reproducibility/idr.optimal_peak.inStriatum_nonCDS_enhancerShort_mm10_summitExtendedMin50Max2XProtect5_nonMouseBrain.bed.gz --sequenceLength 500 --genomeFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseGenome/mm10.fa --chroms chr1 --chroms chr2 --RC --classification --path /home/ikaplow/RegulatoryElementEvolutionProject/src/deeplearning
# Performance: loss: 0.437845, accuracy: 0.817775, sensitivity: 0.762611, specificity: 0.859551, auc: 0.875449, precision: 0.804382, NPV: 0.827027, auprc: 0.849273, auprg: 0.826584, F1: 0.782940, recallAt0.05FDR: 0.197329, recallAt0.1FDR: 0.562315
# Performance on non-restrictive human-specific enhancers and non-enhancers
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/predictNewSequences.py --architectureFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerVsOtherEuarchontaglireEnhLoosePeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_euarchontaglireEnhLooseNeg_500bp_conv5MuchMuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal_architecture.json --weightsFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerVsOtherEuarchontaglireEnhLoosePeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_euarchontaglireEnhLooseNeg_500bp_conv5MuchMuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal.hdf5 --narrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/HumanDNase/PMC/cromwell-executions/atac/45c4bd8c-c055-4cea-80c9-5ec0eeaf8024/call-reproducibility_idr/execution/optimal_peak.inStriatum_nonCDS_enhancerShort_nonMacaqueBrain_mouseTest.bed.gz --negNarrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MacaqueAtac/OfM/peak/idr_reproducibility/idr.optimal_peak.inM1AndStriatum_nonCDS_enhancerShort_hg38_summitExtendedMin50Max2XProtect5_nonHumanBrain_mouseTest.bed.gz --sequenceLength 500 --genomeFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/HumanGenome/hg38.fa --RC --classification --path /home/ikaplow/RegulatoryElementEvolutionProject/src/deeplearning
# Performance: loss: inf, accuracy: 0.534091, sensitivity: 0.712121, specificity: 0.474747, auc: 0.612871, precision: 0.311258, NPV: 0.831858, auprc: 0.314232, auprg: 0.342550, F1: 0.433180, recallAt0.05FDR: 0.000000, recallAt0.1FDR: 0.000000
# Performance on non-restrictive macaque-specific enhancers and non-enhancers
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/predictNewSequences.py --architectureFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerVsOtherEuarchontaglireEnhLoosePeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_euarchontaglireEnhLooseNeg_500bp_conv5MuchMuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal_architecture.json --weightsFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerVsOtherEuarchontaglireEnhLoosePeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_euarchontaglireEnhLooseNeg_500bp_conv5MuchMuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal.hdf5 --narrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MacaqueAtac/OfM/peak/idr_reproducibility/idr.optimal_peak.inM1AndStriatum_nonCDS_enhancerShort_nonHumanBrain_mouseTest.bed.gz --negNarrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/HumanDNase/PMC/cromwell-executions/atac/45c4bd8c-c055-4cea-80c9-5ec0eeaf8024/call-reproducibility_idr/execution/optimal_peak.inStriatum_nonCDS_enhancerShort_rheMac8_summitExtendedMin50Max2XProtect5_UCSCNames_nonMacaqueBrain_mouseTest.bed.gz --sequenceLength 500 --genomeFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MacaqueGenome/rheMac8.fa --RC --classification --path /home/ikaplow/RegulatoryElementEvolutionProject/src/deeplearning
# Performance: loss: 0.560823, accuracy: 0.700758, sensitivity: 0.702020, specificity: 0.696970, auc: 0.771656, precision: 0.874214, NPV: 0.438095, auprc: 0.529758, auprg: 0.710681, F1: 0.778711, recallAt0.05FDR: 0.000000, recallAt0.1FDR: 0.000000
# Performance on shared brain and liver enhancers and liver enhancers that are not brain enhancers
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/predictNewSequences.py --architectureFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerVsOtherEuarchontaglireEnhLoosePeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_euarchontaglireEnhLooseNeg_500bp_conv5MuchMuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal_architecture.json --weightsFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerVsOtherEuarchontaglireEnhLoosePeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_euarchontaglireEnhLooseNeg_500bp_conv5MuchMuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal.hdf5 --narrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_andLiverLiuPfenning.narrowPeak.gz --negNarrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/LiverPfenning_ATAC_out/atac/68eb27d1-23c7-4d42-80f9-b3dd46f5e18c/call-reproducibility_idr/execution/idr.optimal_peak.narrowPeak_inLiuAll_nonCDS_enhancerShort_nonBrain.narrowPeak.gz --sequenceLength 500 --genomeFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseGenome/mm10.fa  --chroms chr1 --chroms chr2 --RC --classification --path /home/ikaplow/RegulatoryElementEvolutionProject/src/deeplearning
# Performance: loss: 0.271033, accuracy: 0.909091, sensitivity: 0.849038, specificity: 0.927558, auc: 0.947164, precision: 0.782801, NPV: 0.952338, auprc: 0.902142, auprg: 0.976597, F1: 0.814576, recallAt0.05FDR: 0.660577, recallAt0.1FDR: 0.738462
# Performance on shared brain and liver enhancers and liver enhancers that are not brain enhancers, calibrated
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/predictNewSequences.py --architectureFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerVsOtherEuarchontaglireEnhLoosePeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_euarchontaglireEnhLooseNeg_500bp_conv5MuchMuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal_architecture.json --weightsFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerVsOtherEuarchontaglireEnhLoosePeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_euarchontaglireEnhLooseNeg_500bp_conv5MuchMuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal.hdf5 --narrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_andLiverLiuPfenning.narrowPeak.gz --negNarrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/LiverPfenning_ATAC_out/atac/68eb27d1-23c7-4d42-80f9-b3dd46f5e18c/call-reproducibility_idr/execution/idr.optimal_peak.narrowPeak_inLiuAll_nonCDS_enhancerShort_nonBrain.narrowPeak.gz --sequenceLength 500 --genomeFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseGenome/mm10.fa  --chroms chr1 --chroms chr2 --RC --classification --path /home/ikaplow/RegulatoryElementEvolutionProject/src/deeplearning --recalibrateModel --calibrationXTrFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerVsOtherEuarchontaglireEnhLoosePeakLessThan1kb500bp_data_train.npy --calibrationYTrFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerVsOtherEuarchontaglireEnhLoosePeakLessThan1kb500bp_labels_train.npy
# Performance: loss: 0.256505, accuracy: 0.905699, sensitivity: 0.859615, specificity: 0.919870, auc: 0.947164, precision: 0.767382, NPV: 0.955173, auprc: 0.902142, auprg: 0.976597, F1: 0.810884, recallAt0.05FDR: 0.660577, recallAt0.1FDR: 0.738462
# Predictions for brain only enhancers, shared brain and liver enhancers, liver only enhancers, and negative set
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/predictNewSequencesNoEvaluation.py --architectureFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerVsOtherEuarchontaglireEnhLoosePeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_euarchontaglireEnhLooseNeg_500bp_conv5MuchMuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal_architecture.json --weightsFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerVsOtherEuarchontaglireEnhLoosePeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_euarchontaglireEnhLooseNeg_500bp_conv5MuchMuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal.hdf5 --narrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_nonLiver.narrowPeak.gz --sequenceLength 500 --predictedProbaFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_nonLiver_test_predictedProba.txt --genomeFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseGenome/mm10.fa --chromSizesFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseGenome/mm10.chrom.sizes --chroms chr1 --chroms chr2 --classification
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/predictNewSequencesNoEvaluation.py --architectureFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerVsOtherEuarchontaglireEnhLoosePeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_euarchontaglireEnhLooseNeg_500bp_conv5MuchMuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal_architecture.json --weightsFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerVsOtherEuarchontaglireEnhLoosePeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_euarchontaglireEnhLooseNeg_500bp_conv5MuchMuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal.hdf5 --narrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_andLiverLiuPfenning.narrowPeak.gz --sequenceLength 500 --predictedProbaFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_andLiverLiuPfenning_test_predictedProba.txt --genomeFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseGenome/mm10.fa --chromSizesFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseGenome/mm10.chrom.sizes --chroms chr1 --chroms chr2 --classification
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/predictNewSequencesNoEvaluation.py --architectureFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerVsOtherEuarchontaglireEnhLoosePeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_euarchontaglireEnhLooseNeg_500bp_conv5MuchMuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal_architecture.json --weightsFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerVsOtherEuarchontaglireEnhLoosePeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_euarchontaglireEnhLooseNeg_500bp_conv5MuchMuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal.hdf5 --narrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/LiverPfenning_ATAC_out/atac/68eb27d1-23c7-4d42-80f9-b3dd46f5e18c/call-reproducibility_idr/execution/idr.optimal_peak.narrowPeak_inLiuAll_nonCDS_enhancerShort_nonBrain.narrowPeak.gz --sequenceLength 500 --predictedProbaFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/LiverPfenning_ATAC_out/atac/68eb27d1-23c7-4d42-80f9-b3dd46f5e18c/call-reproducibility_idr/execution/idr.optimal_peak.narrowPeak_inLiuAll_nonCDS_enhancerShort_nonBrain_test_predictedProba.txt --genomeFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseGenome/mm10.fa --chromSizesFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseGenome/mm10.chrom.sizes --chroms chr1 --chroms chr2 --classification
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/predictNewSequencesNoEvaluation.py --architectureFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerVsOtherEuarchontaglireEnhLoosePeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_euarchontaglireEnhLooseNeg_500bp_conv5MuchMuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal_architecture.json --weightsFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerVsOtherEuarchontaglireEnhLoosePeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_euarchontaglireEnhLooseNeg_500bp_conv5MuchMuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal.hdf5 --narrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/HumanDNase/PMC/cromwell-executions/atac/45c4bd8c-c055-4cea-80c9-5ec0eeaf8024/call-reproducibility_idr/execution/optimal_peak.inStriatumLoose_nonCDS_enhancerShort_mm10_summitExtendedMin50Max2XProtect5_nonMouseBrain_andMacaque_andRat.narrowPeak.gz --sequenceLength 500 --predictedProbaFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/HumanDNase/PMC/cromwell-executions/atac/45c4bd8c-c055-4cea-80c9-5ec0eeaf8024/call-reproducibility_idr/execution/optimal_peak.inStriatumLoose_nonCDS_enhancerShort_mm10_summitExtendedMin50Max2XProtect5_nonMouseBrain_andMacaque_andRat_test_predictedProba.txt --genomeFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseGenome/mm10.fa --chromSizesFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseGenome/mm10.chrom.sizes --chroms chr1 --chroms chr2 --classification
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/makeViolinPlotTissueComparison.py --tissueOnlyPredFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_nonLiver_test_flankNeg_predictedProba.txt --tissueSharedPredFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_andLiverLiuPfenning_test_flankNeg_predictedProba.txt --tissueOtherPredFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/LiverPfenning_ATAC_out/atac/68eb27d1-23c7-4d42-80f9-b3dd46f5e18c/call-reproducibility_idr/execution/idr.optimal_peak.narrowPeak_inLiuAll_nonCDS_enhancerShort_nonBrain_test_flankNeg_predictedProba.txt --negSetPredFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/HumanDNase/PMC/cromwell-executions/atac/45c4bd8c-c055-4cea-80c9-5ec0eeaf8024/call-reproducibility_idr/execution/optimal_peak.inStriatumLoose_nonCDS_enhancerShort_mm10_summitExtendedMin50Max2XProtect5_nonMouseBrain_andMacaque_andRat_test_predictedProba.txt --outputFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_brainNonLiverVsBrainLiverVsLiverNonBrainVsOtherEuarchontaglireEnhLooseNeg_predictedProba.svg
# Performance on macaque shared brain and liver enhancers and liver enhancers that are not brain enhancers
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/predictNewSequences.py --architectureFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerVsOtherEuarchontaglireEnhLoosePeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_euarchontaglireEnhLooseNeg_500bp_conv5MuchMuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal_architecture.json --weightsFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerVsOtherEuarchontaglireEnhLoosePeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_euarchontaglireEnhLooseNeg_500bp_conv5MuchMuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal.hdf5 --narrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MacaqueAtac/OfM/peak/idr_reproducibility/idr.optimal_peak.inM1AndStriatum_nonCDS_enhancerShort_andLiver_mouseTest.bed.gz --negNarrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MacaqueAtac/Liver/cromwell-executions/atac/1f5fd9f0-d389-4184-99cb-c1125fd7f064/call-reproducibility_idr/execution/optimal_peak_nonCDS_enhancerShort_mouseTest_nonMacaqueBrain.bed.gz --sequenceLength 500 --genomeFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MacaqueGenome/rheMac8.fa --chroms chr1 --chroms chr2 --chroms chr8 --chroms chr9 --chroms chr3 --chroms chr4 --chroms chr5 --chroms chr6 --chroms chr7 --chroms chr10 --chroms chr11 --chroms chr12 --chroms chr13 --chroms chr14 --chroms chr15 --chroms chr16 --chroms chr17 --chroms chr18 --chroms chr19 --chroms chr20 --chroms chrX --chroms chrY --RC --classification --path /home/ikaplow/RegulatoryElementEvolutionProject/src/deeplearning
# Performance: loss: 0.306492, accuracy: 0.891320, sensitivity: 0.873522, specificity: 0.899262, auc: 0.949166, precision: 0.794624, NPV: 0.940949, auprc: 0.904419, auprg: 0.955446, F1: 0.832207, recallAt0.05FDR: 0.491726, recallAt0.1FDR: 0.724586
# Performance on macaque shared brain and liver enhancers and liver enhancers that are not brain enhancers, calibrated
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/predictNewSequences.py --architectureFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerVsOtherEuarchontaglireEnhLoosePeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_euarchontaglireEnhLooseNeg_500bp_conv5MuchMuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal_architecture.json --weightsFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerVsOtherEuarchontaglireEnhLoosePeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_euarchontaglireEnhLooseNeg_500bp_conv5MuchMuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal.hdf5 --narrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MacaqueAtac/OfM/peak/idr_reproducibility/idr.optimal_peak.inM1AndStriatum_nonCDS_enhancerShort_andLiver_mouseTest.bed.gz --negNarrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MacaqueAtac/Liver/cromwell-executions/atac/1f5fd9f0-d389-4184-99cb-c1125fd7f064/call-reproducibility_idr/execution/optimal_peak_nonCDS_enhancerShort_mouseTest_nonMacaqueBrain.bed.gz --sequenceLength 500 --genomeFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MacaqueGenome/rheMac8.fa --chroms chr1 --chroms chr2 --chroms chr8 --chroms chr9 --chroms chr3 --chroms chr4 --chroms chr5 --chroms chr6 --chroms chr7 --chroms chr10 --chroms chr11 --chroms chr12 --chroms chr13 --chroms chr14 --chroms chr15 --chroms chr16 --chroms chr17 --chroms chr18 --chroms chr19 --chroms chr20 --chroms chrX --chroms chrY --RC --classification --path /home/ikaplow/RegulatoryElementEvolutionProject/src/deeplearning --recalibrateModel --calibrationXTrFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerVsOtherEuarchontaglireEnhLoosePeakLessThan1kb500bp_data_train.npy --calibrationYTrFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerVsOtherEuarchontaglireEnhLoosePeakLessThan1kb500bp_labels_train.npy
# Performance: loss: 0.297734, accuracy: 0.886579, sensitivity: 0.879433, specificity: 0.889768, auc: 0.949166, precision: 0.780693, NPV: 0.942985, auprc: 0.904419, auprg: 0.955446, F1: 0.827126, recallAt0.05FDR: 0.491726, recallAt0.1FDR: 0.724586
# Performance on rat shared brain and liver enhancers and liver enhancers that are not brain enhancers
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/predictNewSequences.py --architectureFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerVsOtherEuarchontaglireEnhLoosePeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_euarchontaglireEnhLooseNeg_500bp_conv5MuchMuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal_architecture.json --weightsFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerVsOtherEuarchontaglireEnhLoosePeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_euarchontaglireEnhLooseNeg_500bp_conv5MuchMuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal.hdf5 --narrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/RatAtac/M1_AllReps/peak/idr_reproducibility/idr.optimal_peak.inStriatum_nonCDS_enhancerShort_andLiver_mouseTest.bed.gz --negNarrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/RatAtac/Liver_GoodReps/peak/idr_reproducibility/idr.optimal_peak_nonCDS_enhancerShort_nonRatBrain_mouseTest.bed.gz --sequenceLength 500 --genomeFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/RatGenome/rn6.fa --chroms chr1 --chroms chr2 --chroms chr8 --chroms chr9 --chroms chr3 --chroms chr4 --chroms chr5 --chroms chr6 --chroms chr7 --chroms chr10 --chroms chr11 --chroms chr12 --chroms chr13 --chroms chr14 --chroms chr15 --chroms chr16 --chroms chr17 --chroms chr18 --chroms chr19 --chroms chr20 --chroms chrX --chroms chrY --RC --classification
# Performance: loss: 0.360945, accuracy: 0.854955, sensitivity: 0.757062, specificity: 0.904207, auc: 0.906129, precision: 0.799046, NPV: 0.880919, auprc: 0.857488, auprg: 0.911824, F1: 0.777488, recallAt0.05FDR: 0.419209, recallAt0.1FDR: 0.600565
# Performance on rat shared brain and liver enhancers and liver enhancers that are not brain enhancers, calibrated
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/predictNewSequences.py --architectureFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerVsOtherEuarchontaglireEnhLoosePeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_euarchontaglireEnhLooseNeg_500bp_conv5MuchMuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal_architecture.json --weightsFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerVsOtherEuarchontaglireEnhLoosePeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_euarchontaglireEnhLooseNeg_500bp_conv5MuchMuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal.hdf5 --narrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/RatAtac/M1_AllReps/peak/idr_reproducibility/idr.optimal_peak.inStriatum_nonCDS_enhancerShort_andLiver_mouseTest.bed.gz --negNarrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/RatAtac/Liver_GoodReps/peak/idr_reproducibility/idr.optimal_peak_nonCDS_enhancerShort_nonRatBrain_mouseTest.bed.gz --sequenceLength 500 --genomeFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/RatGenome/rn6.fa --chroms chr1 --chroms chr2 --chroms chr8 --chroms chr9 --chroms chr3 --chroms chr4 --chroms chr5 --chroms chr6 --chroms chr7 --chroms chr10 --chroms chr11 --chroms chr12 --chroms chr13 --chroms chr14 --chroms chr15 --chroms chr16 --chroms chr17 --chroms chr18 --chroms chr19 --chroms chr20 --chroms chrX --chroms chrY --RC --classification --recalibrateModel --calibrationXTrFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerVsOtherEuarchontaglireEnhLoosePeakLessThan1kb500bp_data_train.npy --calibrationYTrFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerVsOtherEuarchontaglireEnhLoosePeakLessThan1kb500bp_labels_train.npy
# Performance: loss: 0.380541, accuracy: 0.849660, sensitivity: 0.764407, specificity: 0.892553, auc: 0.906129, precision: 0.781629, NPV: 0.882766, auprc: 0.857488, auprg: 0.911824, F1: 0.772922, recallAt0.05FDR: 0.419209, recallAt0.1FDR: 0.600565
# Get brain test set predictions across Zoonomia mammals
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/makePredictNewSequencesNoEvaluationScript.py --architectureFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerVsOtherEuarchontaglireEnhLoosePeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_euarchontaglireEnhLooseNeg_500bp_conv5MuchMuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal_architecture.json --weightsFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/brainEnhancerVsOtherEuarchontaglireEnhLoosePeakLessThan1kb500bp_data_train.KerasModels/hybrid_brainEnhancerShort_euarchontaglireEnhLooseNeg_500bp_conv5MuchMuchMoreFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal.hdf5 --narrowPeakFileNameListFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_orLiver_LiftedPeaks/liftedPeakMouseTestNarrowPeakFileNamesFiltConverted.txt --genomeFileNameListFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/200MammalsFastas/200MammalsAssembliesFileNamesFilt.txt --chromSizesFileNameListFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/200MammalsFastas/chromSizesFileNames.txt --classification --scriptFileName /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/predictMouseBrainTestHugeCactusScript.sh
sh /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/predictMouseBrainTestHugeCactusScript.sh 2> /home/ikaplow/RegulatoryElementEvolutionProject/src/predictMouseBrainTestHugeCactusScript.e
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/gatherPeakPredictionsAcrossSpecies.py --predictionsFileNameListFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/testPredictionsGliresFileNames.txt --baseSpeciesLine 35 --outputFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_test_peakPredictionsGlires.txt # Filtered correlation (Pearson, Spearman): -0.8931*, -0.5740*
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/gatherPeakPredictionsAcrossSpecies.py --predictionsFileNameListFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/testPredictionsEuarchontaFileNames.txt --baseSpeciesLine 17 --outputFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_test_peakPredictionsEuarchonta.txt

# Performance of liver model with non-enhancer orthologs of loose enhancers negatives on test set
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/predictNewSequences.py --architectureFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/liverPfenningLiuEnhancerVsOtherEuarchontaglireEnhPeakLessThan1kb500bp_data_train.KerasModels/hybrid_liverPfenningLiuEnhancerShort_euarchontaglireEnhNeg_500bp_conv5MostFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal_architecture.json --weightsFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/liverPfenningLiuEnhancerVsOtherEuarchontaglireEnhPeakLessThan1kb500bp_data_train.KerasModels/hybrid_liverPfenningLiuEnhancerShort_euarchontaglireEnhNeg_500bp_conv5MostFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal.hdf5 --narrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/LiverPfenning_ATAC_out/atac/68eb27d1-23c7-4d42-80f9-b3dd46f5e18c/call-reproducibility_idr/execution/idr.optimal_peak.narrowPeak_inLiuAll_nonCDS_enhancerShort.narrowPeak.gz --negNarrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MacaqueAtac/Liver/cromwell-executions/atac/1f5fd9f0-d389-4184-99cb-c1125fd7f064/call-reproducibility_idr/execution/optimal_peak_nonCDS_enhancerShort_mm10Huge_summitExtendedMin50Max2Protect5_nonMouseLiver_andRat.narrowPeak.gz --sequenceLength 500 --genomeFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseGenome/mm10.fa --chroms chr1 --chroms chr2 --predictedProbaFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/LiverPfenning_ATAC_out/atac/68eb27d1-23c7-4d42-80f9-b3dd46f5e18c/call-reproducibility_idr/execution/idr.optimal_peak.narrowPeak_inLiuAll_nonCDS_enhancerShort_andNeg_test_conv5MostFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal_predictedProba.txt --RC --classification --path /home/ikaplow/RegulatoryElementEvolutionProject/src/deeplearning
# Performance: loss: 0.492612, accuracy: 0.756716, sensitivity: 0.736229, specificity: 0.791610, auc: 0.843743, precision: 0.857495, NPV: 0.637950, auprc: 0.747208, auprg: 0.766310, F1: 0.792249, recallAt0.05FDR: 0.027289, recallAt0.1FDR: 0.055706
# Performance on mouse enhancers whose orthologs are not liver enhancers in at least one other species and negative set
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/predictNewSequences.py --architectureFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/liverPfenningLiuEnhancerVsOtherEuarchontaglireEnhPeakLessThan1kb500bp_data_train.KerasModels/hybrid_liverPfenningLiuEnhancerShort_euarchontaglireEnhNeg_500bp_conv5MostFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal_architecture.json --weightsFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/liverPfenningLiuEnhancerVsOtherEuarchontaglireEnhPeakLessThan1kb500bp_data_train.KerasModels/hybrid_liverPfenningLiuEnhancerShort_euarchontaglireEnhNeg_500bp_conv5MostFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal.hdf5 --narrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/LiverPfenning_ATAC_out/atac/68eb27d1-23c7-4d42-80f9-b3dd46f5e18c/call-reproducibility_idr/execution/idr.optimal_peak.narrowPeak_inLiuAll_nonCDS_enhancerShort_nonLiverMacaqueOrRat.bed.gz --negNarrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MacaqueAtac/Liver_AllReps/peak/idr_reproducibility/idr.optimal_peak_nonCDS_enhancerShort_mm10_summitExtendedMin50Max2XProtect5_nonMouseLiver_andRat.narrowPeak.gz --sequenceLength 500 --genomeFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseGenome/mm10.fa --chroms chr1 --chroms chr2 --RC --classification --path /home/ikaplow/RegulatoryElementEvolutionProject/src/deeplearning
# Performance: loss: 0.511234, accuracy: 0.748657, sensitivity: 0.717228, specificity: 0.777451, auc: 0.828386, precision: 0.747005, NPV: 0.750059, auprc: 0.815566, auprg: 0.689106, F1: 0.731814, recallAt0.05FDR: 0.118780, recallAt0.1FDR: 0.346977
# Performance on non-restrictive mouse-specific enhancers and non-enhancers
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/predictNewSequences.py --architectureFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/liverPfenningLiuEnhancerVsOtherEuarchontaglireEnhPeakLessThan1kb500bp_data_train.KerasModels/hybrid_liverPfenningLiuEnhancerShort_euarchontaglireEnhNeg_500bp_conv5MostFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal_architecture.json --weightsFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/liverPfenningLiuEnhancerVsOtherEuarchontaglireEnhPeakLessThan1kb500bp_data_train.KerasModels/hybrid_liverPfenningLiuEnhancerShort_euarchontaglireEnhNeg_500bp_conv5MostFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal.hdf5 --narrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/LiverPfenning_ATAC_out/atac/68eb27d1-23c7-4d42-80f9-b3dd46f5e18c/call-reproducibility_idr/execution/idr.optimal_peak.narrowPeak_inLiuAll_nonCDS_enhancerShort_nonRatLiver.bed.gz --negNarrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/RatAtac/Liver_GoodReps/peak/idr_reproducibility/idr.optimal_peak_nonCDS_enhancerShort_mm10_summitExtendedMin50Max2XProtect5_nonMouseLiver.bed.gz --sequenceLength 500 --genomeFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseGenome/mm10.fa --chroms chr1 --chroms chr2 --RC --classification --path /home/ikaplow/RegulatoryElementEvolutionProject/src/deeplearning
# Performance: loss: 0.535171, accuracy: 0.727493, sensitivity: 0.726430, specificity: 0.728780, auc: 0.806063, precision: 0.764307, NPV: 0.687529, auprc: 0.766077, auprg: 0.654585, F1: 0.744887, recallAt0.05FDR: 0.013171, recallAt0.1FDR: 0.163415
# Performance on macaque enhancers whose orthologs are not liver enhancers in mouse and negative set corresponding to non-enhancer orthologs of mouse enhancers
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/predictNewSequences.py --architectureFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/liverPfenningLiuEnhancerVsOtherEuarchontaglireEnhPeakLessThan1kb500bp_data_train.KerasModels/hybrid_liverPfenningLiuEnhancerShort_euarchontaglireEnhNeg_500bp_conv5MostFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal_architecture.json --weightsFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/liverPfenningLiuEnhancerVsOtherEuarchontaglireEnhPeakLessThan1kb500bp_data_train.KerasModels/hybrid_liverPfenningLiuEnhancerShort_euarchontaglireEnhNeg_500bp_conv5MostFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal.hdf5 --narrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MacaqueAtac/Liver/cromwell-executions/atac/1f5fd9f0-d389-4184-99cb-c1125fd7f064/call-reproducibility_idr/execution/optimal_peak_nonCDS_enhancerShort_nonMouseLiver_mouseTest.bed.gz --negNarrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/LiverPfenning_ATAC_out/atac/68eb27d1-23c7-4d42-80f9-b3dd46f5e18c/call-reproducibility_idr/execution/idr.optimal_peak.narrowPeak_inLiuAll_nonCDS_enhancerShort_rheMac8_summitExtendedMin50Max2XProtect5_UCSCNames_nonMacaqueLiver_mouseTest.bed.gz --sequenceLength 500 --genomeFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MacaqueGenome/rheMac8.fa --RC --classification --path /home/ikaplow/RegulatoryElementEvolutionProject/src/deeplearning
# Performance: loss: 0.543670, accuracy: 0.722029, sensitivity: 0.637584, specificity: 0.812388, auc: 0.812156, precision: 0.784314, NPV: 0.676889, auprc: 0.800311, auprg: 0.656169, F1: 0.703378, recallAt0.05FDR: 0.072262, recallAt0.1FDR: 0.203770
# Performance on rat enhancers whose orthologs are not liver enhancers in mouse and negative set corresponding to non-enhancer orthologs of mouse enhancers
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/predictNewSequences.py --architectureFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/liverPfenningLiuEnhancerVsOtherEuarchontaglireEnhPeakLessThan1kb500bp_data_train.KerasModels/hybrid_liverPfenningLiuEnhancerShort_euarchontaglireEnhNeg_500bp_conv5MostFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal_architecture.json --weightsFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/liverPfenningLiuEnhancerVsOtherEuarchontaglireEnhPeakLessThan1kb500bp_data_train.KerasModels/hybrid_liverPfenningLiuEnhancerShort_euarchontaglireEnhNeg_500bp_conv5MostFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal.hdf5 --narrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/RatAtac/Liver_GoodReps/peak/idr_reproducibility/idr.optimal_peak_nonCDS_enhancerShort_nonMouseLiver_mouseTest.bed.gz --negNarrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/LiverPfenning_ATAC_out/atac/68eb27d1-23c7-4d42-80f9-b3dd46f5e18c/call-reproducibility_idr/execution/idr.optimal_peak.narrowPeak_inLiuAll_nonCDS_enhancerShort_rn6_summitExtendedMin50Max2XProtect5_nonRatLiver_mouseTest.bed.gz --sequenceLength 500 --genomeFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/RatGenome/rn6.fa --RC --classification --path /home/ikaplow/RegulatoryElementEvolutionProject/src/deeplearning
# Performance: loss: 0.664618, accuracy: 0.651147, sensitivity: 0.677561, specificity: 0.629331, auc: 0.705107, precision: 0.601559, NPV: 0.702654, auprc: 0.641004, auprg: 0.455148, F1: 0.637302, recallAt0.05FDR: 0.000000, recallAt0.1FDR: 0.000000
# Performance on non-restrictive glires-specific enhancers and non-enhancers
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/predictNewSequences.py --architectureFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/liverPfenningLiuEnhancerVsOtherEuarchontaglireEnhPeakLessThan1kb500bp_data_train.KerasModels/hybrid_liverPfenningLiuEnhancerShort_euarchontaglireEnhNeg_500bp_conv5MostFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal_architecture.json --weightsFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/liverPfenningLiuEnhancerVsOtherEuarchontaglireEnhPeakLessThan1kb500bp_data_train.KerasModels/hybrid_liverPfenningLiuEnhancerShort_euarchontaglireEnhNeg_500bp_conv5MostFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal.hdf5 --narrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/LiverPfenning_ATAC_out/atac/68eb27d1-23c7-4d42-80f9-b3dd46f5e18c/call-reproducibility_idr/execution/idr.optimal_peak.narrowPeak_inLiuAll_nonCDS_enhancerShort_inRatLiverLoose_nonMacaqueLiver.bed.gz --negNarrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MacaqueAtac/Liver/cromwell-executions/atac/1f5fd9f0-d389-4184-99cb-c1125fd7f064/call-reproducibility_idr/execution/optimal_peak_nonCDS_enhancerShort_mm10Huge_summitExtendedMin50Max2Protect5_nonMouseLiver_nonRatLiver.bed.gz --sequenceLength 500 --genomeFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseGenome/mm10.fa --chroms chr1 --chroms chr2 --RC --classification --path /home/ikaplow/RegulatoryElementEvolutionProject/src/deeplearning
# Performance: loss: 0.455394, accuracy: 0.800351, sensitivity: 0.698242, specificity: 0.857612, auc: 0.862520, precision: 0.733333, NPV: 0.835200, auprc: 0.795285, auprg: 0.826016, F1: 0.715358, recallAt0.05FDR: 0.141602, recallAt0.1FDR: 0.299805
# Performance on non-restrictive euarchonta-specific enhancers and non-enhancers
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/predictNewSequences.py --architectureFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/liverPfenningLiuEnhancerVsOtherEuarchontaglireEnhPeakLessThan1kb500bp_data_train.KerasModels/hybrid_liverPfenningLiuEnhancerShort_euarchontaglireEnhNeg_500bp_conv5MostFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal_architecture.json --weightsFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/liverPfenningLiuEnhancerVsOtherEuarchontaglireEnhPeakLessThan1kb500bp_data_train.KerasModels/hybrid_liverPfenningLiuEnhancerShort_euarchontaglireEnhNeg_500bp_conv5MostFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal.hdf5 --narrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MacaqueAtac/Liver/cromwell-executions/atac/1f5fd9f0-d389-4184-99cb-c1125fd7f064/call-reproducibility_idr/execution/optimal_peak_nonCDS_enhancerShort_nonMouseLiver_nonRatLiver_mouseTest.bed.gz --negNarrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/LiverPfenning_ATAC_out/atac/68eb27d1-23c7-4d42-80f9-b3dd46f5e18c/call-reproducibility_idr/execution/idr.optimal_peak.narrowPeak_inLiuAll_nonCDS_enhancerShort_rheMac8_summitExtendedMin50Max2XProtect5_UCSCNames_nonMacaqueLiver_inRatLiverLoose_mouseTest.bed.gz --sequenceLength 500 --genomeFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MacaqueGenome/rheMac8.fa --RC --classification --path /home/ikaplow/RegulatoryElementEvolutionProject/src/deeplearning
# Performance: loss: 0.588580, accuracy: 0.679298, sensitivity: 0.631982, specificity: 0.763672, auc: 0.777558, precision: 0.826648, NPV: 0.537827, auprc: 0.680274, auprg: 0.670008, F1: 0.716325, recallAt0.05FDR: 0.061523, recallAt0.1FDR: 0.074219
# Performance on shared liver and brain enhancers and brain enhancers that are not liver enhancers
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/predictNewSequences.py --architectureFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/liverPfenningLiuEnhancerVsOtherEuarchontaglireEnhPeakLessThan1kb500bp_data_train.KerasModels/hybrid_liverPfenningLiuEnhancerShort_euarchontaglireEnhNeg_500bp_conv5MostFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal_architecture.json --weightsFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/liverPfenningLiuEnhancerVsOtherEuarchontaglireEnhPeakLessThan1kb500bp_data_train.KerasModels/hybrid_liverPfenningLiuEnhancerShort_euarchontaglireEnhNeg_500bp_conv5MostFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal.hdf5 --narrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/LiverPfenning_ATAC_out/atac/68eb27d1-23c7-4d42-80f9-b3dd46f5e18c/call-reproducibility_idr/execution/idr.optimal_peak.narrowPeak_inLiuAll_nonCDS_enhancerShort_inBrain.narrowPeak.gz --negNarrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/Cortex_AgeB_ATAC_out/peak/macs2/idr/optimal_set/Cortex_AgeB_ATAC_out_ppr.IDR0.1.filt.inAgeBAndStriatum_enhancerShort_nonLiver.narrowPeak.gz --sequenceLength 500 --genomeFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseGenome/mm10.fa --chroms chr1 --chroms chr2 --RC --classification --path /home/ikaplow/RegulatoryElementEvolutionProject/src/deeplearning
# Performance: loss: 0.399791, accuracy: 0.840436, sensitivity: 0.797217, specificity: 0.879747, auc: 0.903458, precision: 0.857754, NPV: 0.826678, auprc: 0.914491, auprg: 0.866872, F1: 0.826378, recallAt0.05FDR: 0.633698, recallAt0.1FDR: 0.755467
# Performance on macaque shared liver and brain enhancers and brain enhancers that are not liver enhancers
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/predictNewSequences.py --architectureFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/liverPfenningLiuEnhancerVsOtherEuarchontaglireEnhPeakLessThan1kb500bp_data_train.KerasModels/hybrid_liverPfenningLiuEnhancerShort_euarchontaglireEnhNeg_500bp_conv5MostFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal_architecture.json --weightsFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/liverPfenningLiuEnhancerVsOtherEuarchontaglireEnhPeakLessThan1kb500bp_data_train.KerasModels/hybrid_liverPfenningLiuEnhancerShort_euarchontaglireEnhNeg_500bp_conv5MostFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal.hdf5 --narrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MacaqueAtac/Liver/cromwell-executions/atac/1f5fd9f0-d389-4184-99cb-c1125fd7f064/call-reproducibility_idr/execution/optimal_peak_nonCDS_enhancerShort_andBrain_mouseTest.bed.gz --negNarrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MacaqueAtac/OfM/peak/idr_reproducibility/idr.optimal_peak.inM1AndStriatum_nonCDS_enhancerShort_nonLiver_mouseTest.bed.gz --sequenceLength 500 --genomeFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MacaqueGenome/rheMac8.fa --RC --classification --path /home/ikaplow/RegulatoryElementEvolutionProject/src/deeplearning
# Performance: loss: 0.438131, accuracy: 0.815922, sensitivity: 0.800211, specificity: 0.826331, auc: 0.889856, precision: 0.753234, NPV: 0.861943, auprc: 0.858749, auprg: 0.859000, F1: 0.776012, recallAt0.05FDR: 0.403805, recallAt0.1FDR: 0.506342
# Performance on rat shared liver and brain enhancers and brain enhancers that are not liver enhancers
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/predictNewSequences.py --architectureFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/liverPfenningLiuEnhancerVsOtherEuarchontaglireEnhPeakLessThan1kb500bp_data_train.KerasModels/hybrid_liverPfenningLiuEnhancerShort_euarchontaglireEnhNeg_500bp_conv5MostFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal_architecture.json --weightsFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/liverPfenningLiuEnhancerVsOtherEuarchontaglireEnhPeakLessThan1kb500bp_data_train.KerasModels/hybrid_liverPfenningLiuEnhancerShort_euarchontaglireEnhNeg_500bp_conv5MostFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal.hdf5 --narrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/RatAtac/Liver_GoodReps/peak/idr_reproducibility/idr.optimal_peak_nonCDS_enhancerShort_inBrain_mouseTest.bed.gz --negNarrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/RatAtac/M1_AllReps/peak/idr_reproducibility/idr.optimal_peak.inStriatum_nonCDS_enhancerShort_nonBrain_mouseTest.bed.gz --sequenceLength 500 --genomeFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/RatGenome/rn6.fa --RC --classification --path /home/ikaplow/RegulatoryElementEvolutionProject/src/deeplearning
# Performance: loss: 0.471635, accuracy: 0.806483, sensitivity: 0.823894, specificity: 0.799796, auc: 0.883914, precision: 0.612500, NPV: 0.922022, auprc: 0.752439, auprg: 0.886720, F1: 0.702642, recallAt0.05FDR: 0.000000, recallAt0.1FDR: 0.000000
# Get liver test set predictions across Zoonomia mammals
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/makePredictNewSequencesNoEvaluationScript.py --architectureFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/liverPfenningLiuEnhancerVsOtherEuarchontaglireEnhPeakLessThan1kb500bp_data_train.KerasModels/hybrid_liverPfenningLiuEnhancerShort_euarchontaglireEnhNeg_500bp_conv5MostFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal_architecture.json --weightsFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/liverPfenningLiuEnhancerVsOtherEuarchontaglireEnhPeakLessThan1kb500bp_data_train.KerasModels/hybrid_liverPfenningLiuEnhancerShort_euarchontaglireEnhNeg_500bp_conv5MostFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal.hdf5 --narrowPeakFileNameListFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/LiverPfenning_ATAC_out/atac/68eb27d1-23c7-4d42-80f9-b3dd46f5e18c/call-reproducibility_idr/execution/LiftedPeaks/liftedPeakTestFileNamesConverted.txt --genomeFileNameListFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/200MammalsFastas/200MammalsAssembliesFileNamesFilt.txt --chromSizesFileNameListFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/200MammalsFastas/chromSizesFileNames.txt --classification --scriptFileName /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/predictMouseLiverTestHugeCactusScript.sh
sh /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/predictMouseLiverTestHugeCactusScript.sh > /home/ikaplow/RegulatoryElementEvolutionProject/src/predictMouseLiverTestHugeCactusScript.e
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/gatherPeakPredictionsAcrossSpecies.py --predictionsFileNameListFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/LiverPfenning_ATAC_out/atac/68eb27d1-23c7-4d42-80f9-b3dd46f5e18c/call-reproducibility_idr/execution/testPredictionsGliresFileNames.txt --baseSpeciesLine 35 --outputFileName  /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/LiverPfenning_ATAC_out/atac/68eb27d1-23c7-4d42-80f9-b3dd46f5e18c/call-reproducibility_idr/execution/idr.optimal_peak.narrowPeak_inLiuAll_nonCDS_enhancerShort_test_peakPredictionsGlires.txt # Filtered correlation (Pearson, Spearman): -0.7771*, -0.5246*
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/gatherPeakPredictionsAcrossSpecies.py --predictionsFileNameListFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/LiverPfenning_ATAC_out/atac/68eb27d1-23c7-4d42-80f9-b3dd46f5e18c/call-reproducibility_idr/execution/testPredictionsEuarchontaFileNames.txt --baseSpeciesLine 17 --outputFileName  /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/LiverPfenning_ATAC_out/atac/68eb27d1-23c7-4d42-80f9-b3dd46f5e18c/call-reproducibility_idr/execution/idr.optimal_peak.narrowPeak_inLiuAll_nonCDS_enhancerShort_test_peakPredictionsEuarchonta.txt
# Performance on non-restrictive laurasatheria-specific enhancers and non-enhancers
python /home/ikaplow/RegulatoryElementEvolutionProject/src/RegElEvoCode/predictNewSequences.py --architectureFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/liverPfenningLiuEnhancerVsOtherEuarchontaglireEnhPeakLessThan1kb500bp_data_train.KerasModels/hybrid_liverPfenningLiuEnhancerShort_euarchontaglireEnhNeg_500bp_conv5MostFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal_architecture.json --weightsFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/liverPfenningLiuEnhancerVsOtherEuarchontaglireEnhPeakLessThan1kb500bp_data_train.KerasModels/hybrid_liverPfenningLiuEnhancerShort_euarchontaglireEnhNeg_500bp_conv5MostFiltNoInterPoolVeryHighMomHighLRL2SmallBatchPretrainBal.hdf5 --narrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/CowDNase/Liver/atac/888ae501-7469-4afa-8890-e36f14f6c83b/peak/idr_reproducibility/idr.optimal_peak_nonCDS_enhancerShort_inPigLiverLoose_nonMouseLiver_nonRatLiver_nonMacaqueLiver_test.bed.gz --negNarrowPeakFileName /projects/pfenninggroup/machineLearningForComputationalBiology/regElEvoGrant/MouseDNase/LiverPfenning_ATAC_out/atac/68eb27d1-23c7-4d42-80f9-b3dd46f5e18c/call-reproducibility_idr/execution/idr.optimal_peak.inLiuAllNonLoose_nonCDS_enhancerShort_bosTau2015Fixed_summitExtendedMin50Max2XProtect5_RefSeqNames_nonCowLiver_inRatLiverLoose_inMacaqueLiverLoose_nonPigLiver_test.bed.gz --sequenceLength 500 --genomeFileName /home/ikaplow/RegulatoryElementEvolutionProject/GenomeDataPipelines/bosTau2015/GCF_000003205.7_Btau_5.0.1_genomic_withChrM.fa --RC --classification --path /home/ikaplow/RegulatoryElementEvolutionProject/src/deeplearning
# Performance: loss: 0.701225, accuracy: 0.619632, sensitivity: 0.590909, specificity: 0.645349, auc: 0.693710, precision: 0.598684, NPV: 0.637931, auprc: 0.635183, auprg: 0.366179, F1: 0.594771, recallAt0.05FDR: 0.000000, recallAt0.1FDR: 0.000000